<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Scott Bruce">
<meta name="dcterms.date" content="2025-01-13">

<title>REHAB-HF Proteomics Analysis</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/clipboard/clipboard.min.js"></script>
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/popper.min.js"></script>
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/anchor.min.js"></script>
<link href="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/libs/bootstrap/bootstrap-6e70023e03052827610af39d5bd37d6e.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">REHAB-HF Proteomics Analysis</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Scott Bruce </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">January 13, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<section id="setup-and-preliminaries" class="level1">
<h1>Setup and Preliminaries</h1>
<section id="load-libraries" class="level2">
<h2 class="anchored" data-anchor-id="load-libraries">Load Libraries</h2>
<p>First, we load all the necessary R packages for the analysis. These packages provide functions for data manipulation, statistical modeling, and visualization.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Data manipulation and reading</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(XLConnect) </span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(janitor)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Tables and plotting</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gt)</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gtsummary)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(reshape2)</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(sjPlot)</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggforce)</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gridExtra)</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggcorrplot)</span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(RColorBrewer)</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(rpart.plot)</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(GGally)</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Modeling and analysis</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(glinternet)</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(bartMachine)</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(optmatch)</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(rpart)</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lsmeans)</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(treeClust)</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(pdfCluster)</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(mediation)</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(HDtest)</span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(WGCNA)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="data-dictionary" class="level2">
<h2 class="anchored" data-anchor-id="data-dictionary">Data Dictionary</h2>
<p>Here is a brief description of the key variables used in the dataset:</p>
<ul>
<li><strong>age</strong>: Age in years</li>
<li><strong>sex</strong>: 1 for female, 0 for male</li>
<li><strong>race___4</strong>: 1 for white, 0 for non-white</li>
<li><strong>hf_cat</strong>: 1 for HFpEF (EF &gt;= 45%), 0 for HFrEF (EF &lt; 45%)</li>
<li><strong>creatinine_value</strong>: Creatinine level in mg/dL</li>
<li><strong>bl_sppb, fu_sppb</strong>: Baseline (bl) and follow-up (fu) Short Physical Performance Battery (SPPB) score</li>
<li><strong>bl_smw, fu_smw</strong>: Baseline (bl) and follow-up (fu) six-minute walk test distance in meters</li>
<li><strong>rehosp</strong>: Rehospitalization (event)</li>
<li><strong>death</strong>: Death (event)</li>
<li><strong>event</strong>: Rehospitalization or death</li>
<li><strong>lfu</strong>: Lost to follow-up</li>
<li><strong>eGFR</strong>: Estimated Glomerular Filtration Rate, derived from age, sex, and creatinine level.</li>
</ul>
</section>
<section id="ggplot2-theme" class="level2">
<h2 class="anchored" data-anchor-id="ggplot2-theme">ggplot2 Theme</h2>
<p>A custom theme is set for all <code>ggplot2</code> plots to ensure a consistent and professional appearance.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>hw <span class="ot">&lt;-</span> <span class="fu">theme_gray</span>() <span class="sc">+</span> <span class="fu">theme</span>(</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">18</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">12</span>),</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">plot.caption =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="sc">-</span><span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">strip.background =</span> <span class="fu">element_rect</span>(<span class="at">fill =</span> <span class="fu">rgb</span>(<span class="fl">0.9</span>, <span class="fl">0.95</span>, <span class="dv">1</span>), <span class="at">colour =</span> <span class="fu">gray</span>(<span class="fl">0.5</span>), <span class="at">size =</span> <span class="fl">0.2</span>),</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">panel.border =</span> <span class="fu">element_rect</span>(<span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">colour =</span> <span class="fu">gray</span>(<span class="fl">0.70</span>)),</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">panel.grid.minor.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">panel.grid.minor.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">panel.spacing.x =</span> <span class="fu">unit</span>(<span class="fl">0.2</span>, <span class="st">"cm"</span>),</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">panel.spacing.y =</span> <span class="fu">unit</span>(<span class="fl">0.2</span>, <span class="st">"cm"</span>),</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">axis.text =</span> <span class="fu">element_text</span>(<span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">margin =</span> ggplot2<span class="sc">::</span><span class="fu">margin</span>(<span class="dv">0</span>, <span class="dv">3</span>, <span class="dv">0</span>, <span class="dv">3</span>)),</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">margin =</span> ggplot2<span class="sc">::</span><span class="fu">margin</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="dv">0</span>, <span class="dv">3</span>, <span class="dv">0</span>)),</span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">16</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">14</span>),</span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">legend.title =</span> <span class="fu">element_blank</span>()</span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="data-loading-and-pre-processing" class="level1">
<h1>Data Loading and Pre-processing</h1>
<p>This section covers the loading of raw Olink proteomics data and clinical datasets, followed by cleaning, merging, and transformation steps to prepare the data for analysis.</p>
</section>
<section id="consort-diagram" class="level1">
<h1>Consort Diagram</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Define helper datasets</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>sppb_cohort <span class="ot">&lt;-</span> rhfds[rhfds<span class="sc">$</span>study_id <span class="sc">%in%</span> rprotblds<span class="sc">$</span>subject_id <span class="sc">&amp;</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>                       <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>fu_sppb) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>bl_sppb), ]</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>smw_cohort <span class="ot">&lt;-</span> rhfds[rhfds<span class="sc">$</span>study_id <span class="sc">%in%</span> rprotblds<span class="sc">$</span>subject_id <span class="sc">&amp;</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>                      <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>fu_sppb) <span class="sc">&amp;</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>                      <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>bl_smw) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>fu_smw), ]</span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>proteomics_cohort <span class="ot">&lt;-</span> rhfds[rhfds<span class="sc">$</span>study_id <span class="sc">%in%</span> rprotblds<span class="sc">$</span>subject_id <span class="sc">&amp;</span></span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>                             <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>fu_sppb) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(rhfds<span class="sc">$</span>bl_sppb) <span class="sc">&amp;</span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>                             rhfds<span class="sc">$</span>study_id <span class="sc">%in%</span> rprotfuds<span class="sc">$</span>subject_id, ]</span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>lost_or_died <span class="ot">&lt;-</span> rhfds[rhfds<span class="sc">$</span>study_id <span class="sc">%in%</span> rprotblds<span class="sc">$</span>subject_id <span class="sc">&amp;</span> <span class="fu">is.na</span>(rhfds<span class="sc">$</span>fu_sppb), ]</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Create consort summary table</span></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>consort_summary <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">Stage =</span> <span class="fu">c</span>(</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Total REHAB-HF enrolled"</span>,</span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>    <span class="st">"With baseline proteomics"</span>,</span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Lost to follow-up"</span>,</span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Deaths"</span>,</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>    <span class="st">""</span>,</span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>    <span class="st">"With baseline proteomics and baseline + follow-up SPPB"</span>,</span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Control (AC)"</span>,</span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Intervention (RI)"</span>,</span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a>    <span class="st">""</span>,</span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a>    <span class="st">"With baseline proteomics and baseline + follow-up SPPB and 6MWD"</span>,</span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Control (AC)"</span>,</span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Intervention (RI)"</span>,</span>
<span id="cb3-30"><a href="#cb3-30" aria-hidden="true" tabindex="-1"></a>    <span class="st">""</span>,</span>
<span id="cb3-31"><a href="#cb3-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">"With follow-up proteomics"</span>,</span>
<span id="cb3-32"><a href="#cb3-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Control (AC)"</span>,</span>
<span id="cb3-33"><a href="#cb3-33" aria-hidden="true" tabindex="-1"></a>    <span class="st">"  → Intervention (RI)"</span></span>
<span id="cb3-34"><a href="#cb3-34" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb3-35"><a href="#cb3-35" aria-hidden="true" tabindex="-1"></a>  <span class="at">N =</span> <span class="fu">c</span>(</span>
<span id="cb3-36"><a href="#cb3-36" aria-hidden="true" tabindex="-1"></a>    <span class="fu">nrow</span>(rhfds),</span>
<span id="cb3-37"><a href="#cb3-37" aria-hidden="true" tabindex="-1"></a>    <span class="fu">nrow</span>(rprotblds),</span>
<span id="cb3-38"><a href="#cb3-38" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(lost_or_died<span class="sc">$</span>lfu, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-39"><a href="#cb3-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(lost_or_died<span class="sc">$</span>death, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-40"><a href="#cb3-40" aria-hidden="true" tabindex="-1"></a>    <span class="cn">NA</span>,</span>
<span id="cb3-41"><a href="#cb3-41" aria-hidden="true" tabindex="-1"></a>    <span class="fu">nrow</span>(sppb_cohort),</span>
<span id="cb3-42"><a href="#cb3-42" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(sppb_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">0</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-43"><a href="#cb3-43" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(sppb_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">1</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-44"><a href="#cb3-44" aria-hidden="true" tabindex="-1"></a>    <span class="cn">NA</span>,</span>
<span id="cb3-45"><a href="#cb3-45" aria-hidden="true" tabindex="-1"></a>    <span class="fu">nrow</span>(smw_cohort),</span>
<span id="cb3-46"><a href="#cb3-46" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(smw_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">0</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-47"><a href="#cb3-47" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(smw_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">1</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-48"><a href="#cb3-48" aria-hidden="true" tabindex="-1"></a>    <span class="cn">NA</span>,</span>
<span id="cb3-49"><a href="#cb3-49" aria-hidden="true" tabindex="-1"></a>    <span class="fu">nrow</span>(proteomics_cohort),</span>
<span id="cb3-50"><a href="#cb3-50" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(proteomics_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">0</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb3-51"><a href="#cb3-51" aria-hidden="true" tabindex="-1"></a>    <span class="fu">sum</span>(proteomics_cohort<span class="sc">$</span>rand <span class="sc">==</span> <span class="dv">1</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb3-52"><a href="#cb3-52" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb3-53"><a href="#cb3-53" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb3-54"><a href="#cb3-54" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-55"><a href="#cb3-55" aria-hidden="true" tabindex="-1"></a><span class="fu">gt</span>(consort_summary) <span class="sc">%&gt;%</span></span>
<span id="cb3-56"><a href="#cb3-56" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb3-57"><a href="#cb3-57" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"CONSORT Diagram: Study Participant Flow"</span>,</span>
<span id="cb3-58"><a href="#cb3-58" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"REHAB-HF Baseline and Follow-up Analysis"</span></span>
<span id="cb3-59"><a href="#cb3-59" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-60"><a href="#cb3-60" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb3-61"><a href="#cb3-61" aria-hidden="true" tabindex="-1"></a>    <span class="at">Stage =</span> <span class="st">"Study Stage"</span>,</span>
<span id="cb3-62"><a href="#cb3-62" aria-hidden="true" tabindex="-1"></a>    <span class="at">N =</span> <span class="st">"N"</span></span>
<span id="cb3-63"><a href="#cb3-63" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-64"><a href="#cb3-64" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_style</span>(</span>
<span id="cb3-65"><a href="#cb3-65" aria-hidden="true" tabindex="-1"></a>    <span class="at">style =</span> <span class="fu">cell_text</span>(<span class="at">weight =</span> <span class="st">"bold"</span>, <span class="at">size =</span> <span class="st">"medium"</span>),</span>
<span id="cb3-66"><a href="#cb3-66" aria-hidden="true" tabindex="-1"></a>    <span class="at">locations =</span> <span class="fu">cells_body</span>(</span>
<span id="cb3-67"><a href="#cb3-67" aria-hidden="true" tabindex="-1"></a>      <span class="at">rows =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">5</span>, <span class="dv">6</span>, <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">13</span>, <span class="dv">14</span>)</span>
<span id="cb3-68"><a href="#cb3-68" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb3-69"><a href="#cb3-69" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-70"><a href="#cb3-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_style</span>(</span>
<span id="cb3-71"><a href="#cb3-71" aria-hidden="true" tabindex="-1"></a>    <span class="at">style =</span> <span class="fu">cell_text</span>(<span class="at">indent =</span> <span class="fu">px</span>(<span class="dv">20</span>)),</span>
<span id="cb3-72"><a href="#cb3-72" aria-hidden="true" tabindex="-1"></a>    <span class="at">locations =</span> <span class="fu">cells_body</span>(</span>
<span id="cb3-73"><a href="#cb3-73" aria-hidden="true" tabindex="-1"></a>      <span class="at">rows =</span> <span class="fu">c</span>(<span class="dv">7</span>, <span class="dv">8</span>, <span class="dv">11</span>, <span class="dv">12</span>, <span class="dv">15</span>, <span class="dv">16</span>)</span>
<span id="cb3-74"><a href="#cb3-74" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb3-75"><a href="#cb3-75" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-76"><a href="#cb3-76" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_style</span>(</span>
<span id="cb3-77"><a href="#cb3-77" aria-hidden="true" tabindex="-1"></a>    <span class="at">style =</span> <span class="fu">cell_fill</span>(<span class="at">color =</span> <span class="st">"#F0F0F0"</span>),</span>
<span id="cb3-78"><a href="#cb3-78" aria-hidden="true" tabindex="-1"></a>    <span class="at">locations =</span> <span class="fu">cells_body</span>(<span class="at">rows =</span> <span class="fu">c</span>(<span class="dv">5</span>, <span class="dv">9</span>, <span class="dv">13</span>))</span>
<span id="cb3-79"><a href="#cb3-79" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-80"><a href="#cb3-80" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb3-81"><a href="#cb3-81" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> N,</span>
<span id="cb3-82"><a href="#cb3-82" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb3-83"><a href="#cb3-83" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb3-84"><a href="#cb3-84" aria-hidden="true" tabindex="-1"></a>  <span class="fu">sub_missing</span>(<span class="at">missing_text =</span> <span class="st">""</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="uvricmjiva" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#uvricmjiva table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#uvricmjiva thead, #uvricmjiva tbody, #uvricmjiva tfoot, #uvricmjiva tr, #uvricmjiva td, #uvricmjiva th {
  border-style: none;
}

#uvricmjiva p {
  margin: 0;
  padding: 0;
}

#uvricmjiva .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#uvricmjiva .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#uvricmjiva .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#uvricmjiva .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#uvricmjiva .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#uvricmjiva .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#uvricmjiva .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#uvricmjiva .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#uvricmjiva .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#uvricmjiva .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#uvricmjiva .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#uvricmjiva .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#uvricmjiva .gt_spanner_row {
  border-bottom-style: hidden;
}

#uvricmjiva .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#uvricmjiva .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#uvricmjiva .gt_from_md > :first-child {
  margin-top: 0;
}

#uvricmjiva .gt_from_md > :last-child {
  margin-bottom: 0;
}

#uvricmjiva .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#uvricmjiva .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#uvricmjiva .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#uvricmjiva .gt_row_group_first td {
  border-top-width: 2px;
}

#uvricmjiva .gt_row_group_first th {
  border-top-width: 2px;
}

#uvricmjiva .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#uvricmjiva .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#uvricmjiva .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#uvricmjiva .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#uvricmjiva .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#uvricmjiva .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#uvricmjiva .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#uvricmjiva .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#uvricmjiva .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#uvricmjiva .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#uvricmjiva .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#uvricmjiva .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#uvricmjiva .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#uvricmjiva .gt_left {
  text-align: left;
}

#uvricmjiva .gt_center {
  text-align: center;
}

#uvricmjiva .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#uvricmjiva .gt_font_normal {
  font-weight: normal;
}

#uvricmjiva .gt_font_bold {
  font-weight: bold;
}

#uvricmjiva .gt_font_italic {
  font-style: italic;
}

#uvricmjiva .gt_super {
  font-size: 65%;
}

#uvricmjiva .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#uvricmjiva .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#uvricmjiva .gt_indent_1 {
  text-indent: 5px;
}

#uvricmjiva .gt_indent_2 {
  text-indent: 10px;
}

#uvricmjiva .gt_indent_3 {
  text-indent: 15px;
}

#uvricmjiva .gt_indent_4 {
  text-indent: 20px;
}

#uvricmjiva .gt_indent_5 {
  text-indent: 25px;
}

#uvricmjiva .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#uvricmjiva div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<colgroup>
<col style="width: 50%">
<col style="width: 50%">
</colgroup>
<thead>
<tr class="gt_heading header">
<th colspan="2" class="gt_heading gt_title gt_font_normal">CONSORT Diagram: Study Participant Flow</th>
</tr>
<tr class="gt_heading even">
<th colspan="2" class="gt_heading gt_subtitle gt_font_normal gt_bottom_border">REHAB-HF Baseline and Follow-up Analysis</th>
</tr>
<tr class="gt_col_headings header">
<th id="Stage" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Study Stage</th>
<th id="N" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">N</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold">Total REHAB-HF enrolled</td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold">349</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold">With baseline proteomics</td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold">243</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage">Lost to follow-up</td>
<td class="gt_row gt_right" headers="N">16</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage">Deaths</td>
<td class="gt_row gt_right" headers="N">14</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"></td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold">With baseline proteomics and baseline + follow-up SPPB</td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold">213</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Control (AC)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">101</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Intervention (RI)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">112</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"></td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold">With baseline proteomics and baseline + follow-up SPPB and 6MWD</td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold">181</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Control (AC)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">82</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Intervention (RI)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">99</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"></td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold; background-color: #F0F0F0"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="font-size: medium; font-weight: bold">With follow-up proteomics</td>
<td class="gt_row gt_right" headers="N" style="font-size: medium; font-weight: bold">153</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Control (AC)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">73</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Stage" style="text-indent: 20px">→ Intervention (RI)</td>
<td class="gt_row gt_right" headers="N" style="text-indent: 20px">80</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
</section>
<section id="baseline-patient-characteristics" class="level1">
<h1>Baseline Patient Characteristics</h1>
<section id="table-1" class="level2">
<h2 class="anchored" data-anchor-id="table-1">Table 1</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>protlist <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ALCAM"</span>,<span class="st">"Gal-4"</span>,<span class="st">"PECAM-1"</span>,<span class="st">"GP6"</span>,<span class="st">"ST2"</span>)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>varlist <span class="ot">=</span> <span class="fu">c</span>(<span class="fu">c</span>(<span class="st">"intervention_1_control_0"</span>,<span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>            <span class="st">"hf_cat"</span>,<span class="st">"egfr"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"bl_smw"</span>),protlist)</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a><span class="fu">tbl_summary</span>(rprotblds[,varlist],</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>            <span class="at">by=</span>intervention_1_control_0,</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>            <span class="at">type =</span> <span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="st">"continuous2"</span>,</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>            <span class="at">statistic =</span> <span class="fu">list</span>(<span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="fu">c</span>(<span class="st">"{mean} ({sd})"</span>,</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{median} ({p25}, {p75})"</span>, </span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{min}, {max}"</span>),</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>                             <span class="fu">all_categorical</span>() <span class="sc">~</span> <span class="st">"{n} / {N} ({p}%)"</span></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>            ),<span class="at">label =</span> <span class="fu">list</span>(age <span class="sc">~</span> <span class="st">"Age"</span>,</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>                           sex <span class="sc">~</span> <span class="st">"Sex (Female=1)"</span>,</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>                           race___4 <span class="sc">~</span> <span class="st">"Race(White=1)"</span>,</span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>                           hf_cat <span class="sc">~</span> <span class="st">"HF Category (HFpEF=1) (Study)"</span>,</span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a>                           egfr <span class="sc">~</span> <span class="st">"EGFR (Study)"</span>,</span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a>                           bl_sppb <span class="sc">~</span> <span class="st">"Baseline SPPB"</span>,</span>
<span id="cb4-19"><a href="#cb4-19" aria-hidden="true" tabindex="-1"></a>                           bl_smw <span class="sc">~</span> <span class="st">"Baseline 6MWD"</span></span>
<span id="cb4-20"><a href="#cb4-20" aria-hidden="true" tabindex="-1"></a>            ),</span>
<span id="cb4-21"><a href="#cb4-21" aria-hidden="true" tabindex="-1"></a>            <span class="at">missing_text =</span> <span class="st">"(Missing)"</span></span>
<span id="cb4-22"><a href="#cb4-22" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span> <span class="fu">add_p</span>(<span class="fu">list</span>(<span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="st">"wilcox.test"</span>,<span class="fu">all_categorical</span>() <span class="sc">~</span> <span class="st">"fisher.test"</span>)) <span class="sc">%&gt;%</span> <span class="fu">add_n</span>(</span>
<span id="cb4-23"><a href="#cb4-23" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span> <span class="fu">modify_header</span>(label <span class="sc">~</span> <span class="st">"**Variable**"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb4-24"><a href="#cb4-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">modify_caption</span>(<span class="st">"**Table 1. Baseline Patient Characteristics**"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-25"><a href="#cb4-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">modify_spanning_header</span>(<span class="fu">c</span>(<span class="st">"stat_1"</span>, <span class="st">"stat_2"</span>) <span class="sc">~</span> <span class="st">"**Control (0) vs. Intervention (1)**"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="qarhrawtry" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#qarhrawtry table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#qarhrawtry thead, #qarhrawtry tbody, #qarhrawtry tfoot, #qarhrawtry tr, #qarhrawtry td, #qarhrawtry th {
  border-style: none;
}

#qarhrawtry p {
  margin: 0;
  padding: 0;
}

#qarhrawtry .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#qarhrawtry .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#qarhrawtry .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#qarhrawtry .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#qarhrawtry .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qarhrawtry .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qarhrawtry .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#qarhrawtry .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#qarhrawtry .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#qarhrawtry .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#qarhrawtry .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#qarhrawtry .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#qarhrawtry .gt_spanner_row {
  border-bottom-style: hidden;
}

#qarhrawtry .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#qarhrawtry .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#qarhrawtry .gt_from_md > :first-child {
  margin-top: 0;
}

#qarhrawtry .gt_from_md > :last-child {
  margin-bottom: 0;
}

#qarhrawtry .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#qarhrawtry .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#qarhrawtry .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#qarhrawtry .gt_row_group_first td {
  border-top-width: 2px;
}

#qarhrawtry .gt_row_group_first th {
  border-top-width: 2px;
}

#qarhrawtry .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qarhrawtry .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#qarhrawtry .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#qarhrawtry .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qarhrawtry .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#qarhrawtry .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#qarhrawtry .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#qarhrawtry .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#qarhrawtry .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#qarhrawtry .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qarhrawtry .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qarhrawtry .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#qarhrawtry .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#qarhrawtry .gt_left {
  text-align: left;
}

#qarhrawtry .gt_center {
  text-align: center;
}

#qarhrawtry .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#qarhrawtry .gt_font_normal {
  font-weight: normal;
}

#qarhrawtry .gt_font_bold {
  font-weight: bold;
}

#qarhrawtry .gt_font_italic {
  font-style: italic;
}

#qarhrawtry .gt_super {
  font-size: 65%;
}

#qarhrawtry .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#qarhrawtry .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#qarhrawtry .gt_indent_1 {
  text-indent: 5px;
}

#qarhrawtry .gt_indent_2 {
  text-indent: 10px;
}

#qarhrawtry .gt_indent_3 {
  text-indent: 15px;
}

#qarhrawtry .gt_indent_4 {
  text-indent: 20px;
}

#qarhrawtry .gt_indent_5 {
  text-indent: 25px;
}

#qarhrawtry .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#qarhrawtry div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<caption><strong>Table 1. Baseline Patient Characteristics</strong></caption>
<colgroup>
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="gt_col_headings gt_spanner_row header">
<th rowspan="2" id="label" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><strong>Variable</strong></th>
<th rowspan="2" id="n" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>N</strong></th>
<th colspan="2" id="**Control (0) vs. Intervention (1)**" class="gt_center gt_columns_top_border gt_column_spanner_outer" data-quarto-table-cell-role="th" scope="colgroup"><div class="gt_column_spanner">
<strong>Control (0) vs.&nbsp;Intervention (1)</strong>
</div></th>
<th rowspan="2" id="p.value" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>p-value</strong><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>2</sup></span></th>
</tr>
<tr class="gt_col_headings even">
<th id="stat_1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>0</strong><br> N = 113<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>1</strong><br> N = 130<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">&gt;0.9</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">72 (8)</td>
<td class="gt_row gt_center" headers="stat_2">73 (8)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">72 (66, 77)</td>
<td class="gt_row gt_center" headers="stat_2">72 (65, 79)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">60, 90</td>
<td class="gt_row gt_center" headers="stat_2">60, 100</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Sex (Female=1)</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.12</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">48 / 113 (42%)</td>
<td class="gt_row gt_center" headers="stat_2">69 / 130 (53%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">65 / 113 (58%)</td>
<td class="gt_row gt_center" headers="stat_2">61 / 130 (47%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Race(White=1)</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.4</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">55 / 113 (49%)</td>
<td class="gt_row gt_center" headers="stat_2">55 / 130 (42%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">58 / 113 (51%)</td>
<td class="gt_row gt_center" headers="stat_2">75 / 130 (58%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">HF Category (HFpEF=1) (Study)</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.5</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">56 / 113 (50%)</td>
<td class="gt_row gt_center" headers="stat_2">59 / 130 (45%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">57 / 113 (50%)</td>
<td class="gt_row gt_center" headers="stat_2">71 / 130 (55%)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">EGFR (Study)</td>
<td class="gt_row gt_center" headers="n">242</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">56 (23)</td>
<td class="gt_row gt_center" headers="stat_2">53 (22)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">57 (34, 70)</td>
<td class="gt_row gt_center" headers="stat_2">49 (36, 70)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">18, 101</td>
<td class="gt_row gt_center" headers="stat_2">17, 107</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;(Missing)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Baseline SPPB</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.8</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.15 (2.58)</td>
<td class="gt_row gt_center" headers="stat_2">6.07 (2.74)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.00 (4.00, 8.00)</td>
<td class="gt_row gt_center" headers="stat_2">6.00 (4.00, 8.00)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1.00, 12.00</td>
<td class="gt_row gt_center" headers="stat_2">1.00, 12.00</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Baseline 6MWD</td>
<td class="gt_row gt_center" headers="n">237</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">194 (109)</td>
<td class="gt_row gt_center" headers="stat_2">200 (108)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">175 (102, 260)</td>
<td class="gt_row gt_center" headers="stat_2">188 (100, 270)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">30, 489</td>
<td class="gt_row gt_center" headers="stat_2">12, 547</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;(Missing)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1</td>
<td class="gt_row gt_center" headers="stat_2">5</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">ALCAM</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.6</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.68 (0.38)</td>
<td class="gt_row gt_center" headers="stat_2">6.68 (0.39)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.68 (6.44, 6.91)</td>
<td class="gt_row gt_center" headers="stat_2">6.66 (6.40, 6.89)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">5.16, 7.82</td>
<td class="gt_row gt_center" headers="stat_2">5.99, 8.28</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Gal-4</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.8</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">3.94 (0.67)</td>
<td class="gt_row gt_center" headers="stat_2">3.96 (0.69)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">3.93 (3.58, 4.36)</td>
<td class="gt_row gt_center" headers="stat_2">3.95 (3.54, 4.37)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">2.45, 6.13</td>
<td class="gt_row gt_center" headers="stat_2">2.38, 6.45</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">PECAM-1</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.5</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">3.46 (0.41)</td>
<td class="gt_row gt_center" headers="stat_2">3.41 (0.34)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">3.45 (3.22, 3.71)</td>
<td class="gt_row gt_center" headers="stat_2">3.44 (3.19, 3.65)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">2.17, 4.75</td>
<td class="gt_row gt_center" headers="stat_2">2.36, 4.34</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">GP6</td>
<td class="gt_row gt_center" headers="n">242</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1.86 (0.54)</td>
<td class="gt_row gt_center" headers="stat_2">1.78 (0.58)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1.86 (1.50, 2.16)</td>
<td class="gt_row gt_center" headers="stat_2">1.73 (1.39, 2.17)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">0.75, 4.00</td>
<td class="gt_row gt_center" headers="stat_2">0.55, 3.55</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;(Missing)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">1</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">ST2</td>
<td class="gt_row gt_center" headers="n">243</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="p.value">0.053</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.64 (0.88)</td>
<td class="gt_row gt_center" headers="stat_2">6.85 (0.89)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">6.55 (5.95, 7.26)</td>
<td class="gt_row gt_center" headers="stat_2">6.77 (6.29, 7.39)</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="n"><br>
</td>
<td class="gt_row gt_center" headers="stat_1">4.89, 9.44</td>
<td class="gt_row gt_center" headers="stat_2">4.76, 9.43</td>
<td class="gt_row gt_center" headers="p.value"><br>
</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="5" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span> n / N (%)</td>
</tr>
<tr class="even">
<td colspan="5" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>2</sup></span> Wilcoxon rank sum test; Fisher’s exact test</td>
</tr>
</tfoot>

</table>

</div>
</div>
</div>
</section>
</section>
<section id="baseline-proteomics" class="level1">
<h1>Baseline Proteomics</h1>
<section id="data-view" class="level2">
<h2 class="anchored" data-anchor-id="data-view">Data View</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">gt</span>(<span class="fu">head</span>(rprotblds[,<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>]))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="deodemfyql" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#deodemfyql table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#deodemfyql thead, #deodemfyql tbody, #deodemfyql tfoot, #deodemfyql tr, #deodemfyql td, #deodemfyql th {
  border-style: none;
}

#deodemfyql p {
  margin: 0;
  padding: 0;
}

#deodemfyql .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#deodemfyql .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#deodemfyql .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#deodemfyql .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#deodemfyql .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#deodemfyql .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#deodemfyql .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#deodemfyql .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#deodemfyql .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#deodemfyql .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#deodemfyql .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#deodemfyql .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#deodemfyql .gt_spanner_row {
  border-bottom-style: hidden;
}

#deodemfyql .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#deodemfyql .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#deodemfyql .gt_from_md > :first-child {
  margin-top: 0;
}

#deodemfyql .gt_from_md > :last-child {
  margin-bottom: 0;
}

#deodemfyql .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#deodemfyql .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#deodemfyql .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#deodemfyql .gt_row_group_first td {
  border-top-width: 2px;
}

#deodemfyql .gt_row_group_first th {
  border-top-width: 2px;
}

#deodemfyql .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#deodemfyql .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#deodemfyql .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#deodemfyql .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#deodemfyql .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#deodemfyql .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#deodemfyql .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#deodemfyql .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#deodemfyql .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#deodemfyql .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#deodemfyql .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#deodemfyql .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#deodemfyql .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#deodemfyql .gt_left {
  text-align: left;
}

#deodemfyql .gt_center {
  text-align: center;
}

#deodemfyql .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#deodemfyql .gt_font_normal {
  font-weight: normal;
}

#deodemfyql .gt_font_bold {
  font-weight: bold;
}

#deodemfyql .gt_font_italic {
  font-style: italic;
}

#deodemfyql .gt_super {
  font-size: 65%;
}

#deodemfyql .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#deodemfyql .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#deodemfyql .gt_indent_1 {
  text-indent: 5px;
}

#deodemfyql .gt_indent_2 {
  text-indent: 10px;
}

#deodemfyql .gt_indent_3 {
  text-indent: 15px;
}

#deodemfyql .gt_indent_4 {
  text-indent: 20px;
}

#deodemfyql .gt_indent_5 {
  text-indent: 25px;
}

#deodemfyql .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#deodemfyql div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_col_headings header">
<th id="TNFRSF14" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TNFRSF14</th>
<th id="LDL-receptor" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">LDL receptor</th>
<th id="ITGB2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">ITGB2</th>
<th id="IL-17RA" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL-17RA</th>
<th id="TNF-R2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TNF-R2</th>
<th id="MMP-9" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MMP-9</th>
<th id="EPHB4" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">EPHB4</th>
<th id="IL2-RA" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL2-RA</th>
<th id="OPG" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">OPG</th>
<th id="ALCAM" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">ALCAM</th>
<th id="TFF3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TFF3</th>
<th id="SELP" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">SELP</th>
<th id="CSTB" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CSTB</th>
<th id="MCP-1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MCP-1</th>
<th id="CD163" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CD163</th>
<th id="Gal-3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Gal-3</th>
<th id="GRN" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">GRN</th>
<th id="NT-proBNP" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">NT-proBNP</th>
<th id="BLM-hydrolase" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">BLM hydrolase</th>
<th id="PLC" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PLC</th>
<th id="LTBR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">LTBR</th>
<th id="Notch-3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Notch 3</th>
<th id="TIMP4" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TIMP4</th>
<th id="CNTN1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CNTN1</th>
<th id="CDH5" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CDH5</th>
<th id="TLT-2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TLT-2</th>
<th id="FABP4" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">FABP4</th>
<th id="TFPI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TFPI</th>
<th id="PAI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PAI</th>
<th id="CCL24" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CCL24</th>
<th id="TR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TR</th>
<th id="TNFRSF10C" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TNFRSF10C</th>
<th id="GDF-15" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">GDF-15</th>
<th id="SELE" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">SELE</th>
<th id="AZU1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">AZU1</th>
<th id="DLK-1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">DLK-1</th>
<th id="SPON1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">SPON1</th>
<th id="MPO" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MPO</th>
<th id="CXCL16" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CXCL16</th>
<th id="IL-6RA" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL-6RA</th>
<th id="RETN" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">RETN</th>
<th id="IGFBP-1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IGFBP-1</th>
<th id="CHIT1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CHIT1</th>
<th id="TR-AP" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TR-AP</th>
<th id="GP6" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">GP6</th>
<th id="PSP-D" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PSP-D</th>
<th id="PI3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PI3</th>
<th id="Ep-CAM" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Ep-CAM</th>
<th id="AP-N" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">AP-N</th>
<th id="AXL" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">AXL</th>
<th id="IL-1RT1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL-1RT1</th>
<th id="MMP-2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MMP-2</th>
<th id="FAS" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">FAS</th>
<th id="MB" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MB</th>
<th id="TNFSF13B" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TNFSF13B</th>
<th id="PRTN3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PRTN3</th>
<th id="PCSK9" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PCSK9</th>
<th id="U-PAR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">U-PAR</th>
<th id="OPN" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">OPN</th>
<th id="CTSD" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CTSD</th>
<th id="PGLYRP1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PGLYRP1</th>
<th id="CPA1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CPA1</th>
<th id="JAM-A" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">JAM-A</th>
<th id="Gal-4" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Gal-4</th>
<th id="IL-1RT2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL-1RT2</th>
<th id="SHPS-1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">SHPS-1</th>
<th id="CCL15" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CCL15</th>
<th id="CASP-3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CASP-3</th>
<th id="uPA" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">uPA</th>
<th id="CPB1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CPB1</th>
<th id="CHI3L1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CHI3L1</th>
<th id="ST2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">ST2</th>
<th id="t-PA" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">t-PA</th>
<th id="SCGB3A2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">SCGB3A2</th>
<th id="EGFR" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">EGFR</th>
<th id="IGFBP-7" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IGFBP-7</th>
<th id="CD93" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CD93</th>
<th id="IL-18BP" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IL-18BP</th>
<th id="COL1A1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">COL1A1</th>
<th id="PON3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PON3</th>
<th id="CTSZ" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CTSZ</th>
<th id="MMP-3" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MMP-3</th>
<th id="RARRES2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">RARRES2</th>
<th id="ICAM-2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">ICAM-2</th>
<th id="KLK6" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">KLK6</th>
<th id="PDGF-subunit-A" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PDGF subunit A</th>
<th id="TNF-R1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">TNF-R1</th>
<th id="IGFBP-2" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">IGFBP-2</th>
<th id="vWF" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">vWF</th>
<th id="PECAM-1" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">PECAM-1</th>
<th id="MEPE" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">MEPE</th>
<th id="CCL16" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">CCL16</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_right" headers="TNFRSF14">4.73661</td>
<td class="gt_row gt_right" headers="LDL receptor">3.85547</td>
<td class="gt_row gt_right" headers="ITGB2">4.96392</td>
<td class="gt_row gt_right" headers="IL-17RA">3.08914</td>
<td class="gt_row gt_right" headers="TNF-R2">5.80069</td>
<td class="gt_row gt_right" headers="MMP-9">5.74557</td>
<td class="gt_row gt_right" headers="EPHB4">4.93904</td>
<td class="gt_row gt_right" headers="IL2-RA">3.77207</td>
<td class="gt_row gt_right" headers="OPG">3.37529</td>
<td class="gt_row gt_right" headers="ALCAM">6.87897</td>
<td class="gt_row gt_right" headers="TFF3">6.65321</td>
<td class="gt_row gt_right" headers="SELP">8.48135</td>
<td class="gt_row gt_right" headers="CSTB">4.83981</td>
<td class="gt_row gt_right" headers="MCP-1">4.41079</td>
<td class="gt_row gt_right" headers="CD163">7.76543</td>
<td class="gt_row gt_right" headers="Gal-3">4.29580</td>
<td class="gt_row gt_right" headers="GRN">5.13775</td>
<td class="gt_row gt_right" headers="NT-proBNP">5.74908</td>
<td class="gt_row gt_right" headers="BLM hydrolase">2.58250</td>
<td class="gt_row gt_right" headers="PLC">7.92702</td>
<td class="gt_row gt_right" headers="LTBR">3.56195</td>
<td class="gt_row gt_right" headers="Notch 3">5.23867</td>
<td class="gt_row gt_right" headers="TIMP4">4.18374</td>
<td class="gt_row gt_right" headers="CNTN1">3.91939</td>
<td class="gt_row gt_right" headers="CDH5">3.35420</td>
<td class="gt_row gt_right" headers="TLT-2">4.89858</td>
<td class="gt_row gt_right" headers="FABP4">5.59521</td>
<td class="gt_row gt_right" headers="TFPI">8.27001</td>
<td class="gt_row gt_right" headers="PAI">7.15176</td>
<td class="gt_row gt_right" headers="CCL24">5.39381</td>
<td class="gt_row gt_right" headers="TR">5.27216</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.50506</td>
<td class="gt_row gt_right" headers="GDF-15">5.92491</td>
<td class="gt_row gt_right" headers="SELE">9.02221</td>
<td class="gt_row gt_right" headers="AZU1">4.62573</td>
<td class="gt_row gt_right" headers="DLK-1">5.57743</td>
<td class="gt_row gt_right" headers="SPON1">0.82724</td>
<td class="gt_row gt_right" headers="MPO">2.84667</td>
<td class="gt_row gt_right" headers="CXCL16">5.53610</td>
<td class="gt_row gt_right" headers="IL-6RA">11.95979</td>
<td class="gt_row gt_right" headers="RETN">6.14425</td>
<td class="gt_row gt_right" headers="IGFBP-1">2.77427</td>
<td class="gt_row gt_right" headers="CHIT1">4.33382</td>
<td class="gt_row gt_right" headers="TR-AP">4.01411</td>
<td class="gt_row gt_right" headers="GP6">1.69832</td>
<td class="gt_row gt_right" headers="PSP-D">4.37844</td>
<td class="gt_row gt_right" headers="PI3">1.17044</td>
<td class="gt_row gt_right" headers="Ep-CAM">5.23741</td>
<td class="gt_row gt_right" headers="AP-N">5.17074</td>
<td class="gt_row gt_right" headers="AXL">8.49625</td>
<td class="gt_row gt_right" headers="IL-1RT1">5.34028</td>
<td class="gt_row gt_right" headers="MMP-2">3.00813</td>
<td class="gt_row gt_right" headers="FAS">5.53691</td>
<td class="gt_row gt_right" headers="MB">5.86565</td>
<td class="gt_row gt_right" headers="TNFSF13B">6.71477</td>
<td class="gt_row gt_right" headers="PRTN3">4.27130</td>
<td class="gt_row gt_right" headers="PCSK9">2.39691</td>
<td class="gt_row gt_right" headers="U-PAR">5.10845</td>
<td class="gt_row gt_right" headers="OPN">8.03561</td>
<td class="gt_row gt_right" headers="CTSD">3.34399</td>
<td class="gt_row gt_right" headers="PGLYRP1">7.25139</td>
<td class="gt_row gt_right" headers="CPA1">5.37452</td>
<td class="gt_row gt_right" headers="JAM-A">4.92844</td>
<td class="gt_row gt_right" headers="Gal-4">4.36884</td>
<td class="gt_row gt_right" headers="IL-1RT2">5.55845</td>
<td class="gt_row gt_right" headers="SHPS-1">3.71619</td>
<td class="gt_row gt_right" headers="CCL15">7.10926</td>
<td class="gt_row gt_right" headers="CASP-3">4.48483</td>
<td class="gt_row gt_right" headers="uPA">4.71291</td>
<td class="gt_row gt_right" headers="CPB1">5.77280</td>
<td class="gt_row gt_right" headers="CHI3L1">6.75214</td>
<td class="gt_row gt_right" headers="ST2">6.47088</td>
<td class="gt_row gt_right" headers="t-PA">5.73131</td>
<td class="gt_row gt_right" headers="SCGB3A2">1.77978</td>
<td class="gt_row gt_right" headers="EGFR">1.97475</td>
<td class="gt_row gt_right" headers="IGFBP-7">9.14003</td>
<td class="gt_row gt_right" headers="CD93">10.55061</td>
<td class="gt_row gt_right" headers="IL-18BP">6.15788</td>
<td class="gt_row gt_right" headers="COL1A1">2.19270</td>
<td class="gt_row gt_right" headers="PON3">3.38888</td>
<td class="gt_row gt_right" headers="CTSZ">4.24600</td>
<td class="gt_row gt_right" headers="MMP-3">6.15439</td>
<td class="gt_row gt_right" headers="RARRES2">11.02278</td>
<td class="gt_row gt_right" headers="ICAM-2">4.77760</td>
<td class="gt_row gt_right" headers="KLK6">3.55571</td>
<td class="gt_row gt_right" headers="PDGF subunit A">3.37982</td>
<td class="gt_row gt_right" headers="TNF-R1">6.79072</td>
<td class="gt_row gt_right" headers="IGFBP-2">6.72250</td>
<td class="gt_row gt_right" headers="vWF">7.17247</td>
<td class="gt_row gt_right" headers="PECAM-1">3.53220</td>
<td class="gt_row gt_right" headers="MEPE">3.84404</td>
<td class="gt_row gt_right" headers="CCL16">5.76281</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="TNFRSF14">4.98605</td>
<td class="gt_row gt_right" headers="LDL receptor">3.91227</td>
<td class="gt_row gt_right" headers="ITGB2">4.55702</td>
<td class="gt_row gt_right" headers="IL-17RA">3.31345</td>
<td class="gt_row gt_right" headers="TNF-R2">5.82976</td>
<td class="gt_row gt_right" headers="MMP-9">6.50820</td>
<td class="gt_row gt_right" headers="EPHB4">5.11034</td>
<td class="gt_row gt_right" headers="IL2-RA">4.36992</td>
<td class="gt_row gt_right" headers="OPG">3.36807</td>
<td class="gt_row gt_right" headers="ALCAM">6.53996</td>
<td class="gt_row gt_right" headers="TFF3">6.01821</td>
<td class="gt_row gt_right" headers="SELP">9.07367</td>
<td class="gt_row gt_right" headers="CSTB">4.64078</td>
<td class="gt_row gt_right" headers="MCP-1">5.00372</td>
<td class="gt_row gt_right" headers="CD163">7.17236</td>
<td class="gt_row gt_right" headers="Gal-3">3.59748</td>
<td class="gt_row gt_right" headers="GRN">4.52829</td>
<td class="gt_row gt_right" headers="NT-proBNP">6.06865</td>
<td class="gt_row gt_right" headers="BLM hydrolase">2.10993</td>
<td class="gt_row gt_right" headers="PLC">7.77587</td>
<td class="gt_row gt_right" headers="LTBR">3.76963</td>
<td class="gt_row gt_right" headers="Notch 3">4.99723</td>
<td class="gt_row gt_right" headers="TIMP4">4.92089</td>
<td class="gt_row gt_right" headers="CNTN1">3.66466</td>
<td class="gt_row gt_right" headers="CDH5">2.97815</td>
<td class="gt_row gt_right" headers="TLT-2">4.90635</td>
<td class="gt_row gt_right" headers="FABP4">5.47129</td>
<td class="gt_row gt_right" headers="TFPI">7.71035</td>
<td class="gt_row gt_right" headers="PAI">7.34568</td>
<td class="gt_row gt_right" headers="CCL24">5.81635</td>
<td class="gt_row gt_right" headers="TR">5.88100</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.91758</td>
<td class="gt_row gt_right" headers="GDF-15">5.69169</td>
<td class="gt_row gt_right" headers="SELE">10.26007</td>
<td class="gt_row gt_right" headers="AZU1">5.17222</td>
<td class="gt_row gt_right" headers="DLK-1">5.57787</td>
<td class="gt_row gt_right" headers="SPON1">0.91788</td>
<td class="gt_row gt_right" headers="MPO">2.72447</td>
<td class="gt_row gt_right" headers="CXCL16">4.89757</td>
<td class="gt_row gt_right" headers="IL-6RA">11.11881</td>
<td class="gt_row gt_right" headers="RETN">6.36880</td>
<td class="gt_row gt_right" headers="IGFBP-1">5.95786</td>
<td class="gt_row gt_right" headers="CHIT1">3.94635</td>
<td class="gt_row gt_right" headers="TR-AP">3.09403</td>
<td class="gt_row gt_right" headers="GP6">2.27435</td>
<td class="gt_row gt_right" headers="PSP-D">3.97338</td>
<td class="gt_row gt_right" headers="PI3">2.31011</td>
<td class="gt_row gt_right" headers="Ep-CAM">3.14197</td>
<td class="gt_row gt_right" headers="AP-N">5.10695</td>
<td class="gt_row gt_right" headers="AXL">8.49731</td>
<td class="gt_row gt_right" headers="IL-1RT1">5.50637</td>
<td class="gt_row gt_right" headers="MMP-2">2.95037</td>
<td class="gt_row gt_right" headers="FAS">5.53515</td>
<td class="gt_row gt_right" headers="MB">6.06977</td>
<td class="gt_row gt_right" headers="TNFSF13B">6.54673</td>
<td class="gt_row gt_right" headers="PRTN3">4.78984</td>
<td class="gt_row gt_right" headers="PCSK9">1.66808</td>
<td class="gt_row gt_right" headers="U-PAR">5.37624</td>
<td class="gt_row gt_right" headers="OPN">8.67810</td>
<td class="gt_row gt_right" headers="CTSD">3.16246</td>
<td class="gt_row gt_right" headers="PGLYRP1">7.78350</td>
<td class="gt_row gt_right" headers="CPA1">3.88712</td>
<td class="gt_row gt_right" headers="JAM-A">4.98776</td>
<td class="gt_row gt_right" headers="Gal-4">3.67846</td>
<td class="gt_row gt_right" headers="IL-1RT2">4.87369</td>
<td class="gt_row gt_right" headers="SHPS-1">3.36575</td>
<td class="gt_row gt_right" headers="CCL15">7.22818</td>
<td class="gt_row gt_right" headers="CASP-3">4.85531</td>
<td class="gt_row gt_right" headers="uPA">4.15206</td>
<td class="gt_row gt_right" headers="CPB1">4.01961</td>
<td class="gt_row gt_right" headers="CHI3L1">7.13188</td>
<td class="gt_row gt_right" headers="ST2">6.78645</td>
<td class="gt_row gt_right" headers="t-PA">4.71672</td>
<td class="gt_row gt_right" headers="SCGB3A2">1.47188</td>
<td class="gt_row gt_right" headers="EGFR">1.67870</td>
<td class="gt_row gt_right" headers="IGFBP-7">8.90523</td>
<td class="gt_row gt_right" headers="CD93">10.49207</td>
<td class="gt_row gt_right" headers="IL-18BP">6.05445</td>
<td class="gt_row gt_right" headers="COL1A1">2.49735</td>
<td class="gt_row gt_right" headers="PON3">3.33081</td>
<td class="gt_row gt_right" headers="CTSZ">3.97091</td>
<td class="gt_row gt_right" headers="MMP-3">6.37387</td>
<td class="gt_row gt_right" headers="RARRES2">10.85461</td>
<td class="gt_row gt_right" headers="ICAM-2">4.95675</td>
<td class="gt_row gt_right" headers="KLK6">2.96120</td>
<td class="gt_row gt_right" headers="PDGF subunit A">4.39554</td>
<td class="gt_row gt_right" headers="TNF-R1">7.02890</td>
<td class="gt_row gt_right" headers="IGFBP-2">9.19431</td>
<td class="gt_row gt_right" headers="vWF">6.53802</td>
<td class="gt_row gt_right" headers="PECAM-1">3.51001</td>
<td class="gt_row gt_right" headers="MEPE">4.05974</td>
<td class="gt_row gt_right" headers="CCL16">6.28460</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="TNFRSF14">4.18683</td>
<td class="gt_row gt_right" headers="LDL receptor">3.07163</td>
<td class="gt_row gt_right" headers="ITGB2">4.07255</td>
<td class="gt_row gt_right" headers="IL-17RA">3.09242</td>
<td class="gt_row gt_right" headers="TNF-R2">4.92586</td>
<td class="gt_row gt_right" headers="MMP-9">6.51368</td>
<td class="gt_row gt_right" headers="EPHB4">4.79185</td>
<td class="gt_row gt_right" headers="IL2-RA">4.50928</td>
<td class="gt_row gt_right" headers="OPG">3.21778</td>
<td class="gt_row gt_right" headers="ALCAM">6.33962</td>
<td class="gt_row gt_right" headers="TFF3">6.20203</td>
<td class="gt_row gt_right" headers="SELP">8.61842</td>
<td class="gt_row gt_right" headers="CSTB">4.18718</td>
<td class="gt_row gt_right" headers="MCP-1">5.69525</td>
<td class="gt_row gt_right" headers="CD163">7.07982</td>
<td class="gt_row gt_right" headers="Gal-3">3.88953</td>
<td class="gt_row gt_right" headers="GRN">4.92804</td>
<td class="gt_row gt_right" headers="NT-proBNP">6.23507</td>
<td class="gt_row gt_right" headers="BLM hydrolase">2.43337</td>
<td class="gt_row gt_right" headers="PLC">7.62832</td>
<td class="gt_row gt_right" headers="LTBR">3.33873</td>
<td class="gt_row gt_right" headers="Notch 3">5.04138</td>
<td class="gt_row gt_right" headers="TIMP4">4.81471</td>
<td class="gt_row gt_right" headers="CNTN1">3.71833</td>
<td class="gt_row gt_right" headers="CDH5">2.75434</td>
<td class="gt_row gt_right" headers="TLT-2">3.80098</td>
<td class="gt_row gt_right" headers="FABP4">5.23224</td>
<td class="gt_row gt_right" headers="TFPI">8.55892</td>
<td class="gt_row gt_right" headers="PAI">6.25006</td>
<td class="gt_row gt_right" headers="CCL24">5.79869</td>
<td class="gt_row gt_right" headers="TR">5.47892</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.74976</td>
<td class="gt_row gt_right" headers="GDF-15">5.41217</td>
<td class="gt_row gt_right" headers="SELE">11.27452</td>
<td class="gt_row gt_right" headers="AZU1">7.45180</td>
<td class="gt_row gt_right" headers="DLK-1">4.99980</td>
<td class="gt_row gt_right" headers="SPON1">1.40045</td>
<td class="gt_row gt_right" headers="MPO">4.02239</td>
<td class="gt_row gt_right" headers="CXCL16">5.77493</td>
<td class="gt_row gt_right" headers="IL-6RA">10.61581</td>
<td class="gt_row gt_right" headers="RETN">6.55207</td>
<td class="gt_row gt_right" headers="IGFBP-1">4.76064</td>
<td class="gt_row gt_right" headers="CHIT1">5.03763</td>
<td class="gt_row gt_right" headers="TR-AP">3.55755</td>
<td class="gt_row gt_right" headers="GP6">1.29529</td>
<td class="gt_row gt_right" headers="PSP-D">2.48330</td>
<td class="gt_row gt_right" headers="PI3">1.74581</td>
<td class="gt_row gt_right" headers="Ep-CAM">5.86567</td>
<td class="gt_row gt_right" headers="AP-N">5.43826</td>
<td class="gt_row gt_right" headers="AXL">7.42888</td>
<td class="gt_row gt_right" headers="IL-1RT1">5.21229</td>
<td class="gt_row gt_right" headers="MMP-2">3.13342</td>
<td class="gt_row gt_right" headers="FAS">5.16669</td>
<td class="gt_row gt_right" headers="MB">6.12702</td>
<td class="gt_row gt_right" headers="TNFSF13B">7.32581</td>
<td class="gt_row gt_right" headers="PRTN3">6.82551</td>
<td class="gt_row gt_right" headers="PCSK9">2.27477</td>
<td class="gt_row gt_right" headers="U-PAR">5.27055</td>
<td class="gt_row gt_right" headers="OPN">7.56103</td>
<td class="gt_row gt_right" headers="CTSD">2.45063</td>
<td class="gt_row gt_right" headers="PGLYRP1">7.79729</td>
<td class="gt_row gt_right" headers="CPA1">6.11994</td>
<td class="gt_row gt_right" headers="JAM-A">4.71136</td>
<td class="gt_row gt_right" headers="Gal-4">4.06539</td>
<td class="gt_row gt_right" headers="IL-1RT2">5.02968</td>
<td class="gt_row gt_right" headers="SHPS-1">3.52999</td>
<td class="gt_row gt_right" headers="CCL15">7.92853</td>
<td class="gt_row gt_right" headers="CASP-3">3.83171</td>
<td class="gt_row gt_right" headers="uPA">4.23713</td>
<td class="gt_row gt_right" headers="CPB1">5.75794</td>
<td class="gt_row gt_right" headers="CHI3L1">6.32524</td>
<td class="gt_row gt_right" headers="ST2">7.38306</td>
<td class="gt_row gt_right" headers="t-PA">3.61521</td>
<td class="gt_row gt_right" headers="SCGB3A2">2.29058</td>
<td class="gt_row gt_right" headers="EGFR">1.83603</td>
<td class="gt_row gt_right" headers="IGFBP-7">8.72970</td>
<td class="gt_row gt_right" headers="CD93">10.26743</td>
<td class="gt_row gt_right" headers="IL-18BP">5.47330</td>
<td class="gt_row gt_right" headers="COL1A1">2.15861</td>
<td class="gt_row gt_right" headers="PON3">5.56441</td>
<td class="gt_row gt_right" headers="CTSZ">3.48881</td>
<td class="gt_row gt_right" headers="MMP-3">6.14958</td>
<td class="gt_row gt_right" headers="RARRES2">11.34378</td>
<td class="gt_row gt_right" headers="ICAM-2">4.78563</td>
<td class="gt_row gt_right" headers="KLK6">2.84551</td>
<td class="gt_row gt_right" headers="PDGF subunit A">4.20527</td>
<td class="gt_row gt_right" headers="TNF-R1">6.10766</td>
<td class="gt_row gt_right" headers="IGFBP-2">8.56939</td>
<td class="gt_row gt_right" headers="vWF">6.57856</td>
<td class="gt_row gt_right" headers="PECAM-1">3.49040</td>
<td class="gt_row gt_right" headers="MEPE">2.41851</td>
<td class="gt_row gt_right" headers="CCL16">8.94744</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="TNFRSF14">4.87168</td>
<td class="gt_row gt_right" headers="LDL receptor">2.99546</td>
<td class="gt_row gt_right" headers="ITGB2">4.16440</td>
<td class="gt_row gt_right" headers="IL-17RA">3.22660</td>
<td class="gt_row gt_right" headers="TNF-R2">5.40797</td>
<td class="gt_row gt_right" headers="MMP-9">6.02094</td>
<td class="gt_row gt_right" headers="EPHB4">4.85082</td>
<td class="gt_row gt_right" headers="IL2-RA">3.65696</td>
<td class="gt_row gt_right" headers="OPG">3.09417</td>
<td class="gt_row gt_right" headers="ALCAM">6.70734</td>
<td class="gt_row gt_right" headers="TFF3">5.97697</td>
<td class="gt_row gt_right" headers="SELP">8.03929</td>
<td class="gt_row gt_right" headers="CSTB">4.91757</td>
<td class="gt_row gt_right" headers="MCP-1">4.34817</td>
<td class="gt_row gt_right" headers="CD163">7.03458</td>
<td class="gt_row gt_right" headers="Gal-3">4.07475</td>
<td class="gt_row gt_right" headers="GRN">4.18723</td>
<td class="gt_row gt_right" headers="NT-proBNP">8.92449</td>
<td class="gt_row gt_right" headers="BLM hydrolase">2.25081</td>
<td class="gt_row gt_right" headers="PLC">7.82053</td>
<td class="gt_row gt_right" headers="LTBR">3.65792</td>
<td class="gt_row gt_right" headers="Notch 3">4.95558</td>
<td class="gt_row gt_right" headers="TIMP4">4.64706</td>
<td class="gt_row gt_right" headers="CNTN1">3.59840</td>
<td class="gt_row gt_right" headers="CDH5">2.98176</td>
<td class="gt_row gt_right" headers="TLT-2">4.91505</td>
<td class="gt_row gt_right" headers="FABP4">5.60347</td>
<td class="gt_row gt_right" headers="TFPI">7.55418</td>
<td class="gt_row gt_right" headers="PAI">7.08866</td>
<td class="gt_row gt_right" headers="CCL24">4.14482</td>
<td class="gt_row gt_right" headers="TR">5.03613</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.65257</td>
<td class="gt_row gt_right" headers="GDF-15">5.41055</td>
<td class="gt_row gt_right" headers="SELE">9.05306</td>
<td class="gt_row gt_right" headers="AZU1">6.25433</td>
<td class="gt_row gt_right" headers="DLK-1">4.85468</td>
<td class="gt_row gt_right" headers="SPON1">1.19305</td>
<td class="gt_row gt_right" headers="MPO">1.43446</td>
<td class="gt_row gt_right" headers="CXCL16">5.81279</td>
<td class="gt_row gt_right" headers="IL-6RA">10.76887</td>
<td class="gt_row gt_right" headers="RETN">7.59871</td>
<td class="gt_row gt_right" headers="IGFBP-1">4.09585</td>
<td class="gt_row gt_right" headers="CHIT1">4.79312</td>
<td class="gt_row gt_right" headers="TR-AP">2.74528</td>
<td class="gt_row gt_right" headers="GP6">2.47942</td>
<td class="gt_row gt_right" headers="PSP-D">3.85713</td>
<td class="gt_row gt_right" headers="PI3">2.98898</td>
<td class="gt_row gt_right" headers="Ep-CAM">3.66793</td>
<td class="gt_row gt_right" headers="AP-N">5.72536</td>
<td class="gt_row gt_right" headers="AXL">8.45526</td>
<td class="gt_row gt_right" headers="IL-1RT1">5.43662</td>
<td class="gt_row gt_right" headers="MMP-2">2.67488</td>
<td class="gt_row gt_right" headers="FAS">5.63450</td>
<td class="gt_row gt_right" headers="MB">6.71305</td>
<td class="gt_row gt_right" headers="TNFSF13B">6.47682</td>
<td class="gt_row gt_right" headers="PRTN3">5.73557</td>
<td class="gt_row gt_right" headers="PCSK9">1.88701</td>
<td class="gt_row gt_right" headers="U-PAR">5.42844</td>
<td class="gt_row gt_right" headers="OPN">8.17395</td>
<td class="gt_row gt_right" headers="CTSD">3.17106</td>
<td class="gt_row gt_right" headers="PGLYRP1">7.81271</td>
<td class="gt_row gt_right" headers="CPA1">3.77678</td>
<td class="gt_row gt_right" headers="JAM-A">5.12724</td>
<td class="gt_row gt_right" headers="Gal-4">3.61910</td>
<td class="gt_row gt_right" headers="IL-1RT2">5.41809</td>
<td class="gt_row gt_right" headers="SHPS-1">3.78599</td>
<td class="gt_row gt_right" headers="CCL15">7.38986</td>
<td class="gt_row gt_right" headers="CASP-3">4.99611</td>
<td class="gt_row gt_right" headers="uPA">4.58787</td>
<td class="gt_row gt_right" headers="CPB1">4.29205</td>
<td class="gt_row gt_right" headers="CHI3L1">4.98688</td>
<td class="gt_row gt_right" headers="ST2">6.57340</td>
<td class="gt_row gt_right" headers="t-PA">4.02072</td>
<td class="gt_row gt_right" headers="SCGB3A2">1.33801</td>
<td class="gt_row gt_right" headers="EGFR">1.84058</td>
<td class="gt_row gt_right" headers="IGFBP-7">9.07595</td>
<td class="gt_row gt_right" headers="CD93">10.60866</td>
<td class="gt_row gt_right" headers="IL-18BP">5.80099</td>
<td class="gt_row gt_right" headers="COL1A1">2.60031</td>
<td class="gt_row gt_right" headers="PON3">1.82350</td>
<td class="gt_row gt_right" headers="CTSZ">3.81345</td>
<td class="gt_row gt_right" headers="MMP-3">5.90877</td>
<td class="gt_row gt_right" headers="RARRES2">11.35561</td>
<td class="gt_row gt_right" headers="ICAM-2">5.18074</td>
<td class="gt_row gt_right" headers="KLK6">3.30156</td>
<td class="gt_row gt_right" headers="PDGF subunit A">4.18267</td>
<td class="gt_row gt_right" headers="TNF-R1">6.91411</td>
<td class="gt_row gt_right" headers="IGFBP-2">8.53832</td>
<td class="gt_row gt_right" headers="vWF">6.54869</td>
<td class="gt_row gt_right" headers="PECAM-1">3.59838</td>
<td class="gt_row gt_right" headers="MEPE">3.47907</td>
<td class="gt_row gt_right" headers="CCL16">6.38948</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="TNFRSF14">4.80082</td>
<td class="gt_row gt_right" headers="LDL receptor">4.46968</td>
<td class="gt_row gt_right" headers="ITGB2">4.26597</td>
<td class="gt_row gt_right" headers="IL-17RA">2.99842</td>
<td class="gt_row gt_right" headers="TNF-R2">5.25180</td>
<td class="gt_row gt_right" headers="MMP-9">5.86503</td>
<td class="gt_row gt_right" headers="EPHB4">4.69245</td>
<td class="gt_row gt_right" headers="IL2-RA">2.83051</td>
<td class="gt_row gt_right" headers="OPG">3.00020</td>
<td class="gt_row gt_right" headers="ALCAM">6.72305</td>
<td class="gt_row gt_right" headers="TFF3">6.11342</td>
<td class="gt_row gt_right" headers="SELP">8.68037</td>
<td class="gt_row gt_right" headers="CSTB">5.10206</td>
<td class="gt_row gt_right" headers="MCP-1">3.62979</td>
<td class="gt_row gt_right" headers="CD163">6.83571</td>
<td class="gt_row gt_right" headers="Gal-3">4.79233</td>
<td class="gt_row gt_right" headers="GRN">4.48840</td>
<td class="gt_row gt_right" headers="NT-proBNP">4.51528</td>
<td class="gt_row gt_right" headers="BLM hydrolase">1.49635</td>
<td class="gt_row gt_right" headers="PLC">7.86945</td>
<td class="gt_row gt_right" headers="LTBR">3.22385</td>
<td class="gt_row gt_right" headers="Notch 3">5.07763</td>
<td class="gt_row gt_right" headers="TIMP4">3.60749</td>
<td class="gt_row gt_right" headers="CNTN1">4.06516</td>
<td class="gt_row gt_right" headers="CDH5">2.47677</td>
<td class="gt_row gt_right" headers="TLT-2">4.51258</td>
<td class="gt_row gt_right" headers="FABP4">6.24876</td>
<td class="gt_row gt_right" headers="TFPI">8.44298</td>
<td class="gt_row gt_right" headers="PAI">7.34032</td>
<td class="gt_row gt_right" headers="CCL24">3.23951</td>
<td class="gt_row gt_right" headers="TR">5.43164</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.64321</td>
<td class="gt_row gt_right" headers="GDF-15">4.99756</td>
<td class="gt_row gt_right" headers="SELE">10.18247</td>
<td class="gt_row gt_right" headers="AZU1">4.29898</td>
<td class="gt_row gt_right" headers="DLK-1">4.85437</td>
<td class="gt_row gt_right" headers="SPON1">0.74468</td>
<td class="gt_row gt_right" headers="MPO">2.90799</td>
<td class="gt_row gt_right" headers="CXCL16">5.27218</td>
<td class="gt_row gt_right" headers="IL-6RA">10.79453</td>
<td class="gt_row gt_right" headers="RETN">6.19474</td>
<td class="gt_row gt_right" headers="IGFBP-1">4.05843</td>
<td class="gt_row gt_right" headers="CHIT1">2.96620</td>
<td class="gt_row gt_right" headers="TR-AP">3.62133</td>
<td class="gt_row gt_right" headers="GP6">1.71324</td>
<td class="gt_row gt_right" headers="PSP-D">3.05304</td>
<td class="gt_row gt_right" headers="PI3">2.39321</td>
<td class="gt_row gt_right" headers="Ep-CAM">3.19271</td>
<td class="gt_row gt_right" headers="AP-N">5.06954</td>
<td class="gt_row gt_right" headers="AXL">7.18605</td>
<td class="gt_row gt_right" headers="IL-1RT1">4.73092</td>
<td class="gt_row gt_right" headers="MMP-2">2.23797</td>
<td class="gt_row gt_right" headers="FAS">5.06875</td>
<td class="gt_row gt_right" headers="MB">7.85047</td>
<td class="gt_row gt_right" headers="TNFSF13B">6.57038</td>
<td class="gt_row gt_right" headers="PRTN3">3.94726</td>
<td class="gt_row gt_right" headers="PCSK9">2.00853</td>
<td class="gt_row gt_right" headers="U-PAR">5.01430</td>
<td class="gt_row gt_right" headers="OPN">8.06772</td>
<td class="gt_row gt_right" headers="CTSD">2.78930</td>
<td class="gt_row gt_right" headers="PGLYRP1">7.27746</td>
<td class="gt_row gt_right" headers="CPA1">4.25616</td>
<td class="gt_row gt_right" headers="JAM-A">4.66746</td>
<td class="gt_row gt_right" headers="Gal-4">3.60540</td>
<td class="gt_row gt_right" headers="IL-1RT2">4.16736</td>
<td class="gt_row gt_right" headers="SHPS-1">2.73188</td>
<td class="gt_row gt_right" headers="CCL15">6.78463</td>
<td class="gt_row gt_right" headers="CASP-3">4.67830</td>
<td class="gt_row gt_right" headers="uPA">3.78583</td>
<td class="gt_row gt_right" headers="CPB1">4.35087</td>
<td class="gt_row gt_right" headers="CHI3L1">4.86657</td>
<td class="gt_row gt_right" headers="ST2">5.60744</td>
<td class="gt_row gt_right" headers="t-PA">5.52495</td>
<td class="gt_row gt_right" headers="SCGB3A2">2.04703</td>
<td class="gt_row gt_right" headers="EGFR">2.08128</td>
<td class="gt_row gt_right" headers="IGFBP-7">7.91713</td>
<td class="gt_row gt_right" headers="CD93">10.06987</td>
<td class="gt_row gt_right" headers="IL-18BP">5.59126</td>
<td class="gt_row gt_right" headers="COL1A1">2.14850</td>
<td class="gt_row gt_right" headers="PON3">4.67881</td>
<td class="gt_row gt_right" headers="CTSZ">3.38853</td>
<td class="gt_row gt_right" headers="MMP-3">5.60527</td>
<td class="gt_row gt_right" headers="RARRES2">11.07459</td>
<td class="gt_row gt_right" headers="ICAM-2">5.28632</td>
<td class="gt_row gt_right" headers="KLK6">2.65096</td>
<td class="gt_row gt_right" headers="PDGF subunit A">4.53751</td>
<td class="gt_row gt_right" headers="TNF-R1">6.53179</td>
<td class="gt_row gt_right" headers="IGFBP-2">7.08070</td>
<td class="gt_row gt_right" headers="vWF">6.31329</td>
<td class="gt_row gt_right" headers="PECAM-1">3.53453</td>
<td class="gt_row gt_right" headers="MEPE">3.19367</td>
<td class="gt_row gt_right" headers="CCL16">5.78551</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="TNFRSF14">4.31973</td>
<td class="gt_row gt_right" headers="LDL receptor">4.37623</td>
<td class="gt_row gt_right" headers="ITGB2">4.16434</td>
<td class="gt_row gt_right" headers="IL-17RA">4.23795</td>
<td class="gt_row gt_right" headers="TNF-R2">5.08520</td>
<td class="gt_row gt_right" headers="MMP-9">5.25349</td>
<td class="gt_row gt_right" headers="EPHB4">4.63230</td>
<td class="gt_row gt_right" headers="IL2-RA">3.00558</td>
<td class="gt_row gt_right" headers="OPG">3.17529</td>
<td class="gt_row gt_right" headers="ALCAM">6.75875</td>
<td class="gt_row gt_right" headers="TFF3">6.51092</td>
<td class="gt_row gt_right" headers="SELP">8.94958</td>
<td class="gt_row gt_right" headers="CSTB">5.40882</td>
<td class="gt_row gt_right" headers="MCP-1">4.04586</td>
<td class="gt_row gt_right" headers="CD163">7.65342</td>
<td class="gt_row gt_right" headers="Gal-3">4.83942</td>
<td class="gt_row gt_right" headers="GRN">4.52515</td>
<td class="gt_row gt_right" headers="NT-proBNP">6.54076</td>
<td class="gt_row gt_right" headers="BLM hydrolase">2.97148</td>
<td class="gt_row gt_right" headers="PLC">7.76675</td>
<td class="gt_row gt_right" headers="LTBR">3.20099</td>
<td class="gt_row gt_right" headers="Notch 3">4.68663</td>
<td class="gt_row gt_right" headers="TIMP4">3.69843</td>
<td class="gt_row gt_right" headers="CNTN1">3.24286</td>
<td class="gt_row gt_right" headers="CDH5">2.48926</td>
<td class="gt_row gt_right" headers="TLT-2">3.80521</td>
<td class="gt_row gt_right" headers="FABP4">5.66444</td>
<td class="gt_row gt_right" headers="TFPI">8.23639</td>
<td class="gt_row gt_right" headers="PAI">6.48794</td>
<td class="gt_row gt_right" headers="CCL24">5.22736</td>
<td class="gt_row gt_right" headers="TR">6.34953</td>
<td class="gt_row gt_right" headers="TNFRSF10C">4.45629</td>
<td class="gt_row gt_right" headers="GDF-15">6.16002</td>
<td class="gt_row gt_right" headers="SELE">10.07301</td>
<td class="gt_row gt_right" headers="AZU1">3.75971</td>
<td class="gt_row gt_right" headers="DLK-1">5.16906</td>
<td class="gt_row gt_right" headers="SPON1">0.76820</td>
<td class="gt_row gt_right" headers="MPO">1.89575</td>
<td class="gt_row gt_right" headers="CXCL16">5.86178</td>
<td class="gt_row gt_right" headers="IL-6RA">10.58538</td>
<td class="gt_row gt_right" headers="RETN">6.37949</td>
<td class="gt_row gt_right" headers="IGFBP-1">4.77158</td>
<td class="gt_row gt_right" headers="CHIT1">5.36963</td>
<td class="gt_row gt_right" headers="TR-AP">3.49137</td>
<td class="gt_row gt_right" headers="GP6">1.71631</td>
<td class="gt_row gt_right" headers="PSP-D">3.42800</td>
<td class="gt_row gt_right" headers="PI3">2.47362</td>
<td class="gt_row gt_right" headers="Ep-CAM">4.85831</td>
<td class="gt_row gt_right" headers="AP-N">4.90928</td>
<td class="gt_row gt_right" headers="AXL">7.99517</td>
<td class="gt_row gt_right" headers="IL-1RT1">5.51444</td>
<td class="gt_row gt_right" headers="MMP-2">2.58369</td>
<td class="gt_row gt_right" headers="FAS">5.31680</td>
<td class="gt_row gt_right" headers="MB">6.91246</td>
<td class="gt_row gt_right" headers="TNFSF13B">6.45898</td>
<td class="gt_row gt_right" headers="PRTN3">4.38051</td>
<td class="gt_row gt_right" headers="PCSK9">1.92950</td>
<td class="gt_row gt_right" headers="U-PAR">5.30131</td>
<td class="gt_row gt_right" headers="OPN">7.47483</td>
<td class="gt_row gt_right" headers="CTSD">2.84863</td>
<td class="gt_row gt_right" headers="PGLYRP1">6.99019</td>
<td class="gt_row gt_right" headers="CPA1">5.54611</td>
<td class="gt_row gt_right" headers="JAM-A">4.82693</td>
<td class="gt_row gt_right" headers="Gal-4">4.07646</td>
<td class="gt_row gt_right" headers="IL-1RT2">5.69839</td>
<td class="gt_row gt_right" headers="SHPS-1">3.36521</td>
<td class="gt_row gt_right" headers="CCL15">7.96000</td>
<td class="gt_row gt_right" headers="CASP-3">4.35064</td>
<td class="gt_row gt_right" headers="uPA">4.00462</td>
<td class="gt_row gt_right" headers="CPB1">5.32546</td>
<td class="gt_row gt_right" headers="CHI3L1">4.55477</td>
<td class="gt_row gt_right" headers="ST2">7.42739</td>
<td class="gt_row gt_right" headers="t-PA">4.46509</td>
<td class="gt_row gt_right" headers="SCGB3A2">2.17803</td>
<td class="gt_row gt_right" headers="EGFR">2.18225</td>
<td class="gt_row gt_right" headers="IGFBP-7">8.80074</td>
<td class="gt_row gt_right" headers="CD93">10.32481</td>
<td class="gt_row gt_right" headers="IL-18BP">5.65342</td>
<td class="gt_row gt_right" headers="COL1A1">1.99290</td>
<td class="gt_row gt_right" headers="PON3">4.50998</td>
<td class="gt_row gt_right" headers="CTSZ">3.88637</td>
<td class="gt_row gt_right" headers="MMP-3">8.29931</td>
<td class="gt_row gt_right" headers="RARRES2">11.38195</td>
<td class="gt_row gt_right" headers="ICAM-2">4.80253</td>
<td class="gt_row gt_right" headers="KLK6">4.23683</td>
<td class="gt_row gt_right" headers="PDGF subunit A">3.49872</td>
<td class="gt_row gt_right" headers="TNF-R1">6.39474</td>
<td class="gt_row gt_right" headers="IGFBP-2">7.91963</td>
<td class="gt_row gt_right" headers="vWF">6.54171</td>
<td class="gt_row gt_right" headers="PECAM-1">3.38378</td>
<td class="gt_row gt_right" headers="MEPE">3.52397</td>
<td class="gt_row gt_right" headers="CCL16">7.64941</td>
</tr>
</tbody>
</table>

</div>
<p>Protein expressions for a subset of individuals.</p>
</div>
</div>
</section>
<section id="figure-1" class="level2">
<h2 class="anchored" data-anchor-id="figure-1">Figure 1</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>protlist <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ALCAM"</span>,<span class="st">"Gal-4"</span>,<span class="st">"PECAM-1"</span>,<span class="st">"GP6"</span>,<span class="st">"ST2"</span>)</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>scmat <span class="ot">&lt;-</span> <span class="fu">round</span>(<span class="fu">cor</span>(rprotblds[,<span class="fu">which</span>(<span class="fu">colnames</span>(rprotblds) <span class="sc">%in%</span> protlist)],</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">method=</span><span class="st">"spearman"</span>,<span class="at">use=</span><span class="st">"pairwise.complete.obs"</span>),<span class="dv">2</span>)</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>scmat[<span class="fu">lower.tri</span>(scmat)] <span class="ot">&lt;-</span> <span class="cn">NA</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="co">#rownames(scmat) &lt;- colnames(scmat) &lt;- protlist</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>scmat_melt <span class="ot">&lt;-</span> <span class="fu">melt</span>(scmat, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a><span class="co">#compute p-values</span></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(scmat_melt)){</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>    b1 <span class="ot">&lt;-</span> <span class="fu">as.character</span>(scmat_melt<span class="sc">$</span>Var1[i])</span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>    b2 <span class="ot">&lt;-</span> <span class="fu">as.character</span>(scmat_melt<span class="sc">$</span>Var2[i])</span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>    scmat_melt<span class="sc">$</span>pval[i] <span class="ot">&lt;-</span> <span class="fu">cor.test</span>(rprotblds[,<span class="fu">which</span>(<span class="fu">colnames</span>(rprotblds) <span class="sc">%in%</span> b1)],</span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>                                   rprotblds[,<span class="fu">which</span>(<span class="fu">colnames</span>(rprotblds) <span class="sc">%in%</span> b2)],</span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>                    <span class="at">method=</span><span class="st">"spearman"</span>,<span class="at">use=</span><span class="st">"pairwise.complete.obs"</span>)<span class="sc">$</span>p.value</span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>scmat_melt<span class="sc">$</span>sig <span class="ot">=</span> scmat_melt<span class="sc">$</span>pval<span class="sc">&lt;</span><span class="fl">0.01</span></span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a>scmat_melt<span class="sc">$</span>text <span class="ot">=</span> <span class="fu">ifelse</span>(scmat_melt<span class="sc">$</span>sig<span class="sc">==</span><span class="cn">TRUE</span> <span class="sc">&amp;</span> scmat_melt<span class="sc">$</span>value <span class="sc">!=</span> <span class="dv">1</span>,<span class="fu">paste0</span>(scmat_melt<span class="sc">$</span>value,<span class="st">"*"</span>),scmat_melt<span class="sc">$</span>value)</span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a>ggheatmap <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(scmat_melt, <span class="fu">aes</span>(Var2, Var1, <span class="at">fill =</span> value))<span class="sc">+</span></span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="at">color =</span> <span class="st">"white"</span>)<span class="sc">+</span></span>
<span id="cb6-21"><a href="#cb6-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_gradient2</span>(<span class="at">low =</span> <span class="st">"blue"</span>, <span class="at">high =</span> <span class="st">"red"</span>, <span class="at">mid =</span> <span class="st">"white"</span>, </span>
<span id="cb6-22"><a href="#cb6-22" aria-hidden="true" tabindex="-1"></a>                       <span class="at">midpoint =</span> <span class="dv">0</span>, <span class="at">limit =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>,<span class="dv">1</span>), <span class="at">space =</span> <span class="st">"Lab"</span>, </span>
<span id="cb6-23"><a href="#cb6-23" aria-hidden="true" tabindex="-1"></a>                       <span class="at">name=</span><span class="st">"Spearman</span><span class="sc">\n</span><span class="st">Correlation"</span>) <span class="sc">+</span></span>
<span id="cb6-24"><a href="#cb6-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span> <span class="co"># minimal theme</span></span>
<span id="cb6-25"><a href="#cb6-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">vjust =</span> <span class="dv">1</span>, </span>
<span id="cb6-26"><a href="#cb6-26" aria-hidden="true" tabindex="-1"></a>                                   <span class="at">size =</span> <span class="dv">12</span>, <span class="at">hjust =</span> <span class="dv">1</span>))<span class="sc">+</span></span>
<span id="cb6-27"><a href="#cb6-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_fixed</span>()<span class="sc">+</span></span>
<span id="cb6-28"><a href="#cb6-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(Var2, Var1, <span class="at">label =</span> text), <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">4</span>) <span class="sc">+</span></span>
<span id="cb6-29"><a href="#cb6-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb6-30"><a href="#cb6-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-31"><a href="#cb6-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-32"><a href="#cb6-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.major =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-33"><a href="#cb6-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.border =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-34"><a href="#cb6-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.background =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-35"><a href="#cb6-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.ticks =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb6-36"><a href="#cb6-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.justification =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">0</span>),</span>
<span id="cb6-37"><a href="#cb6-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="fu">c</span>(<span class="fl">0.5</span>, <span class="fl">0.7</span>),</span>
<span id="cb6-38"><a href="#cb6-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.direction =</span> <span class="st">"horizontal"</span>) <span class="sc">+</span></span>
<span id="cb6-39"><a href="#cb6-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_colorbar</span>(<span class="at">barwidth =</span> <span class="dv">7</span>, <span class="at">barheight =</span> <span class="dv">1</span>,</span>
<span id="cb6-40"><a href="#cb6-40" aria-hidden="true" tabindex="-1"></a>                               <span class="at">title.position =</span> <span class="st">"top"</span>, <span class="at">title.hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb6-41"><a href="#cb6-41" aria-hidden="true" tabindex="-1"></a><span class="co"># Print the heatmap</span></span>
<span id="cb6-42"><a href="#cb6-42" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(ggheatmap)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/figure-1-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="protein-emm-selection" class="level1">
<h1>Protein EMM Selection</h1>
<section id="individual-regressions" class="level2">
<h2 class="anchored" data-anchor-id="individual-regressions">Individual Regressions</h2>
<p>We fit a linear regression model for each protein, including an interaction term between the protein and the intervention group (<code>treatment</code>). Volcano plots are used to visualize the significance and magnitude of these interaction effects.</p>
<section id="sppb" class="level3">
<h3 class="anchored" data-anchor-id="sppb">SPPB</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>intxn_df <span class="ot">=</span> <span class="fu">matrix</span>(<span class="cn">NA</span>,<span class="at">nrow=</span><span class="dv">92</span>,<span class="at">ncol=</span><span class="dv">2</span>)</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(intxn_df) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"effect"</span>,<span class="st">"p-value"</span>)</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>){</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  prot<span class="ot">=</span><span class="fu">names</span>(rprotblds)[i]</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>  formtmp<span class="ot">=</span><span class="fu">paste</span>(<span class="st">"fu_sppb ~ bl_sppb + age + sex + race___4 + hf_cat + `"</span>,prot,</span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>                <span class="st">"`*intervention_1_control_0"</span>,<span class="at">sep=</span><span class="st">""</span>)</span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>  tmpmod<span class="ot">=</span><span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data=</span>rprotblds,<span class="at">formula=</span>formtmp))</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>  intxn_df[i<span class="dv">-2</span>,]<span class="ot">=</span>tmpmod<span class="sc">$</span>coefficients[<span class="fu">nrow</span>(tmpmod<span class="sc">$</span>coefficients),<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">4</span>)]</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>intxn_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">Protein=</span><span class="fu">names</span>(rprotblds)[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>],intxn_df,</span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>                      <span class="at">Significance=</span><span class="fu">ifelse</span>(intxn_df[,<span class="dv">2</span>]<span class="sc">&lt;</span><span class="fl">0.05</span>,<span class="st">"Significant"</span>,<span class="st">"Not Significant"</span>))</span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">p.adjust</span>(intxn_df<span class="sc">$</span>p.value,<span class="at">method=</span><span class="st">'fdr'</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
 0.4430  0.6299  0.8561  0.7804  0.9143  0.9926 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co">#none are significant after FDR adjustment</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="mwd" class="level3">
<h3 class="anchored" data-anchor-id="mwd">6MWD</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co"># --- 6MWD Analysis ---</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>intxn_df <span class="ot">=</span> <span class="fu">matrix</span>(<span class="cn">NA</span>,<span class="at">nrow=</span><span class="dv">92</span>,<span class="at">ncol=</span><span class="dv">2</span>)</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(intxn_df) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"effect"</span>,<span class="st">"p-value"</span>)</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>){</span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  prot<span class="ot">=</span><span class="fu">names</span>(rprotblds)[i]</span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  formtmp<span class="ot">=</span><span class="fu">paste</span>(<span class="st">"fu_smw ~ bl_smw + age + sex + race___4 + hf_cat + `"</span>,prot,</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>                <span class="st">"`*intervention_1_control_0"</span>,<span class="at">sep=</span><span class="st">""</span>)</span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>  tmpmod<span class="ot">=</span><span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data=</span>rprotblds,<span class="at">formula=</span>formtmp))</span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>  intxn_df[i<span class="dv">-2</span>,]<span class="ot">=</span>tmpmod<span class="sc">$</span>coefficients[<span class="fu">nrow</span>(tmpmod<span class="sc">$</span>coefficients),<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">4</span>)]</span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>intxn_df <span class="ot">=</span> <span class="fu">data.frame</span>(<span class="at">Protein=</span><span class="fu">names</span>(rprotblds)[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>],intxn_df,</span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>                      <span class="at">Significance=</span><span class="fu">ifelse</span>(intxn_df[,<span class="dv">2</span>]<span class="sc">&lt;</span><span class="fl">0.05</span>,<span class="st">"Significant"</span>,<span class="st">"Not Significant"</span>))</span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(<span class="fu">p.adjust</span>(intxn_df<span class="sc">$</span>p.value,<span class="at">method=</span><span class="st">'fdr'</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
 0.2583  0.3393  0.6864  0.6255  0.8417  0.9675 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co">#none are significant after FDR adjustment</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="lasso-with-strong-hierarchy-for-learning-interactions" class="level2">
<h2 class="anchored" data-anchor-id="lasso-with-strong-hierarchy-for-learning-interactions">LASSO with Strong Hierarchy for Learning Interactions</h2>
<section id="sppb-1" class="level3">
<h3 class="anchored" data-anchor-id="sppb-1">SPPB</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>sppb_data <span class="ot">&lt;-</span> <span class="fu">cbind</span>(</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>  rprotblds[, <span class="fu">c</span>(<span class="st">"fu_sppb"</span>, <span class="st">"intervention_1_control_0"</span>, <span class="st">"bl_sppb"</span>, <span class="st">"age"</span>, <span class="st">"sex"</span>, <span class="st">"race___4"</span>, <span class="st">"hf_cat"</span>)],</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a>  rprotblds[, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>]</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a>sppb_data <span class="ot">&lt;-</span> <span class="fu">na.omit</span>(sppb_data)</span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">97</span>)</span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>sppb_y <span class="ot">&lt;-</span> sppb_data<span class="sc">$</span>fu_sppb</span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Format for glinternet (factors must be 0/1)</span></span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a>sppb_x <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">intervention_1_control_0 =</span> <span class="fu">as.numeric</span>(sppb_data<span class="sc">$</span>intervention_1_control_0) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a>  sppb_data<span class="sc">$</span>bl_sppb,</span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a>  sppb_data<span class="sc">$</span>age,</span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">sex =</span> <span class="fu">as.numeric</span>(sppb_data<span class="sc">$</span>sex) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">race___4 =</span> <span class="fu">as.numeric</span>(sppb_data<span class="sc">$</span>race___4) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">hf_cat =</span> <span class="fu">as.numeric</span>(sppb_data<span class="sc">$</span>hf_cat) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a>  sppb_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>]</span>
<span id="cb13-20"><a href="#cb13-20" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb13-21"><a href="#cb13-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-22"><a href="#cb13-22" aria-hidden="true" tabindex="-1"></a><span class="co"># Specify number of levels (1 = continuous, 2 = binary)</span></span>
<span id="cb13-23"><a href="#cb13-23" aria-hidden="true" tabindex="-1"></a>numLevels <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">2</span>, <span class="dv">2</span>, <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">92</span>))</span>
<span id="cb13-24"><a href="#cb13-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-25"><a href="#cb13-25" aria-hidden="true" tabindex="-1"></a><span class="co"># Cross-validation to select lambda</span></span>
<span id="cb13-26"><a href="#cb13-26" aria-hidden="true" tabindex="-1"></a>fit.cv <span class="ot">&lt;-</span> <span class="fu">glinternet.cv</span>(sppb_x, sppb_y, <span class="at">numLevels =</span> numLevels, <span class="at">nFolds =</span> <span class="dv">10</span>, <span class="at">nLambda =</span> <span class="dv">10</span>)</span>
<span id="cb13-27"><a href="#cb13-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-28"><a href="#cb13-28" aria-hidden="true" tabindex="-1"></a>fit <span class="ot">&lt;-</span> <span class="fu">glinternet</span>(sppb_x, sppb_y, <span class="at">numLevels =</span> numLevels, <span class="at">lambda =</span> fit.cv<span class="sc">$</span>lambdaHat)</span>
<span id="cb13-29"><a href="#cb13-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-30"><a href="#cb13-30" aria-hidden="true" tabindex="-1"></a>coeffs <span class="ot">&lt;-</span> <span class="fu">coef</span>(fit)[[<span class="dv">2</span>]]</span>
<span id="cb13-31"><a href="#cb13-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-32"><a href="#cb13-32" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">SPPB - Proteins with treatment interaction:</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
SPPB - Proteins with treatment interaction:</code></pre>
</div>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">names</span>(sppb_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])[coeffs<span class="sc">$</span>interactions<span class="sc">$</span>catcont[, <span class="dv">2</span>] <span class="sc">-</span> <span class="dv">2</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] "GP6"</code></pre>
</div>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">SPPB - Main effects:</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
SPPB - Main effects:</code></pre>
</div>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">names</span>(sppb_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])[coeffs<span class="sc">$</span>mainEffects<span class="sc">$</span>cont[<span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="sc">-</span><span class="dv">2</span>)] <span class="sc">-</span> <span class="dv">2</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] "TNFSF13B"       "SHPS-1"         "ST2"            "PI3"           
 [5] "IGFBP-1"        "CTSD"           "IL2-RA"         "ALCAM"         
 [9] "PDGF subunit A" "TFF3"           "KLK6"           "SELP"          
[13] "CDH5"           "MCP-1"          "MPO"            "BLM hydrolase" 
[17] "CPA1"           "TLT-2"          "CHIT1"          "FABP4"         
[21] "CCL24"          "PRTN3"          "SPON1"          "CCL15"         
[25] "ICAM-2"         "CXCL16"         "PGLYRP1"        "CTSZ"          
[29] "Ep-CAM"         "SCGB3A2"        "IGFBP-7"        "GP6"           </code></pre>
</div>
</div>
</section>
<section id="mwd-1" class="level3">
<h3 class="anchored" data-anchor-id="mwd-1">6MWD</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>smw_data <span class="ot">&lt;-</span> <span class="fu">cbind</span>(</span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  rprotblds[, <span class="fu">c</span>(<span class="st">"fu_smw"</span>, <span class="st">"intervention_1_control_0"</span>, <span class="st">"bl_smw"</span>, <span class="st">"age"</span>, <span class="st">"sex"</span>, <span class="st">"race___4"</span>, <span class="st">"hf_cat"</span>)],</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>  rprotblds[, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>]</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>smw_data <span class="ot">&lt;-</span> <span class="fu">na.omit</span>(smw_data)</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">567</span>)</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>smw_y <span class="ot">&lt;-</span> smw_data<span class="sc">$</span>fu_smw</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>smw_x <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">intervention_1_control_0 =</span> <span class="fu">as.numeric</span>(smw_data<span class="sc">$</span>intervention_1_control_0) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>  smw_data<span class="sc">$</span>bl_smw,</span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>  smw_data<span class="sc">$</span>age,</span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">sex =</span> <span class="fu">as.numeric</span>(smw_data<span class="sc">$</span>sex) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">race___4 =</span> <span class="fu">as.numeric</span>(smw_data<span class="sc">$</span>race___4) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">hf_cat =</span> <span class="fu">as.numeric</span>(smw_data<span class="sc">$</span>hf_cat) <span class="sc">-</span> <span class="dv">1</span>,</span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>  smw_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>]</span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>numLevels <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">2</span>, <span class="dv">2</span>, <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">92</span>))</span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a>fit.cv <span class="ot">&lt;-</span> <span class="fu">glinternet.cv</span>(smw_x, smw_y, <span class="at">numLevels =</span> numLevels, <span class="at">nFolds =</span> <span class="dv">10</span>, <span class="at">nLambda =</span> <span class="dv">10</span>)</span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>fit <span class="ot">&lt;-</span> <span class="fu">glinternet</span>(smw_x, smw_y, <span class="at">numLevels =</span> numLevels, <span class="at">lambda =</span> fit.cv<span class="sc">$</span>lambdaHat)</span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>coeffs <span class="ot">&lt;-</span> <span class="fu">coef</span>(fit)[[<span class="dv">2</span>]]</span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-29"><a href="#cb21-29" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">6MWD - Proteins with treatment interaction:</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
6MWD - Proteins with treatment interaction:</code></pre>
</div>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">names</span>(smw_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])[coeffs<span class="sc">$</span>interactions<span class="sc">$</span>catcont[, <span class="dv">2</span>] <span class="sc">-</span> <span class="dv">2</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] "ALCAM" "Gal-4"</code></pre>
</div>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">6MWD - Main effects:</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
6MWD - Main effects:</code></pre>
</div>
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">names</span>(smw_data[, <span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])[coeffs<span class="sc">$</span>mainEffects<span class="sc">$</span>cont[<span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="sc">-</span><span class="dv">2</span>)] <span class="sc">-</span> <span class="dv">2</span>])</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] "CXCL16"         "TNFSF13B"       "PRTN3"          "ITGB2"         
 [5] "SELE"           "LDL receptor"   "MMP-9"          "IL2-RA"        
 [9] "CHIT1"          "PDGF subunit A" "IL-17RA"        "PSP-D"         
[13] "uPA"            "SELP"           "IL-1RT1"        "CPA1"          
[17] "Notch 3"        "ICAM-2"         "CNTN1"          "MEPE"          
[21] "CDH5"           "SCGB3A2"        "TLT-2"          "CTSD"          
[25] "PAI"            "ST2"            "GDF-15"         "CCL16"         
[29] "DLK-1"          "CPB1"           "SPON1"          "IL-6RA"        
[33] "MMP-2"          "RETN"           "IGFBP-1"        "PCSK9"         
[37] "GP6"            "PI3"            "EGFR"           "t-PA"          
[41] "ALCAM"          "Gal-4"         </code></pre>
</div>
</div>
</section>
</section>
<section id="nonlinear-interactions-with-bart" class="level2">
<h2 class="anchored" data-anchor-id="nonlinear-interactions-with-bart">Nonlinear interactions with BART</h2>
<section id="sppb-2" class="level3">
<h3 class="anchored" data-anchor-id="sppb-2">SPPB</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">45</span>)</span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">java.parameters =</span> <span class="st">"-Xmx4000m"</span>)</span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a><span class="fu">set_bart_machine_num_cores</span>(<span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine now using 1 cores.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co">#model fit</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>sppb_data <span class="ot">&lt;-</span> <span class="fu">cbind</span>(rprotblds[,<span class="fu">c</span>(<span class="st">"fu_sppb"</span>,<span class="st">"intervention_1_control_0"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,<span class="st">"hf_cat"</span>)],</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>                   rprotblds[,<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>])</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a><span class="co">#remove incomplete cases (any rows with missingness, mostly due to lack of follow up)</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>sppb_data <span class="ot">=</span> <span class="fu">na.omit</span>(sppb_data);</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>sppb_y <span class="ot">&lt;-</span> sppb_data<span class="sc">$</span>fu_sppb</span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a>sppb_x <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(sppb_data<span class="sc">$</span>intervention_1_control_0,</span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>                     sppb_data<span class="sc">$</span>bl_sppb,sppb_data<span class="sc">$</span>age,</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a>                     sppb_data<span class="sc">$</span>sex,</span>
<span id="cb31-12"><a href="#cb31-12" aria-hidden="true" tabindex="-1"></a>                     sppb_data<span class="sc">$</span>race___4,</span>
<span id="cb31-13"><a href="#cb31-13" aria-hidden="true" tabindex="-1"></a>                     sppb_data<span class="sc">$</span>hf_cat,</span>
<span id="cb31-14"><a href="#cb31-14" aria-hidden="true" tabindex="-1"></a>                     sppb_data[,<span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])</span>
<span id="cb31-15"><a href="#cb31-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-16"><a href="#cb31-16" aria-hidden="true" tabindex="-1"></a>bart_machine_sppb <span class="ot">&lt;-</span> <span class="fu">bartMachine</span>(sppb_x, sppb_y,</span>
<span id="cb31-17"><a href="#cb31-17" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">num_trees=</span><span class="dv">100</span>,</span>
<span id="cb31-18"><a href="#cb31-18" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">num_burn_in=</span><span class="dv">2000</span>,</span>
<span id="cb31-19"><a href="#cb31-19" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">num_iterations_after_burn_in=</span><span class="dv">2000</span>,</span>
<span id="cb31-20"><a href="#cb31-20" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">use_missing_data=</span><span class="cn">TRUE</span>,</span>
<span id="cb31-21"><a href="#cb31-21" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">mem_cache_for_speed =</span> <span class="cn">FALSE</span>,</span>
<span id="cb31-22"><a href="#cb31-22" aria-hidden="true" tabindex="-1"></a>                                 <span class="at">seed=</span><span class="dv">34</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine initializing with 100 trees...
bartMachine vars checked...
bartMachine java init...
bartMachine factors created...
bartMachine before preprocess...
bartMachine after preprocess... 102 total features...
bartMachine sigsq estimated...
bartMachine training data finalized...
Now building bartMachine for regression...Covariate importance prior ON. Missing data feature ON. 
evaluating in sample data...done</code></pre>
</div>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>bart_machine_sppb</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine v1.3.4.1 for regression

Missing data feature ON
training data size: n = 202 and p = 102 
built in 7 secs on 1 core, 100 trees, 2000 burn-in and 2000 post. samples

sigsq est for y beforehand: 2.703 
avg sigsq estimate after burn-in: 4.76879 

in-sample statistics:
 L1 = 283.32 
 L2 = 650.11 
 rmse = 1.79 
 Pseudo-Rsq = 0.707
p-val for shapiro-wilk test of normality of residuals: 0 
p-val for zero-mean noise: 0.98309 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_convergence_diagnostics</span>(bart_machine_sppb)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a>reps<span class="ot">=</span><span class="dv">100</span></span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>intxninvest_sppb<span class="ot">=</span><span class="fu">interaction_investigator</span>(bart_machine_sppb, </span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">plot=</span><span class="cn">FALSE</span>,</span>
<span id="cb36-4"><a href="#cb36-4" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">num_var_plot=</span><span class="dv">25</span>,</span>
<span id="cb36-5"><a href="#cb36-5" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">bottom_margin=</span><span class="dv">20</span>,</span>
<span id="cb36-6"><a href="#cb36-6" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">num_replicates_for_avg=</span>reps,</span>
<span id="cb36-7"><a href="#cb36-7" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">num_trees_bottleneck=</span><span class="dv">20</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>........................................
........................................
....................</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="co">#average count of interactions between protein and treatment/control</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a><span class="co">#sort(rowSums(intxninvest_sppb$interaction_counts_avg[3:94,95:96]),decreasing=TRUE)</span></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a><span class="co">#mean and sd of interaction counts</span></span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a><span class="co">#cbind(intxninvest_sppb$interaction_counts_avg[3:94,95:96],</span></span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a><span class="co">#      1.96*intxninvest_sppb$interaction_counts_sd[3:94,95:96]/sqrt(reps))</span></span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a><span class="co">#sum of count of interactions</span></span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a>bartintxn_means<span class="ot">=</span><span class="fu">apply</span>(intxninvest_sppb<span class="sc">$</span>interaction_counts[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>,<span class="dv">95</span><span class="sc">:</span><span class="dv">96</span>,],<span class="dv">1</span>,sum)<span class="sc">/</span>reps</span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>bartintxn_sds<span class="ot">=</span><span class="fu">apply</span>(<span class="fu">apply</span>(intxninvest_sppb<span class="sc">$</span>interaction_counts[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>,<span class="dv">95</span><span class="sc">:</span><span class="dv">96</span>,],<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>),sum),<span class="dv">1</span>,sd)</span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>bartintxn_moe<span class="ot">=</span><span class="fl">1.96</span> <span class="sc">*</span> bartintxn_sds <span class="sc">/</span> <span class="fu">sqrt</span>(reps)</span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>bartintxn_df<span class="ot">=</span><span class="fu">data.frame</span>(<span class="at">Protein=</span><span class="fu">rownames</span>(intxninvest_sppb<span class="sc">$</span>interaction_counts_avg)[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>],</span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a>                        <span class="at">Importance=</span>bartintxn_means,<span class="at">MOE=</span>bartintxn_moe)</span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a>bartintxn_df <span class="ot">=</span> bartintxn_df[<span class="fu">order</span>(bartintxn_df<span class="sc">$</span>Importance,<span class="at">decreasing=</span><span class="cn">TRUE</span>),]</span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a><span class="co">#top 20 only</span></span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a>bartintxn_df<span class="ot">=</span>bartintxn_df[<span class="dv">1</span><span class="sc">:</span><span class="dv">20</span>,]</span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mar =</span> <span class="fu">c</span>(<span class="dv">7</span>, <span class="dv">6</span>, <span class="dv">3</span>, <span class="dv">0</span>))</span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a>bars <span class="ot">=</span> <span class="fu">barplot</span>(bartintxn_df<span class="sc">$</span>Importance, </span>
<span id="cb38-20"><a href="#cb38-20" aria-hidden="true" tabindex="-1"></a>               <span class="at">names.arg =</span> bartintxn_df<span class="sc">$</span>Protein, </span>
<span id="cb38-21"><a href="#cb38-21" aria-hidden="true" tabindex="-1"></a>               <span class="at">las =</span> <span class="dv">2</span>, </span>
<span id="cb38-22"><a href="#cb38-22" aria-hidden="true" tabindex="-1"></a>               <span class="at">ylab =</span> <span class="st">"Importance"</span>, </span>
<span id="cb38-23"><a href="#cb38-23" aria-hidden="true" tabindex="-1"></a>               <span class="at">col =</span> <span class="st">"gray"</span>,</span>
<span id="cb38-24"><a href="#cb38-24" aria-hidden="true" tabindex="-1"></a>               <span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fu">max</span>(bartintxn_df<span class="sc">$</span>Importance <span class="sc">+</span> bartintxn_df<span class="sc">$</span>MOE))</span>
<span id="cb38-25"><a href="#cb38-25" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb38-26"><a href="#cb38-26" aria-hidden="true" tabindex="-1"></a>conf_upper <span class="ot">=</span> bartintxn_df<span class="sc">$</span>Importance <span class="sc">+</span> bartintxn_df<span class="sc">$</span>MOE</span>
<span id="cb38-27"><a href="#cb38-27" aria-hidden="true" tabindex="-1"></a>conf_lower <span class="ot">=</span> bartintxn_df<span class="sc">$</span>Importance <span class="sc">-</span> bartintxn_df<span class="sc">$</span>MOE</span>
<span id="cb38-28"><a href="#cb38-28" aria-hidden="true" tabindex="-1"></a><span class="fu">segments</span>(bars, bartintxn_df<span class="sc">$</span>Importance, bars, conf_upper, <span class="at">col =</span> <span class="fu">rgb</span>(<span class="fl">0.59</span>, <span class="fl">0.39</span>, <span class="fl">0.39</span>), <span class="at">lwd =</span> <span class="dv">3</span>) <span class="co"># Draw error bars</span></span>
<span id="cb38-29"><a href="#cb38-29" aria-hidden="true" tabindex="-1"></a><span class="fu">segments</span>(bars, bartintxn_df<span class="sc">$</span>Importance, bars, conf_lower, <span class="at">col =</span> <span class="fu">rgb</span>(<span class="fl">0.59</span>, <span class="fl">0.39</span>, <span class="fl">0.39</span>), <span class="at">lwd =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="mwd-2" class="level3">
<h3 class="anchored" data-anchor-id="mwd-2">6MWD</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">231</span>)</span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">java.parameters =</span> <span class="st">"-Xmx4000m"</span>)</span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a><span class="fu">set_bart_machine_num_cores</span>(<span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine now using 1 cores.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="co">#model fit</span></span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a>smw_data <span class="ot">&lt;-</span> <span class="fu">cbind</span>(rprotblds[,<span class="fu">c</span>(<span class="st">"fu_smw"</span>,<span class="st">"intervention_1_control_0"</span>,<span class="st">"bl_smw"</span>,<span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,<span class="st">"hf_cat"</span>)],</span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a>                  rprotblds[,<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>])</span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a><span class="co">#remove incomplete cases (any rows with missingness, mostly due to lack of follow up)</span></span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a>smw_data <span class="ot">=</span> <span class="fu">na.omit</span>(smw_data);</span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a>smw_y <span class="ot">&lt;-</span> smw_data<span class="sc">$</span>fu_smw</span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a>smw_x <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(smw_data<span class="sc">$</span>intervention_1_control_0,</span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>                    smw_data<span class="sc">$</span>bl_smw,smw_data<span class="sc">$</span>age,</span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a>                    smw_data<span class="sc">$</span>sex,</span>
<span id="cb41-12"><a href="#cb41-12" aria-hidden="true" tabindex="-1"></a>                    smw_data<span class="sc">$</span>race___4,</span>
<span id="cb41-13"><a href="#cb41-13" aria-hidden="true" tabindex="-1"></a>                    smw_data<span class="sc">$</span>hf_cat,</span>
<span id="cb41-14"><a href="#cb41-14" aria-hidden="true" tabindex="-1"></a>                    smw_data[,<span class="dv">8</span><span class="sc">:</span><span class="dv">99</span>])</span>
<span id="cb41-15"><a href="#cb41-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-16"><a href="#cb41-16" aria-hidden="true" tabindex="-1"></a>bart_machine_smw <span class="ot">&lt;-</span> <span class="fu">bartMachine</span>(smw_x, smw_y,</span>
<span id="cb41-17"><a href="#cb41-17" aria-hidden="true" tabindex="-1"></a>                                <span class="at">num_trees=</span><span class="dv">100</span>,</span>
<span id="cb41-18"><a href="#cb41-18" aria-hidden="true" tabindex="-1"></a>                                <span class="at">num_burn_in=</span><span class="dv">2000</span>,</span>
<span id="cb41-19"><a href="#cb41-19" aria-hidden="true" tabindex="-1"></a>                                <span class="at">num_iterations_after_burn_in=</span><span class="dv">2000</span>,</span>
<span id="cb41-20"><a href="#cb41-20" aria-hidden="true" tabindex="-1"></a>                                <span class="at">use_missing_data=</span><span class="cn">TRUE</span>,</span>
<span id="cb41-21"><a href="#cb41-21" aria-hidden="true" tabindex="-1"></a>                                <span class="at">mem_cache_for_speed =</span> <span class="cn">FALSE</span>,</span>
<span id="cb41-22"><a href="#cb41-22" aria-hidden="true" tabindex="-1"></a>                                <span class="at">seed=</span><span class="dv">231</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine initializing with 100 trees...
bartMachine vars checked...
bartMachine java init...
bartMachine factors created...
bartMachine before preprocess...
bartMachine after preprocess... 102 total features...
bartMachine sigsq estimated...
bartMachine training data finalized...
Now building bartMachine for regression...Covariate importance prior ON. Missing data feature ON. 
evaluating in sample data...done</code></pre>
</div>
<div class="sourceCode cell-code" id="cb43"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a>bart_machine_smw</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>bartMachine v1.3.4.1 for regression

Missing data feature ON
training data size: n = 172 and p = 102 
built in 5.8 secs on 1 core, 100 trees, 2000 burn-in and 2000 post. samples

sigsq est for y beforehand: 3101.303 
avg sigsq estimate after burn-in: 1712.49861 

in-sample statistics:
 L1 = 2410.51 
 L2 = 61764.6 
 rmse = 18.95 
 Pseudo-Rsq = 0.9788
p-val for shapiro-wilk test of normality of residuals: 5e-05 
p-val for zero-mean noise: 0.95756 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_convergence_diagnostics</span>(bart_machine_smw)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a>reps<span class="ot">=</span><span class="dv">100</span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a>intxninvest_smw<span class="ot">=</span><span class="fu">interaction_investigator</span>(bart_machine_smw,</span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">plot=</span><span class="cn">FALSE</span>,</span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">num_var_plot=</span><span class="dv">25</span>,</span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">bottom_margin=</span><span class="dv">20</span>,</span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">num_replicates_for_avg=</span>reps,</span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">num_trees_bottleneck=</span><span class="dv">20</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>........................................
........................................
....................</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb48"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1" aria-hidden="true" tabindex="-1"></a><span class="co">#average count of interactions between protein and treatment/control</span></span>
<span id="cb48-2"><a href="#cb48-2" aria-hidden="true" tabindex="-1"></a><span class="co">#sort(rowSums(intxninvest_smw$interaction_counts_avg[3:94,95:96]),decreasing=TRUE)</span></span>
<span id="cb48-3"><a href="#cb48-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-4"><a href="#cb48-4" aria-hidden="true" tabindex="-1"></a><span class="co">#mean and sd of interaction counts</span></span>
<span id="cb48-5"><a href="#cb48-5" aria-hidden="true" tabindex="-1"></a><span class="co">#cbind(intxninvest_smw$interaction_counts_avg[3:94,95:96],</span></span>
<span id="cb48-6"><a href="#cb48-6" aria-hidden="true" tabindex="-1"></a><span class="co">#      1.96*intxninvest_smw$interaction_counts_sd[3:94,95:96]/sqrt(reps))</span></span>
<span id="cb48-7"><a href="#cb48-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-8"><a href="#cb48-8" aria-hidden="true" tabindex="-1"></a><span class="co">#sum of count of interactions</span></span>
<span id="cb48-9"><a href="#cb48-9" aria-hidden="true" tabindex="-1"></a>bartintxn_means<span class="ot">=</span><span class="fu">apply</span>(intxninvest_smw<span class="sc">$</span>interaction_counts[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>,<span class="dv">95</span><span class="sc">:</span><span class="dv">96</span>,],<span class="dv">1</span>,sum)<span class="sc">/</span>reps</span>
<span id="cb48-10"><a href="#cb48-10" aria-hidden="true" tabindex="-1"></a>bartintxn_sds<span class="ot">=</span><span class="fu">apply</span>(<span class="fu">apply</span>(intxninvest_smw<span class="sc">$</span>interaction_counts[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>,<span class="dv">95</span><span class="sc">:</span><span class="dv">96</span>,],<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>),sum),<span class="dv">1</span>,sd)</span>
<span id="cb48-11"><a href="#cb48-11" aria-hidden="true" tabindex="-1"></a>bartintxn_moe<span class="ot">=</span><span class="fl">1.96</span> <span class="sc">*</span> bartintxn_sds <span class="sc">/</span> <span class="fu">sqrt</span>(reps)</span>
<span id="cb48-12"><a href="#cb48-12" aria-hidden="true" tabindex="-1"></a>bartintxn_df<span class="ot">=</span><span class="fu">data.frame</span>(<span class="at">Protein=</span><span class="fu">rownames</span>(intxninvest_smw<span class="sc">$</span>interaction_counts_avg)[<span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>],</span>
<span id="cb48-13"><a href="#cb48-13" aria-hidden="true" tabindex="-1"></a>                        <span class="at">Importance=</span>bartintxn_means,<span class="at">MOE=</span>bartintxn_moe)</span>
<span id="cb48-14"><a href="#cb48-14" aria-hidden="true" tabindex="-1"></a>bartintxn_df <span class="ot">=</span> bartintxn_df[<span class="fu">order</span>(bartintxn_df<span class="sc">$</span>Importance,<span class="at">decreasing=</span><span class="cn">TRUE</span>),]</span>
<span id="cb48-15"><a href="#cb48-15" aria-hidden="true" tabindex="-1"></a><span class="co">#top 20 only</span></span>
<span id="cb48-16"><a href="#cb48-16" aria-hidden="true" tabindex="-1"></a>bartintxn_df<span class="ot">=</span>bartintxn_df[<span class="dv">1</span><span class="sc">:</span><span class="dv">20</span>,]</span>
<span id="cb48-17"><a href="#cb48-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-18"><a href="#cb48-18" aria-hidden="true" tabindex="-1"></a><span class="co">#par(mar = c(7, 6, 3, 0))</span></span>
<span id="cb48-19"><a href="#cb48-19" aria-hidden="true" tabindex="-1"></a>bars <span class="ot">=</span> <span class="fu">barplot</span>(bartintxn_df<span class="sc">$</span>Importance, </span>
<span id="cb48-20"><a href="#cb48-20" aria-hidden="true" tabindex="-1"></a>               <span class="at">names.arg =</span> bartintxn_df<span class="sc">$</span>Protein, </span>
<span id="cb48-21"><a href="#cb48-21" aria-hidden="true" tabindex="-1"></a>               <span class="at">las =</span> <span class="dv">2</span>, </span>
<span id="cb48-22"><a href="#cb48-22" aria-hidden="true" tabindex="-1"></a>               <span class="at">ylab =</span> <span class="st">"Importance"</span>, </span>
<span id="cb48-23"><a href="#cb48-23" aria-hidden="true" tabindex="-1"></a>               <span class="at">col =</span> <span class="st">"gray"</span>,</span>
<span id="cb48-24"><a href="#cb48-24" aria-hidden="true" tabindex="-1"></a>               <span class="at">ylim =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fu">max</span>(bartintxn_df<span class="sc">$</span>Importance <span class="sc">+</span> bartintxn_df<span class="sc">$</span>MOE))</span>
<span id="cb48-25"><a href="#cb48-25" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb48-26"><a href="#cb48-26" aria-hidden="true" tabindex="-1"></a>conf_upper <span class="ot">=</span> bartintxn_df<span class="sc">$</span>Importance <span class="sc">+</span> bartintxn_df<span class="sc">$</span>MOE</span>
<span id="cb48-27"><a href="#cb48-27" aria-hidden="true" tabindex="-1"></a>conf_lower <span class="ot">=</span> bartintxn_df<span class="sc">$</span>Importance <span class="sc">-</span> bartintxn_df<span class="sc">$</span>MOE</span>
<span id="cb48-28"><a href="#cb48-28" aria-hidden="true" tabindex="-1"></a><span class="fu">segments</span>(bars, bartintxn_df<span class="sc">$</span>Importance, bars, conf_upper, <span class="at">col =</span> <span class="fu">rgb</span>(<span class="fl">0.59</span>, <span class="fl">0.39</span>, <span class="fl">0.39</span>), <span class="at">lwd =</span> <span class="dv">3</span>) <span class="co"># Draw error bars</span></span>
<span id="cb48-29"><a href="#cb48-29" aria-hidden="true" tabindex="-1"></a><span class="fu">segments</span>(bars, bartintxn_df<span class="sc">$</span>Importance, bars, conf_lower, <span class="at">col =</span> <span class="fu">rgb</span>(<span class="fl">0.59</span>, <span class="fl">0.39</span>, <span class="fl">0.39</span>), <span class="at">lwd =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="selected-emm-proteins" class="level2">
<h2 class="anchored" data-anchor-id="selected-emm-proteins">Selected EMM Proteins</h2>
<table class="caption-top table">
<thead>
<tr class="header">
<th></th>
<th>SPPB</th>
<th>6MWD</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Individual regressions</td>
<td>None</td>
<td>None</td>
</tr>
<tr class="even">
<td>LASSO with strong hierarchy</td>
<td>GP6</td>
<td>ALCAM, Gal-4</td>
</tr>
<tr class="odd">
<td>BART for nonlinear interactions</td>
<td>ST2</td>
<td>Gal-4, PECAM-1</td>
</tr>
</tbody>
</table>
</section>
</section>
<section id="regression-analyses" class="level1">
<h1>Regression analyses</h1>
<section id="table-2" class="level2">
<h2 class="anchored" data-anchor-id="table-2">Table 2</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Rename protein columns to R-safe names</span></span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>tmpdf <span class="ot">&lt;-</span> rprotblds</span>
<span id="cb49-3"><a href="#cb49-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(tmpdf)[<span class="fu">colnames</span>(tmpdf) <span class="sc">%in%</span> protlist] <span class="ot">&lt;-</span> <span class="fu">make.names</span>(protlist, <span class="at">unique =</span> <span class="cn">TRUE</span>)</span>
<span id="cb49-4"><a href="#cb49-4" aria-hidden="true" tabindex="-1"></a>protlistclean <span class="ot">&lt;-</span> <span class="fu">make.names</span>(protlist, <span class="at">unique =</span> <span class="cn">TRUE</span>)</span>
<span id="cb49-5"><a href="#cb49-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-6"><a href="#cb49-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Initialize results data frames</span></span>
<span id="cb49-7"><a href="#cb49-7" aria-hidden="true" tabindex="-1"></a>regresults.sppb <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb49-8"><a href="#cb49-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">Protein =</span> protlist,</span>
<span id="cb49-9"><a href="#cb49-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">Unadjusted.Effect =</span> <span class="cn">NA_character_</span>,</span>
<span id="cb49-10"><a href="#cb49-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">Unadjusted.Pval =</span> <span class="cn">NA_real_</span>,</span>
<span id="cb49-11"><a href="#cb49-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">Adjusted.Effect =</span> <span class="cn">NA_character_</span>,</span>
<span id="cb49-12"><a href="#cb49-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">Adjusted.Pval =</span> <span class="cn">NA_real_</span>,</span>
<span id="cb49-13"><a href="#cb49-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">Interaction.Pval =</span> <span class="cn">NA_real_</span>,</span>
<span id="cb49-14"><a href="#cb49-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span></span>
<span id="cb49-15"><a href="#cb49-15" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb49-16"><a href="#cb49-16" aria-hidden="true" tabindex="-1"></a>regresults<span class="fl">.6</span>mwd <span class="ot">&lt;-</span> regresults.sppb</span>
<span id="cb49-17"><a href="#cb49-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-18"><a href="#cb49-18" aria-hidden="true" tabindex="-1"></a><span class="co"># Run regressions</span></span>
<span id="cb49-19"><a href="#cb49-19" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_along</span>(protlist)) {</span>
<span id="cb49-20"><a href="#cb49-20" aria-hidden="true" tabindex="-1"></a>  prot_col <span class="ot">&lt;-</span> protlistclean[i]</span>
<span id="cb49-21"><a href="#cb49-21" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb49-22"><a href="#cb49-22" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Prepare data for SPPB</span></span>
<span id="cb49-23"><a href="#cb49-23" aria-hidden="true" tabindex="-1"></a>  idx <span class="ot">&lt;-</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>fu_sppb) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>bl_sppb) <span class="sc">&amp;</span> </span>
<span id="cb49-24"><a href="#cb49-24" aria-hidden="true" tabindex="-1"></a>               <span class="sc">!</span><span class="fu">is.na</span>(tmpdf[[prot_col]]) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>intervention_1_control_0))</span>
<span id="cb49-25"><a href="#cb49-25" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb49-26"><a href="#cb49-26" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">length</span>(idx) <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb49-27"><a href="#cb49-27" aria-hidden="true" tabindex="-1"></a>    y_sppb <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>fu_sppb[idx]</span>
<span id="cb49-28"><a href="#cb49-28" aria-hidden="true" tabindex="-1"></a>    x_sppb_bl <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>bl_sppb[idx]</span>
<span id="cb49-29"><a href="#cb49-29" aria-hidden="true" tabindex="-1"></a>    x_prot <span class="ot">&lt;-</span> tmpdf[[prot_col]][idx]</span>
<span id="cb49-30"><a href="#cb49-30" aria-hidden="true" tabindex="-1"></a>    x_int <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>intervention_1_control_0[idx]</span>
<span id="cb49-31"><a href="#cb49-31" aria-hidden="true" tabindex="-1"></a>    x_age <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>age[idx]</span>
<span id="cb49-32"><a href="#cb49-32" aria-hidden="true" tabindex="-1"></a>    x_sex <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>sex[idx]</span>
<span id="cb49-33"><a href="#cb49-33" aria-hidden="true" tabindex="-1"></a>    x_race <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>race___4[idx]</span>
<span id="cb49-34"><a href="#cb49-34" aria-hidden="true" tabindex="-1"></a>    x_hf <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>hf_cat[idx]</span>
<span id="cb49-35"><a href="#cb49-35" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-36"><a href="#cb49-36" aria-hidden="true" tabindex="-1"></a>    <span class="co"># SPPB - Unadjusted</span></span>
<span id="cb49-37"><a href="#cb49-37" aria-hidden="true" tabindex="-1"></a>    mod.sppb.unadj <span class="ot">&lt;-</span> <span class="fu">lm</span>(y_sppb <span class="sc">~</span> x_sppb_bl <span class="sc">+</span> x_prot <span class="sc">+</span> x_int)</span>
<span id="cb49-38"><a href="#cb49-38" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-39"><a href="#cb49-39" aria-hidden="true" tabindex="-1"></a>    <span class="co"># SPPB - Adjusted</span></span>
<span id="cb49-40"><a href="#cb49-40" aria-hidden="true" tabindex="-1"></a>    mod.sppb.adj <span class="ot">&lt;-</span> <span class="fu">lm</span>(y_sppb <span class="sc">~</span> x_sppb_bl <span class="sc">+</span> x_prot <span class="sc">*</span> x_int <span class="sc">+</span> x_age <span class="sc">+</span> x_sex <span class="sc">+</span> x_race <span class="sc">+</span> x_hf <span class="sc">*</span> x_int)</span>
<span id="cb49-41"><a href="#cb49-41" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-42"><a href="#cb49-42" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Extract SPPB unadjusted results</span></span>
<span id="cb49-43"><a href="#cb49-43" aria-hidden="true" tabindex="-1"></a>    s.unadj <span class="ot">&lt;-</span> <span class="fu">summary</span>(mod.sppb.unadj)</span>
<span id="cb49-44"><a href="#cb49-44" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="st">"x_prot"</span> <span class="sc">%in%</span> <span class="fu">rownames</span>(s.unadj<span class="sc">$</span>coefficients)) {</span>
<span id="cb49-45"><a href="#cb49-45" aria-hidden="true" tabindex="-1"></a>      ci.u <span class="ot">&lt;-</span> <span class="fu">confint</span>(mod.sppb.unadj, <span class="st">"x_prot"</span>)</span>
<span id="cb49-46"><a href="#cb49-46" aria-hidden="true" tabindex="-1"></a>      regresults.sppb<span class="sc">$</span>Unadjusted.Effect[i] <span class="ot">&lt;-</span> <span class="fu">sprintf</span>(<span class="st">"%.2f (%.2f, %.2f)"</span>, </span>
<span id="cb49-47"><a href="#cb49-47" aria-hidden="true" tabindex="-1"></a>                                                       s.unadj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">1</span>],</span>
<span id="cb49-48"><a href="#cb49-48" aria-hidden="true" tabindex="-1"></a>                                                       ci.u[<span class="dv">1</span>], ci.u[<span class="dv">2</span>])</span>
<span id="cb49-49"><a href="#cb49-49" aria-hidden="true" tabindex="-1"></a>      regresults.sppb<span class="sc">$</span>Unadjusted.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.unadj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-50"><a href="#cb49-50" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb49-51"><a href="#cb49-51" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-52"><a href="#cb49-52" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Extract SPPB adjusted results</span></span>
<span id="cb49-53"><a href="#cb49-53" aria-hidden="true" tabindex="-1"></a>    s.adj <span class="ot">&lt;-</span> <span class="fu">summary</span>(mod.sppb.adj)</span>
<span id="cb49-54"><a href="#cb49-54" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="st">"x_prot"</span> <span class="sc">%in%</span> <span class="fu">rownames</span>(s.adj<span class="sc">$</span>coefficients)) {</span>
<span id="cb49-55"><a href="#cb49-55" aria-hidden="true" tabindex="-1"></a>      ci.a <span class="ot">&lt;-</span> <span class="fu">confint</span>(mod.sppb.adj, <span class="st">"x_prot"</span>)</span>
<span id="cb49-56"><a href="#cb49-56" aria-hidden="true" tabindex="-1"></a>      regresults.sppb<span class="sc">$</span>Adjusted.Effect[i] <span class="ot">&lt;-</span> <span class="fu">sprintf</span>(<span class="st">"%.2f (%.2f, %.2f)"</span>,</span>
<span id="cb49-57"><a href="#cb49-57" aria-hidden="true" tabindex="-1"></a>                                                     s.adj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">1</span>],</span>
<span id="cb49-58"><a href="#cb49-58" aria-hidden="true" tabindex="-1"></a>                                                     ci.a[<span class="dv">1</span>], ci.a[<span class="dv">2</span>])</span>
<span id="cb49-59"><a href="#cb49-59" aria-hidden="true" tabindex="-1"></a>      regresults.sppb<span class="sc">$</span>Adjusted.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.adj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-60"><a href="#cb49-60" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb49-61"><a href="#cb49-61" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Interaction term</span></span>
<span id="cb49-62"><a href="#cb49-62" aria-hidden="true" tabindex="-1"></a>      int.idx <span class="ot">&lt;-</span> <span class="fu">grep</span>(<span class="st">"x_prot:x_int"</span>, <span class="fu">rownames</span>(s.adj<span class="sc">$</span>coefficients))</span>
<span id="cb49-63"><a href="#cb49-63" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (<span class="fu">length</span>(int.idx) <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb49-64"><a href="#cb49-64" aria-hidden="true" tabindex="-1"></a>        regresults.sppb<span class="sc">$</span>Interaction.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.adj<span class="sc">$</span>coef[int.idx[<span class="dv">1</span>], <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-65"><a href="#cb49-65" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb49-66"><a href="#cb49-66" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb49-67"><a href="#cb49-67" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb49-68"><a href="#cb49-68" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb49-69"><a href="#cb49-69" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Prepare data for 6MWD</span></span>
<span id="cb49-70"><a href="#cb49-70" aria-hidden="true" tabindex="-1"></a>  idx<span class="fl">.6</span>mwd <span class="ot">&lt;-</span> <span class="fu">which</span>(<span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>fu_smw) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>bl_smw) <span class="sc">&amp;</span> </span>
<span id="cb49-71"><a href="#cb49-71" aria-hidden="true" tabindex="-1"></a>                    <span class="sc">!</span><span class="fu">is.na</span>(tmpdf[[prot_col]]) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(tmpdf<span class="sc">$</span>intervention_1_control_0))</span>
<span id="cb49-72"><a href="#cb49-72" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb49-73"><a href="#cb49-73" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">length</span>(idx<span class="fl">.6</span>mwd) <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb49-74"><a href="#cb49-74" aria-hidden="true" tabindex="-1"></a>    y_6mwd <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>fu_smw[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-75"><a href="#cb49-75" aria-hidden="true" tabindex="-1"></a>    x_6mwd_bl <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>bl_smw[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-76"><a href="#cb49-76" aria-hidden="true" tabindex="-1"></a>    x_prot <span class="ot">&lt;-</span> tmpdf[[prot_col]][idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-77"><a href="#cb49-77" aria-hidden="true" tabindex="-1"></a>    x_int <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>intervention_1_control_0[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-78"><a href="#cb49-78" aria-hidden="true" tabindex="-1"></a>    x_age <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>age[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-79"><a href="#cb49-79" aria-hidden="true" tabindex="-1"></a>    x_sex <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>sex[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-80"><a href="#cb49-80" aria-hidden="true" tabindex="-1"></a>    x_race <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>race___4[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-81"><a href="#cb49-81" aria-hidden="true" tabindex="-1"></a>    x_hf <span class="ot">&lt;-</span> tmpdf<span class="sc">$</span>hf_cat[idx<span class="fl">.6</span>mwd]</span>
<span id="cb49-82"><a href="#cb49-82" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-83"><a href="#cb49-83" aria-hidden="true" tabindex="-1"></a>    <span class="co"># 6MWD - Unadjusted</span></span>
<span id="cb49-84"><a href="#cb49-84" aria-hidden="true" tabindex="-1"></a>    mod<span class="fl">.6</span>mwd.unadj <span class="ot">&lt;-</span> <span class="fu">lm</span>(y_6mwd <span class="sc">~</span> x_6mwd_bl <span class="sc">+</span> x_prot <span class="sc">+</span> x_int)</span>
<span id="cb49-85"><a href="#cb49-85" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-86"><a href="#cb49-86" aria-hidden="true" tabindex="-1"></a>    <span class="co"># 6MWD - Adjusted</span></span>
<span id="cb49-87"><a href="#cb49-87" aria-hidden="true" tabindex="-1"></a>    mod<span class="fl">.6</span>mwd.adj <span class="ot">&lt;-</span> <span class="fu">lm</span>(y_6mwd <span class="sc">~</span> x_6mwd_bl <span class="sc">+</span> x_prot <span class="sc">*</span> x_int <span class="sc">+</span> x_age <span class="sc">+</span> x_sex <span class="sc">+</span> x_race <span class="sc">+</span> x_hf <span class="sc">*</span> x_int)</span>
<span id="cb49-88"><a href="#cb49-88" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-89"><a href="#cb49-89" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Extract 6MWD unadjusted results</span></span>
<span id="cb49-90"><a href="#cb49-90" aria-hidden="true" tabindex="-1"></a>    s.unadj <span class="ot">&lt;-</span> <span class="fu">summary</span>(mod<span class="fl">.6</span>mwd.unadj)</span>
<span id="cb49-91"><a href="#cb49-91" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="st">"x_prot"</span> <span class="sc">%in%</span> <span class="fu">rownames</span>(s.unadj<span class="sc">$</span>coefficients)) {</span>
<span id="cb49-92"><a href="#cb49-92" aria-hidden="true" tabindex="-1"></a>      ci.u <span class="ot">&lt;-</span> <span class="fu">confint</span>(mod<span class="fl">.6</span>mwd.unadj, <span class="st">"x_prot"</span>)</span>
<span id="cb49-93"><a href="#cb49-93" aria-hidden="true" tabindex="-1"></a>      regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Unadjusted.Effect[i] <span class="ot">&lt;-</span> <span class="fu">sprintf</span>(<span class="st">"%.2f (%.2f, %.2f)"</span>,</span>
<span id="cb49-94"><a href="#cb49-94" aria-hidden="true" tabindex="-1"></a>                                                       s.unadj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">1</span>],</span>
<span id="cb49-95"><a href="#cb49-95" aria-hidden="true" tabindex="-1"></a>                                                       ci.u[<span class="dv">1</span>], ci.u[<span class="dv">2</span>])</span>
<span id="cb49-96"><a href="#cb49-96" aria-hidden="true" tabindex="-1"></a>      regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Unadjusted.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.unadj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-97"><a href="#cb49-97" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb49-98"><a href="#cb49-98" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb49-99"><a href="#cb49-99" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Extract 6MWD adjusted results</span></span>
<span id="cb49-100"><a href="#cb49-100" aria-hidden="true" tabindex="-1"></a>    s.adj <span class="ot">&lt;-</span> <span class="fu">summary</span>(mod<span class="fl">.6</span>mwd.adj)</span>
<span id="cb49-101"><a href="#cb49-101" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="st">"x_prot"</span> <span class="sc">%in%</span> <span class="fu">rownames</span>(s.adj<span class="sc">$</span>coefficients)) {</span>
<span id="cb49-102"><a href="#cb49-102" aria-hidden="true" tabindex="-1"></a>      ci.a <span class="ot">&lt;-</span> <span class="fu">confint</span>(mod<span class="fl">.6</span>mwd.adj, <span class="st">"x_prot"</span>)</span>
<span id="cb49-103"><a href="#cb49-103" aria-hidden="true" tabindex="-1"></a>      regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Adjusted.Effect[i] <span class="ot">&lt;-</span> <span class="fu">sprintf</span>(<span class="st">"%.2f (%.2f, %.2f)"</span>,</span>
<span id="cb49-104"><a href="#cb49-104" aria-hidden="true" tabindex="-1"></a>                                                     s.adj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">1</span>],</span>
<span id="cb49-105"><a href="#cb49-105" aria-hidden="true" tabindex="-1"></a>                                                     ci.a[<span class="dv">1</span>], ci.a[<span class="dv">2</span>])</span>
<span id="cb49-106"><a href="#cb49-106" aria-hidden="true" tabindex="-1"></a>      regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Adjusted.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.adj<span class="sc">$</span>coef[<span class="st">"x_prot"</span>, <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-107"><a href="#cb49-107" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb49-108"><a href="#cb49-108" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Interaction term</span></span>
<span id="cb49-109"><a href="#cb49-109" aria-hidden="true" tabindex="-1"></a>      int.idx <span class="ot">&lt;-</span> <span class="fu">grep</span>(<span class="st">"x_prot:x_int"</span>, <span class="fu">rownames</span>(s.adj<span class="sc">$</span>coefficients))</span>
<span id="cb49-110"><a href="#cb49-110" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span> (<span class="fu">length</span>(int.idx) <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb49-111"><a href="#cb49-111" aria-hidden="true" tabindex="-1"></a>        regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Interaction.Pval[i] <span class="ot">&lt;-</span> <span class="fu">round</span>(s.adj<span class="sc">$</span>coef[int.idx[<span class="dv">1</span>], <span class="dv">4</span>], <span class="dv">4</span>)</span>
<span id="cb49-112"><a href="#cb49-112" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb49-113"><a href="#cb49-113" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb49-114"><a href="#cb49-114" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb49-115"><a href="#cb49-115" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb49-116"><a href="#cb49-116" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-117"><a href="#cb49-117" aria-hidden="true" tabindex="-1"></a><span class="co"># Display only selected proteins</span></span>
<span id="cb49-118"><a href="#cb49-118" aria-hidden="true" tabindex="-1"></a>sppb_proteins <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"GP6"</span>, <span class="st">"ST2"</span>)</span>
<span id="cb49-119"><a href="#cb49-119" aria-hidden="true" tabindex="-1"></a>smwd_proteins <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"ALCAM"</span>, <span class="st">"Gal-4"</span>, <span class="st">"PECAM-1"</span>)</span>
<span id="cb49-120"><a href="#cb49-120" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-121"><a href="#cb49-121" aria-hidden="true" tabindex="-1"></a><span class="fu">gt</span>(regresults.sppb[regresults.sppb<span class="sc">$</span>Protein <span class="sc">%in%</span> sppb_proteins, ]) <span class="sc">%&gt;%</span></span>
<span id="cb49-122"><a href="#cb49-122" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(<span class="at">title =</span> <span class="st">"SPPB Regression Results"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="ddpxfuided" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#ddpxfuided table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#ddpxfuided thead, #ddpxfuided tbody, #ddpxfuided tfoot, #ddpxfuided tr, #ddpxfuided td, #ddpxfuided th {
  border-style: none;
}

#ddpxfuided p {
  margin: 0;
  padding: 0;
}

#ddpxfuided .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ddpxfuided .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#ddpxfuided .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ddpxfuided .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ddpxfuided .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ddpxfuided .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ddpxfuided .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ddpxfuided .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ddpxfuided .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ddpxfuided .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ddpxfuided .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ddpxfuided .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ddpxfuided .gt_spanner_row {
  border-bottom-style: hidden;
}

#ddpxfuided .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#ddpxfuided .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ddpxfuided .gt_from_md > :first-child {
  margin-top: 0;
}

#ddpxfuided .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ddpxfuided .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ddpxfuided .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ddpxfuided .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ddpxfuided .gt_row_group_first td {
  border-top-width: 2px;
}

#ddpxfuided .gt_row_group_first th {
  border-top-width: 2px;
}

#ddpxfuided .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ddpxfuided .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#ddpxfuided .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ddpxfuided .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ddpxfuided .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ddpxfuided .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ddpxfuided .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#ddpxfuided .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ddpxfuided .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ddpxfuided .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ddpxfuided .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ddpxfuided .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ddpxfuided .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ddpxfuided .gt_left {
  text-align: left;
}

#ddpxfuided .gt_center {
  text-align: center;
}

#ddpxfuided .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ddpxfuided .gt_font_normal {
  font-weight: normal;
}

#ddpxfuided .gt_font_bold {
  font-weight: bold;
}

#ddpxfuided .gt_font_italic {
  font-style: italic;
}

#ddpxfuided .gt_super {
  font-size: 65%;
}

#ddpxfuided .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#ddpxfuided .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ddpxfuided .gt_indent_1 {
  text-indent: 5px;
}

#ddpxfuided .gt_indent_2 {
  text-indent: 10px;
}

#ddpxfuided .gt_indent_3 {
  text-indent: 15px;
}

#ddpxfuided .gt_indent_4 {
  text-indent: 20px;
}

#ddpxfuided .gt_indent_5 {
  text-indent: 25px;
}

#ddpxfuided .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#ddpxfuided div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border">SPPB Regression Results</th>
</tr>
<tr class="gt_col_headings even">
<th id="Protein" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Protein</th>
<th id="Unadjusted.Effect" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Unadjusted.Effect</th>
<th id="Unadjusted.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Unadjusted.Pval</th>
<th id="Adjusted.Effect" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Adjusted.Effect</th>
<th id="Adjusted.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Adjusted.Pval</th>
<th id="Interaction.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Interaction.Pval</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Protein">GP6</td>
<td class="gt_row gt_right" headers="Unadjusted.Effect">-0.69 (-1.09, -0.30)</td>
<td class="gt_row gt_right" headers="Unadjusted.Pval">0.0007</td>
<td class="gt_row gt_right" headers="Adjusted.Effect">-0.56 (-1.12, 0.00)</td>
<td class="gt_row gt_right" headers="Adjusted.Pval">0.0507</td>
<td class="gt_row gt_right" headers="Interaction.Pval">0.8223</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Protein">ST2</td>
<td class="gt_row gt_right" headers="Unadjusted.Effect">-0.76 (-1.67, 0.15)</td>
<td class="gt_row gt_right" headers="Unadjusted.Pval">0.1004</td>
<td class="gt_row gt_right" headers="Adjusted.Effect">-1.16 (-2.37, 0.05)</td>
<td class="gt_row gt_right" headers="Adjusted.Pval">0.0591</td>
<td class="gt_row gt_right" headers="Interaction.Pval">0.0930</td>
</tr>
</tbody>
</table>

</div>
<p>Adjustment covariates: age, sex, race, heart failure subtype (reduced or preserved ejection fraction) and treatment assignment. Interaction P-value is for protein level * treatment assignment. Units for SPPB are based on an integer score of 0-12 and 6MWD is in meters. The effect size is per NPX expression unit of the protein which on a log2 scale such that an increase of one unit is equivalent to a doubling of the protein level.</p>
</div>
<div class="sourceCode cell-code" id="cb50"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a><span class="fu">gt</span>(regresults<span class="fl">.6</span>mwd[regresults<span class="fl">.6</span>mwd<span class="sc">$</span>Protein <span class="sc">%in%</span> smwd_proteins, ]) <span class="sc">%&gt;%</span></span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(<span class="at">title =</span> <span class="st">"6MWD Regression Results"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="oxhkodhyiu" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#oxhkodhyiu table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#oxhkodhyiu thead, #oxhkodhyiu tbody, #oxhkodhyiu tfoot, #oxhkodhyiu tr, #oxhkodhyiu td, #oxhkodhyiu th {
  border-style: none;
}

#oxhkodhyiu p {
  margin: 0;
  padding: 0;
}

#oxhkodhyiu .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#oxhkodhyiu .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#oxhkodhyiu .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#oxhkodhyiu .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#oxhkodhyiu .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#oxhkodhyiu .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#oxhkodhyiu .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#oxhkodhyiu .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#oxhkodhyiu .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#oxhkodhyiu .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#oxhkodhyiu .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#oxhkodhyiu .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#oxhkodhyiu .gt_spanner_row {
  border-bottom-style: hidden;
}

#oxhkodhyiu .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#oxhkodhyiu .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#oxhkodhyiu .gt_from_md > :first-child {
  margin-top: 0;
}

#oxhkodhyiu .gt_from_md > :last-child {
  margin-bottom: 0;
}

#oxhkodhyiu .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#oxhkodhyiu .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#oxhkodhyiu .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#oxhkodhyiu .gt_row_group_first td {
  border-top-width: 2px;
}

#oxhkodhyiu .gt_row_group_first th {
  border-top-width: 2px;
}

#oxhkodhyiu .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#oxhkodhyiu .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#oxhkodhyiu .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#oxhkodhyiu .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#oxhkodhyiu .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#oxhkodhyiu .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#oxhkodhyiu .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#oxhkodhyiu .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#oxhkodhyiu .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#oxhkodhyiu .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#oxhkodhyiu .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#oxhkodhyiu .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#oxhkodhyiu .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#oxhkodhyiu .gt_left {
  text-align: left;
}

#oxhkodhyiu .gt_center {
  text-align: center;
}

#oxhkodhyiu .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#oxhkodhyiu .gt_font_normal {
  font-weight: normal;
}

#oxhkodhyiu .gt_font_bold {
  font-weight: bold;
}

#oxhkodhyiu .gt_font_italic {
  font-style: italic;
}

#oxhkodhyiu .gt_super {
  font-size: 65%;
}

#oxhkodhyiu .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#oxhkodhyiu .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#oxhkodhyiu .gt_indent_1 {
  text-indent: 5px;
}

#oxhkodhyiu .gt_indent_2 {
  text-indent: 10px;
}

#oxhkodhyiu .gt_indent_3 {
  text-indent: 15px;
}

#oxhkodhyiu .gt_indent_4 {
  text-indent: 20px;
}

#oxhkodhyiu .gt_indent_5 {
  text-indent: 25px;
}

#oxhkodhyiu .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#oxhkodhyiu div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="6" class="gt_heading gt_title gt_font_normal gt_bottom_border">6MWD Regression Results</th>
</tr>
<tr class="gt_col_headings even">
<th id="Protein" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Protein</th>
<th id="Unadjusted.Effect" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Unadjusted.Effect</th>
<th id="Unadjusted.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Unadjusted.Pval</th>
<th id="Adjusted.Effect" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Adjusted.Effect</th>
<th id="Adjusted.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Adjusted.Pval</th>
<th id="Interaction.Pval" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Interaction.Pval</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Protein">ALCAM</td>
<td class="gt_row gt_right" headers="Unadjusted.Effect">-13.19 (-51.67, 25.29)</td>
<td class="gt_row gt_right" headers="Unadjusted.Pval">0.4996</td>
<td class="gt_row gt_right" headers="Adjusted.Effect">-85.50 (-150.83, -20.17)</td>
<td class="gt_row gt_right" headers="Adjusted.Pval">0.0106</td>
<td class="gt_row gt_right" headers="Interaction.Pval">0.0131</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Protein">Gal-4</td>
<td class="gt_row gt_right" headers="Unadjusted.Effect">14.32 (-10.72, 39.37)</td>
<td class="gt_row gt_right" headers="Unadjusted.Pval">0.2606</td>
<td class="gt_row gt_right" headers="Adjusted.Effect">31.56 (-12.70, 75.81)</td>
<td class="gt_row gt_right" headers="Adjusted.Pval">0.1611</td>
<td class="gt_row gt_right" headers="Interaction.Pval">0.2296</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Protein">PECAM-1</td>
<td class="gt_row gt_right" headers="Unadjusted.Effect">-18.79 (-39.36, 1.77)</td>
<td class="gt_row gt_right" headers="Unadjusted.Pval">0.0730</td>
<td class="gt_row gt_right" headers="Adjusted.Effect">-41.75 (-72.11, -11.39)</td>
<td class="gt_row gt_right" headers="Adjusted.Pval">0.0073</td>
<td class="gt_row gt_right" headers="Interaction.Pval">0.0395</td>
</tr>
</tbody>
</table>

</div>
<p>Adjustment covariates: age, sex, race, heart failure subtype (reduced or preserved ejection fraction) and treatment assignment. Interaction P-value is for protein level * treatment assignment. Units for SPPB are based on an integer score of 0-12 and 6MWD is in meters. The effect size is per NPX expression unit of the protein which on a log2 scale such that an increase of one unit is equivalent to a doubling of the protein level.</p>
</div>
</div>
</section>
<section id="figure-2" class="level2">
<h2 class="anchored" data-anchor-id="figure-2">Figure 2</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create long form dataset</span></span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a>rprotblds_long<span class="ot">=</span><span class="fu">pivot_longer</span>(<span class="at">data=</span>rprotblds[,</span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>                                       <span class="fu">c</span>(<span class="st">"subject_id"</span>,<span class="st">"timepoint"</span>,<span class="st">"intervention_1_control_0"</span>,</span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>                                          <span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,protlist)],</span>
<span id="cb51-5"><a href="#cb51-5" aria-hidden="true" tabindex="-1"></a>                             <span class="at">cols=</span>protlist,</span>
<span id="cb51-6"><a href="#cb51-6" aria-hidden="true" tabindex="-1"></a>                          <span class="at">names_to=</span><span class="st">'Protein'</span>,<span class="at">values_to=</span><span class="st">'ExpressionLevel'</span>)</span>
<span id="cb51-7"><a href="#cb51-7" aria-hidden="true" tabindex="-1"></a><span class="co">#rprotblds_long$intervention_1_control_0 &lt;- factor(rcombds_long_bl$intervention_1_control_0,levels=c(0,1))</span></span>
<span id="cb51-8"><a href="#cb51-8" aria-hidden="true" tabindex="-1"></a>rprotblds_long<span class="sc">$</span>sppb_chg <span class="ot">=</span> rprotblds_long<span class="sc">$</span>fu_sppb <span class="sc">-</span> rprotblds_long<span class="sc">$</span>bl_sppb</span>
<span id="cb51-9"><a href="#cb51-9" aria-hidden="true" tabindex="-1"></a>rprotblds_long<span class="sc">$</span>smw_chg <span class="ot">=</span> rprotblds_long<span class="sc">$</span>fu_smw <span class="sc">-</span> rprotblds_long<span class="sc">$</span>bl_smw</span>
<span id="cb51-10"><a href="#cb51-10" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(rprotblds_long)[<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="st">"Intervention"</span></span>
<span id="cb51-11"><a href="#cb51-11" aria-hidden="true" tabindex="-1"></a>rprotblds_long<span class="sc">$</span>Protein <span class="ot">=</span> <span class="fu">factor</span>(rprotblds_long<span class="sc">$</span>Protein,<span class="at">levels=</span>protlist)</span>
<span id="cb51-12"><a href="#cb51-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-13"><a href="#cb51-13" aria-hidden="true" tabindex="-1"></a>protlist<span class="fl">.6</span>mwd <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"ALCAM"</span>,<span class="st">"Gal-4"</span>,<span class="st">"PECAM-1"</span>)</span>
<span id="cb51-14"><a href="#cb51-14" aria-hidden="true" tabindex="-1"></a>protlist.sppb <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"GP6"</span>,<span class="st">"ST2"</span>)</span>
<span id="cb51-15"><a href="#cb51-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-16"><a href="#cb51-16" aria-hidden="true" tabindex="-1"></a><span class="co">#SPPB</span></span>
<span id="cb51-17"><a href="#cb51-17" aria-hidden="true" tabindex="-1"></a>fig2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data=</span>rprotblds_long[rprotblds_long<span class="sc">$</span>Protein <span class="sc">%in%</span> protlist.sppb,],</span>
<span id="cb51-18"><a href="#cb51-18" aria-hidden="true" tabindex="-1"></a>               <span class="at">mapping=</span><span class="fu">aes</span>(<span class="at">x=</span>ExpressionLevel,<span class="at">y=</span>sppb_chg,<span class="at">color=</span>Intervention))<span class="sc">+</span></span>
<span id="cb51-19"><a href="#cb51-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method=</span><span class="st">'lm'</span>)<span class="sc">+</span></span>
<span id="cb51-20"><a href="#cb51-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept=</span><span class="dv">0</span>,<span class="at">color=</span><span class="st">'black'</span>,<span class="at">linetype=</span><span class="st">'dashed'</span>)<span class="sc">+</span></span>
<span id="cb51-21"><a href="#cb51-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="st">'Protein'</span>,<span class="at">scales=</span><span class="st">'free'</span>,<span class="at">ncol=</span><span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb51-22"><a href="#cb51-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x=</span><span class="st">"Protein expression units (NPX)"</span>,<span class="at">y=</span><span class="st">"Change in SPPB Score"</span>)<span class="sc">+</span></span>
<span id="cb51-23"><a href="#cb51-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_discrete</span>(<span class="at">breaks=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">0</span>),</span>
<span id="cb51-24"><a href="#cb51-24" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels=</span><span class="fu">c</span>(<span class="st">"Rehabilitation</span><span class="sc">\n</span><span class="st">Intervention"</span>,<span class="st">"Attention</span><span class="sc">\n</span><span class="st">Control"</span>))<span class="sc">+</span></span>
<span id="cb51-25"><a href="#cb51-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb51-26"><a href="#cb51-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb51-27"><a href="#cb51-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb51-28"><a href="#cb51-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x  =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb51-29"><a href="#cb51-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y  =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb51-30"><a href="#cb51-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text   =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),   <span class="co"># protein facet labels</span></span>
<span id="cb51-31"><a href="#cb51-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb51-32"><a href="#cb51-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="fu">c</span>(<span class="fl">0.15</span>, <span class="fl">0.15</span>),</span>
<span id="cb51-33"><a href="#cb51-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.box.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">"black"</span>))</span>
<span id="cb51-34"><a href="#cb51-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-35"><a href="#cb51-35" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(fig2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/figure-2-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Linear associations between baseline protein expression levels and participants’ change from baseline to 12-weeks in the Short Physical Performance Battery differentiated based on the randomly assigned intervention.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="figure-3" class="level2">
<h2 class="anchored" data-anchor-id="figure-3">Figure 3</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb52"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="co">#6MWD</span></span>
<span id="cb52-2"><a href="#cb52-2" aria-hidden="true" tabindex="-1"></a>fig3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data=</span>rprotblds_long[rprotblds_long<span class="sc">$</span>Protein <span class="sc">%in%</span> protlist<span class="fl">.6</span>mwd,],</span>
<span id="cb52-3"><a href="#cb52-3" aria-hidden="true" tabindex="-1"></a>               <span class="at">mapping=</span><span class="fu">aes</span>(<span class="at">x=</span>ExpressionLevel,<span class="at">y=</span>smw_chg,<span class="at">color=</span>Intervention))<span class="sc">+</span></span>
<span id="cb52-4"><a href="#cb52-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_smooth</span>(<span class="at">method=</span><span class="st">'lm'</span>)<span class="sc">+</span></span>
<span id="cb52-5"><a href="#cb52-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept=</span><span class="dv">0</span>,<span class="at">color=</span><span class="st">'black'</span>,<span class="at">linetype=</span><span class="st">'dashed'</span>)<span class="sc">+</span></span>
<span id="cb52-6"><a href="#cb52-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="st">'Protein'</span>,<span class="at">scales=</span><span class="st">'free'</span>,<span class="at">ncol=</span><span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb52-7"><a href="#cb52-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x=</span><span class="st">"Protein expression units (NPX)"</span>,<span class="at">y=</span><span class="st">"Change in 6MWD (meters)"</span>)<span class="sc">+</span></span>
<span id="cb52-8"><a href="#cb52-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_discrete</span>(<span class="at">breaks=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">0</span>),</span>
<span id="cb52-9"><a href="#cb52-9" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels=</span><span class="fu">c</span>(<span class="st">"Rehabilitation</span><span class="sc">\n</span><span class="st">Intervention"</span>,<span class="st">"Attention</span><span class="sc">\n</span><span class="st">Control"</span>))<span class="sc">+</span></span>
<span id="cb52-10"><a href="#cb52-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb52-11"><a href="#cb52-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb52-12"><a href="#cb52-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb52-13"><a href="#cb52-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x  =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb52-14"><a href="#cb52-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y  =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb52-15"><a href="#cb52-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text   =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),   <span class="co"># protein facet labels</span></span>
<span id="cb52-16"><a href="#cb52-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb52-17"><a href="#cb52-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="fu">c</span>(<span class="fl">0.15</span>, <span class="fl">0.15</span>),</span>
<span id="cb52-18"><a href="#cb52-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.box.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">"black"</span>))</span>
<span id="cb52-19"><a href="#cb52-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb52-20"><a href="#cb52-20" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(fig3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/figure-3-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption>Linear associations between baseline protein expression levels and participants’ change from baseline to 12-weeks in six-minute walk distance differentiated based on the randomly assigned intervention.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="matching-tree-analyses" class="level1">
<h1>Matching Tree Analyses</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a>ps.df <span class="ot">&lt;-</span> rprotblds</span>
<span id="cb53-2"><a href="#cb53-2" aria-hidden="true" tabindex="-1"></a>ps.df<span class="sc">$</span>trt <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(ps.df<span class="sc">$</span>intervention_1_control_0) <span class="sc">-</span> <span class="dv">1</span></span>
<span id="cb53-3"><a href="#cb53-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-4"><a href="#cb53-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Keep only subjects with complete data on key proteins and outcomes</span></span>
<span id="cb53-5"><a href="#cb53-5" aria-hidden="true" tabindex="-1"></a>ps.df <span class="ot">&lt;-</span> ps.df[<span class="fu">complete.cases</span>(ps.df[, <span class="fu">c</span>(<span class="st">"ALCAM"</span>, <span class="st">"Gal-4"</span>, <span class="st">"PECAM-1"</span>,</span>
<span id="cb53-6"><a href="#cb53-6" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">"GP6"</span>, <span class="st">"ST2"</span>,</span>
<span id="cb53-7"><a href="#cb53-7" aria-hidden="true" tabindex="-1"></a>                                        <span class="st">'bl_smw'</span>, <span class="st">'fu_smw'</span>, <span class="st">'bl_sppb'</span>, <span class="st">'fu_sppb'</span>)]), ]</span>
<span id="cb53-8"><a href="#cb53-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-9"><a href="#cb53-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit propensity score model using logistic regression</span></span>
<span id="cb53-10"><a href="#cb53-10" aria-hidden="true" tabindex="-1"></a>ppty <span class="ot">&lt;-</span> <span class="fu">glm</span>(</span>
<span id="cb53-11"><a href="#cb53-11" aria-hidden="true" tabindex="-1"></a>  trt <span class="sc">~</span> age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat,</span>
<span id="cb53-12"><a href="#cb53-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">family =</span> <span class="fu">binomial</span>(<span class="at">link =</span> <span class="st">"logit"</span>),</span>
<span id="cb53-13"><a href="#cb53-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> ps.df</span>
<span id="cb53-14"><a href="#cb53-14" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb53-15"><a href="#cb53-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-16"><a href="#cb53-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Generate propensity scores</span></span>
<span id="cb53-17"><a href="#cb53-17" aria-hidden="true" tabindex="-1"></a>prop.score <span class="ot">&lt;-</span> <span class="fu">predict.glm</span>(ppty, <span class="at">type =</span> <span class="st">"response"</span>, <span class="at">na.action =</span> na.exclude)</span>
<span id="cb53-18"><a href="#cb53-18" aria-hidden="true" tabindex="-1"></a>ps.df <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">cbind</span>(ps.df, prop.score))</span>
<span id="cb53-19"><a href="#cb53-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-20"><a href="#cb53-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Create matching distance matrix based on propensity scores</span></span>
<span id="cb53-21"><a href="#cb53-21" aria-hidden="true" tabindex="-1"></a>ppty.distance <span class="ot">&lt;-</span> <span class="fu">match_on</span>(ppty)</span>
<span id="cb53-22"><a href="#cb53-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-23"><a href="#cb53-23" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Propensity Score Matching Summary</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Propensity Score Matching Summary</code></pre>
</div>
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"==================================</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>==================================</code></pre>
</div>
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Total subjects with complete data:"</span>, <span class="fu">nrow</span>(ps.df), <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Total subjects with complete data: 181 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Treatment:"</span>, <span class="fu">sum</span>(ps.df<span class="sc">$</span>trt <span class="sc">==</span> <span class="dv">1</span>), <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Treatment: 99 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb61-1"><a href="#cb61-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Control:"</span>, <span class="fu">sum</span>(ps.df<span class="sc">$</span>trt <span class="sc">==</span> <span class="dv">0</span>), <span class="st">"</span><span class="sc">\n\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Control: 82 </code></pre>
</div>
</div>
<p>These functions are adapted from Zhang et al.&nbsp;(2021) “Matching Trees for Treatment Effect Heterogeneity” https://doi.org/10.1016/j.csda.2021.107188.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Function to get parent node in tree structure</span></span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a>parent <span class="ot">&lt;-</span> <span class="cf">function</span>(x) {</span>
<span id="cb63-3"><a href="#cb63-3" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (x[<span class="dv">1</span>] <span class="sc">!=</span> <span class="dv">1</span>) {</span>
<span id="cb63-4"><a href="#cb63-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">c</span>(<span class="fu">Recall</span>(<span class="cf">if</span> (x <span class="sc">%%</span> <span class="dv">2</span> <span class="sc">==</span> <span class="dv">0</span>L) x <span class="sc">/</span> <span class="dv">2</span> <span class="cf">else</span> (x <span class="sc">-</span> <span class="dv">1</span>) <span class="sc">/</span> <span class="dv">2</span>), x)</span>
<span id="cb63-5"><a href="#cb63-5" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb63-6"><a href="#cb63-6" aria-hidden="true" tabindex="-1"></a>    x</span>
<span id="cb63-7"><a href="#cb63-7" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb63-8"><a href="#cb63-8" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb63-9"><a href="#cb63-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb63-10"><a href="#cb63-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Function to prune tree based on treatment effect overlap</span></span>
<span id="cb63-11"><a href="#cb63-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Prunes terminal nodes if confidence intervals for treatment effects overlap</span></span>
<span id="cb63-12"><a href="#cb63-12" aria-hidden="true" tabindex="-1"></a>jesse.tree.select <span class="ot">&lt;-</span> <span class="cf">function</span>(input, set1.caliper.tree) {</span>
<span id="cb63-13"><a href="#cb63-13" aria-hidden="true" tabindex="-1"></a>  fit.overlap <span class="ot">&lt;-</span> <span class="cn">TRUE</span></span>
<span id="cb63-14"><a href="#cb63-14" aria-hidden="true" tabindex="-1"></a>  ptree1 <span class="ot">&lt;-</span> input</span>
<span id="cb63-15"><a href="#cb63-15" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb63-16"><a href="#cb63-16" aria-hidden="true" tabindex="-1"></a>  <span class="cf">while</span> (fit.overlap <span class="sc">==</span> <span class="cn">TRUE</span>) {</span>
<span id="cb63-17"><a href="#cb63-17" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">dim</span>(ptree1<span class="sc">$</span>frame)[<span class="dv">1</span>] <span class="sc">==</span> <span class="dv">1</span>) <span class="cf">break</span></span>
<span id="cb63-18"><a href="#cb63-18" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-19"><a href="#cb63-19" aria-hidden="true" tabindex="-1"></a>    rownames <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(<span class="fu">rownames</span>(ptree1<span class="sc">$</span>frame))</span>
<span id="cb63-20"><a href="#cb63-20" aria-hidden="true" tabindex="-1"></a>    size <span class="ot">&lt;-</span> <span class="fu">dim</span>(ptree1<span class="sc">$</span>frame)[<span class="dv">1</span>]</span>
<span id="cb63-21"><a href="#cb63-21" aria-hidden="true" tabindex="-1"></a>    rownamesmax <span class="ot">&lt;-</span> rownames[<span class="fu">which.max</span>(rownames)]</span>
<span id="cb63-22"><a href="#cb63-22" aria-hidden="true" tabindex="-1"></a>    rownamessecond <span class="ot">&lt;-</span> rownames[<span class="fu">which</span>(rownames <span class="sc">==</span> <span class="fu">sort</span>(<span class="fu">unique</span>(rownames), <span class="at">partial =</span> size <span class="sc">-</span> <span class="dv">1</span>)[size <span class="sc">-</span> <span class="dv">1</span>])]</span>
<span id="cb63-23"><a href="#cb63-23" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-24"><a href="#cb63-24" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Get data for terminal nodes</span></span>
<span id="cb63-25"><a href="#cb63-25" aria-hidden="true" tabindex="-1"></a>    subset1 <span class="ot">&lt;-</span> set1.caliper.tree[ptree1<span class="sc">$</span>where <span class="sc">==</span> <span class="fu">which</span>(rownames <span class="sc">==</span> rownamesmax), ]</span>
<span id="cb63-26"><a href="#cb63-26" aria-hidden="true" tabindex="-1"></a>    subset2 <span class="ot">&lt;-</span> set1.caliper.tree[ptree1<span class="sc">$</span>where <span class="sc">==</span> <span class="fu">which</span>(rownames <span class="sc">==</span> rownamessecond), ]</span>
<span id="cb63-27"><a href="#cb63-27" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-28"><a href="#cb63-28" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Create dummy variable for node membership</span></span>
<span id="cb63-29"><a href="#cb63-29" aria-hidden="true" tabindex="-1"></a>    dummy <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rep</span>(<span class="st">"1"</span>, <span class="fu">dim</span>(subset1)[<span class="dv">1</span>]), <span class="fu">rep</span>(<span class="st">"2"</span>, <span class="fu">dim</span>(subset2)[<span class="dv">1</span>]))</span>
<span id="cb63-30"><a href="#cb63-30" aria-hidden="true" tabindex="-1"></a>    subsetf <span class="ot">&lt;-</span> <span class="fu">cbind</span>(<span class="fu">rbind</span>(subset1, subset2), dummy)</span>
<span id="cb63-31"><a href="#cb63-31" aria-hidden="true" tabindex="-1"></a>    subsetf<span class="sc">$</span>dummy <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(subsetf<span class="sc">$</span>dummy)</span>
<span id="cb63-32"><a href="#cb63-32" aria-hidden="true" tabindex="-1"></a>    subsetf<span class="sc">$</span>trt <span class="ot">&lt;-</span> <span class="fu">as.factor</span>(subsetf<span class="sc">$</span>trt)</span>
<span id="cb63-33"><a href="#cb63-33" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-34"><a href="#cb63-34" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Test treatment effect differences between nodes</span></span>
<span id="cb63-35"><a href="#cb63-35" aria-hidden="true" tabindex="-1"></a>    ls.lm <span class="ot">&lt;-</span> <span class="fu">lm</span>(outcome <span class="sc">~</span> dummy <span class="sc">*</span> trt, <span class="at">data =</span> subsetf)</span>
<span id="cb63-36"><a href="#cb63-36" aria-hidden="true" tabindex="-1"></a>    ls.list <span class="ot">&lt;-</span> <span class="fu">lsmeans</span>(ls.lm, <span class="fu">list</span>(pairwise <span class="sc">~</span> dummy <span class="sc">|</span> trt, pairwise <span class="sc">~</span> trt <span class="sc">|</span> dummy))</span>
<span id="cb63-37"><a href="#cb63-37" aria-hidden="true" tabindex="-1"></a>    out1 <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(<span class="fu">summary</span>(ls.list[[<span class="dv">4</span>]]))</span>
<span id="cb63-38"><a href="#cb63-38" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-39"><a href="#cb63-39" aria-hidden="true" tabindex="-1"></a>    q.value <span class="ot">&lt;-</span> <span class="fl">1.96</span>  <span class="co"># 95% confidence interval</span></span>
<span id="cb63-40"><a href="#cb63-40" aria-hidden="true" tabindex="-1"></a>    CI1 <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb63-41"><a href="#cb63-41" aria-hidden="true" tabindex="-1"></a>      <span class="fu">as.numeric</span>(out1[<span class="dv">1</span>, <span class="dv">3</span>]) <span class="sc">-</span> q.value <span class="sc">*</span> <span class="fu">as.numeric</span>(out1[<span class="dv">1</span>, <span class="dv">4</span>]),</span>
<span id="cb63-42"><a href="#cb63-42" aria-hidden="true" tabindex="-1"></a>      <span class="fu">as.numeric</span>(out1[<span class="dv">1</span>, <span class="dv">3</span>]) <span class="sc">+</span> q.value <span class="sc">*</span> <span class="fu">as.numeric</span>(out1[<span class="dv">1</span>, <span class="dv">4</span>])</span>
<span id="cb63-43"><a href="#cb63-43" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb63-44"><a href="#cb63-44" aria-hidden="true" tabindex="-1"></a>    CI2 <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb63-45"><a href="#cb63-45" aria-hidden="true" tabindex="-1"></a>      <span class="fu">as.numeric</span>(out1[<span class="dv">2</span>, <span class="dv">3</span>]) <span class="sc">-</span> q.value <span class="sc">*</span> <span class="fu">as.numeric</span>(out1[<span class="dv">2</span>, <span class="dv">4</span>]),</span>
<span id="cb63-46"><a href="#cb63-46" aria-hidden="true" tabindex="-1"></a>      <span class="fu">as.numeric</span>(out1[<span class="dv">2</span>, <span class="dv">3</span>]) <span class="sc">+</span> q.value <span class="sc">*</span> <span class="fu">as.numeric</span>(out1[<span class="dv">2</span>, <span class="dv">4</span>])</span>
<span id="cb63-47"><a href="#cb63-47" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb63-48"><a href="#cb63-48" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-49"><a href="#cb63-49" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Check for overlap in confidence intervals</span></span>
<span id="cb63-50"><a href="#cb63-50" aria-hidden="true" tabindex="-1"></a>    fit.overlap <span class="ot">&lt;-</span> <span class="fu">max</span>(CI1[<span class="dv">1</span>], CI2[<span class="dv">1</span>]) <span class="sc">&lt;</span> <span class="fu">min</span>(CI1[<span class="dv">2</span>], CI2[<span class="dv">2</span>])</span>
<span id="cb63-51"><a href="#cb63-51" aria-hidden="true" tabindex="-1"></a>    fit.overlap[<span class="fu">is.na</span>(fit.overlap)] <span class="ot">&lt;-</span> <span class="cn">FALSE</span></span>
<span id="cb63-52"><a href="#cb63-52" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-53"><a href="#cb63-53" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (fit.overlap <span class="sc">==</span> <span class="cn">FALSE</span>) <span class="cf">break</span></span>
<span id="cb63-54"><a href="#cb63-54" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb63-55"><a href="#cb63-55" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Prune the node</span></span>
<span id="cb63-56"><a href="#cb63-56" aria-hidden="true" tabindex="-1"></a>    ptree1 <span class="ot">&lt;-</span> <span class="fu">snip.rpart</span>(ptree1, <span class="at">toss =</span> <span class="fu">tail</span>(<span class="fu">head</span>(<span class="fu">parent</span>(rownamesmax), <span class="sc">-</span><span class="dv">1</span>), <span class="at">n =</span> <span class="dv">1</span>))</span>
<span id="cb63-57"><a href="#cb63-57" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb63-58"><a href="#cb63-58" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb63-59"><a href="#cb63-59" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">dim</span>(ptree1<span class="sc">$</span>frame)[<span class="dv">1</span>] <span class="sc">&gt;</span> <span class="dv">1</span>) {</span>
<span id="cb63-60"><a href="#cb63-60" aria-hidden="true" tabindex="-1"></a>    <span class="fu">return</span>(ptree1)</span>
<span id="cb63-61"><a href="#cb63-61" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb63-62"><a href="#cb63-62" aria-hidden="true" tabindex="-1"></a>    <span class="fu">return</span>(input)</span>
<span id="cb63-63"><a href="#cb63-63" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb63-64"><a href="#cb63-64" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="sppb-3" class="level3">
<h3 class="anchored" data-anchor-id="sppb-3">SPPB</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Separate treatment and control groups</span></span>
<span id="cb64-2"><a href="#cb64-2" aria-hidden="true" tabindex="-1"></a>set.trt <span class="ot">&lt;-</span> ps.df[ps.df[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb64-3"><a href="#cb64-3" aria-hidden="true" tabindex="-1"></a>set.cont <span class="ot">&lt;-</span> ps.df[ps.df[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">0</span>, ]</span>
<span id="cb64-4"><a href="#cb64-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-5"><a href="#cb64-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Perform 1:1 matching with caliper on protein values</span></span>
<span id="cb64-6"><a href="#cb64-6" aria-hidden="true" tabindex="-1"></a>match.set <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb64-7"><a href="#cb64-7" aria-hidden="true" tabindex="-1"></a>calp.sd <span class="ot">&lt;-</span> <span class="dv">1</span>  <span class="co"># Caliper width: 1 SD of protein values</span></span>
<span id="cb64-8"><a href="#cb64-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-9"><a href="#cb64-9" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">dim</span>(set.trt)[<span class="dv">1</span>]) {</span>
<span id="cb64-10"><a href="#cb64-10" aria-hidden="true" tabindex="-1"></a>  pair.set1 <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb64-11"><a href="#cb64-11" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb64-12"><a href="#cb64-12" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Check if control subject is within caliper on both GP6 and ST2</span></span>
<span id="cb64-13"><a href="#cb64-13" aria-hidden="true" tabindex="-1"></a>  var.select1 <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(</span>
<span id="cb64-14"><a href="#cb64-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">abs</span>(set.cont<span class="sc">$</span>GP6 <span class="sc">-</span> set.trt<span class="sc">$</span>GP6[i]) <span class="sc">&lt;</span> calp.sd <span class="sc">*</span> <span class="fu">sd</span>(set.trt<span class="sc">$</span>GP6),</span>
<span id="cb64-15"><a href="#cb64-15" aria-hidden="true" tabindex="-1"></a>    <span class="dv">1</span>, <span class="dv">0</span></span>
<span id="cb64-16"><a href="#cb64-16" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb64-17"><a href="#cb64-17" aria-hidden="true" tabindex="-1"></a>  var.select2 <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(</span>
<span id="cb64-18"><a href="#cb64-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">abs</span>(set.cont<span class="sc">$</span>ST2 <span class="sc">-</span> set.trt<span class="sc">$</span>ST2[i]) <span class="sc">&lt;</span> calp.sd <span class="sc">*</span> <span class="fu">sd</span>(set.trt<span class="sc">$</span>ST2),</span>
<span id="cb64-19"><a href="#cb64-19" aria-hidden="true" tabindex="-1"></a>    <span class="dv">1</span>, <span class="dv">0</span></span>
<span id="cb64-20"><a href="#cb64-20" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb64-21"><a href="#cb64-21" aria-hidden="true" tabindex="-1"></a>  var.selectf <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(var.select1 <span class="sc">+</span> var.select2 <span class="sc">==</span> <span class="dv">2</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb64-22"><a href="#cb64-22" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb64-23"><a href="#cb64-23" aria-hidden="true" tabindex="-1"></a>  id <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(var.selectf))</span>
<span id="cb64-24"><a href="#cb64-24" aria-hidden="true" tabindex="-1"></a>  lenset.cont2 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">cbind</span>(id, set.cont, var.selectf))</span>
<span id="cb64-25"><a href="#cb64-25" aria-hidden="true" tabindex="-1"></a>  lenset.cont3 <span class="ot">&lt;-</span> lenset.cont2[lenset.cont2[, <span class="st">"var.selectf"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb64-26"><a href="#cb64-26" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb64-27"><a href="#cb64-27" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Match to nearest propensity score if within caliper</span></span>
<span id="cb64-28"><a href="#cb64-28" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">dim</span>(lenset.cont3)[<span class="dv">1</span>] <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb64-29"><a href="#cb64-29" aria-hidden="true" tabindex="-1"></a>    cont.index <span class="ot">&lt;-</span> <span class="fu">which.min</span>(<span class="fu">abs</span>(lenset.cont3<span class="sc">$</span>prop.score <span class="sc">-</span> set.trt<span class="sc">$</span>prop.score[i]))</span>
<span id="cb64-30"><a href="#cb64-30" aria-hidden="true" tabindex="-1"></a>    pair.set <span class="ot">&lt;-</span> <span class="fu">rbind</span>(</span>
<span id="cb64-31"><a href="#cb64-31" aria-hidden="true" tabindex="-1"></a>      set.trt[i, ],</span>
<span id="cb64-32"><a href="#cb64-32" aria-hidden="true" tabindex="-1"></a>      lenset.cont3[cont.index, <span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="sc">-</span><span class="fu">dim</span>(lenset.cont3)[<span class="dv">2</span>])]</span>
<span id="cb64-33"><a href="#cb64-33" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb64-34"><a href="#cb64-34" aria-hidden="true" tabindex="-1"></a>    pairin <span class="ot">&lt;-</span> <span class="fu">rep</span>(i, <span class="dv">2</span>)</span>
<span id="cb64-35"><a href="#cb64-35" aria-hidden="true" tabindex="-1"></a>    pair.set1 <span class="ot">&lt;-</span> <span class="fu">cbind</span>(pair.set, pairin)</span>
<span id="cb64-36"><a href="#cb64-36" aria-hidden="true" tabindex="-1"></a>    set.cont <span class="ot">&lt;-</span> set.cont[<span class="sc">!</span>set.cont<span class="sc">$</span>prop.score <span class="sc">==</span> lenset.cont3<span class="sc">$</span>prop.score[cont.index], ]</span>
<span id="cb64-37"><a href="#cb64-37" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb64-38"><a href="#cb64-38" aria-hidden="true" tabindex="-1"></a>    pair.set1 <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb64-39"><a href="#cb64-39" aria-hidden="true" tabindex="-1"></a>    set.cont <span class="ot">&lt;-</span> set.cont</span>
<span id="cb64-40"><a href="#cb64-40" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb64-41"><a href="#cb64-41" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb64-42"><a href="#cb64-42" aria-hidden="true" tabindex="-1"></a>  match.set <span class="ot">&lt;-</span> <span class="fu">rbind</span>(match.set, pair.set1)</span>
<span id="cb64-43"><a href="#cb64-43" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb64-44"><a href="#cb64-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-45"><a href="#cb64-45" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate treatment effect (difference in outcome change)</span></span>
<span id="cb64-46"><a href="#cb64-46" aria-hidden="true" tabindex="-1"></a>treat.set <span class="ot">&lt;-</span> match.set[match.set[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb64-47"><a href="#cb64-47" aria-hidden="true" tabindex="-1"></a>control.set <span class="ot">&lt;-</span> match.set[match.set[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">0</span>, ]</span>
<span id="cb64-48"><a href="#cb64-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-49"><a href="#cb64-49" aria-hidden="true" tabindex="-1"></a>outcome.diff <span class="ot">&lt;-</span> (treat.set<span class="sc">$</span>fu_sppb <span class="sc">-</span> treat.set<span class="sc">$</span>bl_sppb) <span class="sc">-</span></span>
<span id="cb64-50"><a href="#cb64-50" aria-hidden="true" tabindex="-1"></a>  (control.set<span class="sc">$</span>fu_sppb <span class="sc">-</span> control.set<span class="sc">$</span>bl_sppb)</span>
<span id="cb64-51"><a href="#cb64-51" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-52"><a href="#cb64-52" aria-hidden="true" tabindex="-1"></a>set.total <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">rbind</span>(</span>
<span id="cb64-53"><a href="#cb64-53" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(treat.set, outcome.diff),</span>
<span id="cb64-54"><a href="#cb64-54" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(control.set, outcome.diff)</span>
<span id="cb64-55"><a href="#cb64-55" aria-hidden="true" tabindex="-1"></a>))</span>
<span id="cb64-56"><a href="#cb64-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-57"><a href="#cb64-57" aria-hidden="true" tabindex="-1"></a>matchedset.sppb <span class="ot">&lt;-</span> set.total  <span class="co"># Store for later use</span></span>
<span id="cb64-58"><a href="#cb64-58" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-59"><a href="#cb64-59" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"SPPB Matching Summary</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>SPPB Matching Summary</code></pre>
</div>
<div class="sourceCode cell-code" id="cb66"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb66-1"><a href="#cb66-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"====================</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>====================</code></pre>
</div>
<div class="sourceCode cell-code" id="cb68"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb68-1"><a href="#cb68-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Matched pairs:"</span>, <span class="fu">nrow</span>(treat.set), <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Matched pairs: 81 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb70"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb70-1"><a href="#cb70-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Total matched subjects:"</span>, <span class="fu">nrow</span>(set.total), <span class="st">"</span><span class="sc">\n\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Total matched subjects: 162 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb72"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb72-1"><a href="#cb72-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Build matching tree on proteins GP6 and ST2</span></span>
<span id="cb72-2"><a href="#cb72-2" aria-hidden="true" tabindex="-1"></a>tree.rpart <span class="ot">&lt;-</span> <span class="fu">rpart</span>(</span>
<span id="cb72-3"><a href="#cb72-3" aria-hidden="true" tabindex="-1"></a>  outcome.diff <span class="sc">~</span> GP6 <span class="sc">+</span> ST2,</span>
<span id="cb72-4"><a href="#cb72-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">method =</span> <span class="st">"anova"</span>,</span>
<span id="cb72-5"><a href="#cb72-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> set.total,</span>
<span id="cb72-6"><a href="#cb72-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">control =</span> <span class="fu">rpart.control</span>(<span class="at">minbucket =</span> <span class="dv">15</span>, <span class="at">maxdepth =</span> <span class="dv">2</span>)</span>
<span id="cb72-7"><a href="#cb72-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb72-8"><a href="#cb72-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-9"><a href="#cb72-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Add outcome variable and prune tree</span></span>
<span id="cb72-10"><a href="#cb72-10" aria-hidden="true" tabindex="-1"></a>set.total<span class="sc">$</span>outcome <span class="ot">&lt;-</span> set.total<span class="sc">$</span>fu_sppb <span class="sc">-</span> set.total<span class="sc">$</span>bl_sppb</span>
<span id="cb72-11"><a href="#cb72-11" aria-hidden="true" tabindex="-1"></a>tree.all <span class="ot">&lt;-</span> <span class="fu">jesse.tree.select</span>(tree.rpart, set.total)</span>
<span id="cb72-12"><a href="#cb72-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-13"><a href="#cb72-13" aria-hidden="true" tabindex="-1"></a><span class="fu">rpart.plot</span>(tree.all, <span class="at">box.palette =</span> <span class="st">'Reds'</span>, <span class="at">extra =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Extract treatment effects by tree node and get confidence intervals</span></span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>set.total<span class="sc">$</span>sppb_node <span class="ot">&lt;-</span> <span class="fu">rpart.predict.leaves</span>(tree.all, set.total, <span class="at">type =</span> <span class="st">"where"</span>)</span>
<span id="cb73-3"><a href="#cb73-3" aria-hidden="true" tabindex="-1"></a>modscore.sppb <span class="ot">&lt;-</span> <span class="fu">lm</span>(<span class="at">data =</span> set.total, <span class="at">formula =</span> outcome.diff <span class="sc">~</span> <span class="dv">0</span> <span class="sc">+</span> <span class="fu">as.factor</span>(sppb_node))</span>
<span id="cb73-4"><a href="#cb73-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-5"><a href="#cb73-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Summary of results</span></span>
<span id="cb73-6"><a href="#cb73-6" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"SPPB Matching Tree - Treatment Effects by Node</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>SPPB Matching Tree - Treatment Effects by Node</code></pre>
</div>
<div class="sourceCode cell-code" id="cb75"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb75-1"><a href="#cb75-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"==============================================</span><span class="sc">\n\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>==============================================</code></pre>
</div>
<div class="sourceCode cell-code" id="cb77"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">confint</span>(modscore.sppb))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>                           2.5 %    97.5 %
as.factor(sppb_node)3 -1.8991465 0.2420037
as.factor(sppb_node)4  0.2071578 2.7928422
as.factor(sppb_node)5  1.4633097 2.7114475</code></pre>
</div>
<div class="sourceCode cell-code" id="cb79"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="sourceCode cell-code" id="cb80"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(modscore.sppb))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = outcome.diff ~ 0 + as.factor(sppb_node), data = set.total)

Residuals:
    Min      1Q  Median      3Q     Max 
-7.0874 -2.1504 -0.0874  1.9126  8.9126 

Coefficients:
                      Estimate Std. Error t value Pr(&gt;|t|)    
as.factor(sppb_node)3  -0.8286     0.5421  -1.529   0.1284    
as.factor(sppb_node)4   1.5000     0.6546   2.291   0.0232 *  
as.factor(sppb_node)5   2.0874     0.3160   6.606 5.64e-10 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 3.207 on 159 degrees of freedom
Multiple R-squared:  0.2437,    Adjusted R-squared:  0.2294 
F-statistic: 17.08 on 3 and 159 DF,  p-value: 1.158e-09</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb82"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ---- K-Fold Cross-Validation Stability Analysis for SPPB ----</span></span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">67</span>)</span>
<span id="cb82-3"><a href="#cb82-3" aria-hidden="true" tabindex="-1"></a>nfolds <span class="ot">&lt;-</span> <span class="dv">10</span></span>
<span id="cb82-4"><a href="#cb82-4" aria-hidden="true" tabindex="-1"></a>folds <span class="ot">&lt;-</span> <span class="fu">cut</span>(<span class="fu">seq</span>(<span class="dv">1</span>, <span class="fu">nrow</span>(treat.set)), <span class="at">breaks =</span> nfolds, <span class="at">labels =</span> <span class="cn">FALSE</span>)</span>
<span id="cb82-5"><a href="#cb82-5" aria-hidden="true" tabindex="-1"></a>folds <span class="ot">&lt;-</span> folds[<span class="fu">sample</span>(<span class="fu">nrow</span>(treat.set))]</span>
<span id="cb82-6"><a href="#cb82-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-7"><a href="#cb82-7" aria-hidden="true" tabindex="-1"></a>maxdepth <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span></span>
<span id="cb82-8"><a href="#cb82-8" aria-hidden="true" tabindex="-1"></a>minbucket <span class="ot">&lt;-</span> <span class="fu">seq</span>(<span class="dv">5</span>, <span class="dv">30</span>, <span class="dv">5</span>)</span>
<span id="cb82-9"><a href="#cb82-9" aria-hidden="true" tabindex="-1"></a>bst <span class="ot">&lt;-</span> <span class="dv">1000</span></span>
<span id="cb82-10"><a href="#cb82-10" aria-hidden="true" tabindex="-1"></a>bst.stg <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">0</span>)</span>
<span id="cb82-11"><a href="#cb82-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-12"><a href="#cb82-12" aria-hidden="true" tabindex="-1"></a>cvres.df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">maxdepth =</span> <span class="cn">NA</span>, <span class="at">minbucket =</span> <span class="cn">NA</span>, <span class="at">testrmse.dt =</span> <span class="cn">NA</span>, <span class="at">testrmse.rf =</span> <span class="cn">NA</span>)</span>
<span id="cb82-13"><a href="#cb82-13" aria-hidden="true" tabindex="-1"></a>idx <span class="ot">&lt;-</span> <span class="dv">0</span></span>
<span id="cb82-14"><a href="#cb82-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-15"><a href="#cb82-15" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (j <span class="cf">in</span> maxdepth) {</span>
<span id="cb82-16"><a href="#cb82-16" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (k <span class="cf">in</span> minbucket) {</span>
<span id="cb82-17"><a href="#cb82-17" aria-hidden="true" tabindex="-1"></a>    idx <span class="ot">&lt;-</span> idx <span class="sc">+</span> <span class="dv">1</span></span>
<span id="cb82-18"><a href="#cb82-18" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb82-19"><a href="#cb82-19" aria-hidden="true" tabindex="-1"></a>    tree<span class="fl">.2</span> <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-20"><a href="#cb82-20" aria-hidden="true" tabindex="-1"></a>    tree.leaves <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-21"><a href="#cb82-21" aria-hidden="true" tabindex="-1"></a>    tree.mod.train <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-22"><a href="#cb82-22" aria-hidden="true" tabindex="-1"></a>    tree.test.rmse <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-23"><a href="#cb82-23" aria-hidden="true" tabindex="-1"></a>    rf.mod.train <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-24"><a href="#cb82-24" aria-hidden="true" tabindex="-1"></a>    rf.test.rmse <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb82-25"><a href="#cb82-25" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb82-26"><a href="#cb82-26" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>nfolds) {</span>
<span id="cb82-27"><a href="#cb82-27" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Identify training and test folds</span></span>
<span id="cb82-28"><a href="#cb82-28" aria-hidden="true" tabindex="-1"></a>      ids <span class="ot">&lt;-</span> treat.set<span class="sc">$</span>subject_id[folds <span class="sc">!=</span> i]</span>
<span id="cb82-29"><a href="#cb82-29" aria-hidden="true" tabindex="-1"></a>      pairs.train <span class="ot">&lt;-</span> set.total<span class="sc">$</span>pairin[set.total<span class="sc">$</span>subject_id <span class="sc">%in%</span> ids]</span>
<span id="cb82-30"><a href="#cb82-30" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-31"><a href="#cb82-31" aria-hidden="true" tabindex="-1"></a>      ids <span class="ot">&lt;-</span> treat.set<span class="sc">$</span>subject_id[folds <span class="sc">==</span> i]</span>
<span id="cb82-32"><a href="#cb82-32" aria-hidden="true" tabindex="-1"></a>      pairs.test <span class="ot">&lt;-</span> set.total<span class="sc">$</span>pairin[set.total<span class="sc">$</span>subject_id <span class="sc">%in%</span> ids]</span>
<span id="cb82-33"><a href="#cb82-33" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-34"><a href="#cb82-34" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Fit tree on training data</span></span>
<span id="cb82-35"><a href="#cb82-35" aria-hidden="true" tabindex="-1"></a>      tmp <span class="ot">&lt;-</span> <span class="fu">rpart</span>(</span>
<span id="cb82-36"><a href="#cb82-36" aria-hidden="true" tabindex="-1"></a>        outcome.diff <span class="sc">~</span> GP6 <span class="sc">+</span> ST2,</span>
<span id="cb82-37"><a href="#cb82-37" aria-hidden="true" tabindex="-1"></a>        <span class="at">method =</span> <span class="st">"anova"</span>,</span>
<span id="cb82-38"><a href="#cb82-38" aria-hidden="true" tabindex="-1"></a>        <span class="at">data =</span> set.total[set.total<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, ],</span>
<span id="cb82-39"><a href="#cb82-39" aria-hidden="true" tabindex="-1"></a>        <span class="at">control =</span> <span class="fu">rpart.control</span>(<span class="at">maxdepth =</span> j, <span class="at">minbucket =</span> k)</span>
<span id="cb82-40"><a href="#cb82-40" aria-hidden="true" tabindex="-1"></a>      )</span>
<span id="cb82-41"><a href="#cb82-41" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-42"><a href="#cb82-42" aria-hidden="true" tabindex="-1"></a>      tree<span class="fl">.2</span>[[i]] <span class="ot">&lt;-</span> <span class="fu">jesse.tree.select</span>(tmp, set.total)</span>
<span id="cb82-43"><a href="#cb82-43" aria-hidden="true" tabindex="-1"></a>      tree.leaves[[i]] <span class="ot">&lt;-</span> <span class="fu">rpart.predict.leaves</span>(tree<span class="fl">.2</span>[[i]], set.total, <span class="at">type =</span> <span class="st">"where"</span>)</span>
<span id="cb82-44"><a href="#cb82-44" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-45"><a href="#cb82-45" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Get predictions on test set</span></span>
<span id="cb82-46"><a href="#cb82-46" aria-hidden="true" tabindex="-1"></a>      df <span class="ot">&lt;-</span> <span class="fu">cbind</span>(set.total, tree.leaves[[i]])</span>
<span id="cb82-47"><a href="#cb82-47" aria-hidden="true" tabindex="-1"></a>      <span class="fu">colnames</span>(df)[<span class="fu">ncol</span>(df)] <span class="ot">&lt;-</span> <span class="st">"tree_leaves_temp"</span></span>
<span id="cb82-48"><a href="#cb82-48" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-49"><a href="#cb82-49" aria-hidden="true" tabindex="-1"></a>      tree.mod.train[[i]] <span class="ot">&lt;-</span> <span class="fu">lm</span>(</span>
<span id="cb82-50"><a href="#cb82-50" aria-hidden="true" tabindex="-1"></a>        outcome.diff <span class="sc">~</span> <span class="dv">0</span> <span class="sc">+</span> <span class="fu">as.factor</span>(tree_leaves_temp),</span>
<span id="cb82-51"><a href="#cb82-51" aria-hidden="true" tabindex="-1"></a>        <span class="at">data =</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, ]</span>
<span id="cb82-52"><a href="#cb82-52" aria-hidden="true" tabindex="-1"></a>      )</span>
<span id="cb82-53"><a href="#cb82-53" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-54"><a href="#cb82-54" aria-hidden="true" tabindex="-1"></a>      tree.test.rmse[[i]] <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(<span class="fu">mean</span>((</span>
<span id="cb82-55"><a href="#cb82-55" aria-hidden="true" tabindex="-1"></a>        df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, <span class="st">'outcome.diff'</span>] <span class="sc">-</span></span>
<span id="cb82-56"><a href="#cb82-56" aria-hidden="true" tabindex="-1"></a>          <span class="fu">predict</span>(tree.mod.train[[i]], df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, ])</span>
<span id="cb82-57"><a href="#cb82-57" aria-hidden="true" tabindex="-1"></a>      )<span class="sc">^</span><span class="dv">2</span>))</span>
<span id="cb82-58"><a href="#cb82-58" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-59"><a href="#cb82-59" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Fit random forest on training data</span></span>
<span id="cb82-60"><a href="#cb82-60" aria-hidden="true" tabindex="-1"></a>      rf.df.train <span class="ot">&lt;-</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, <span class="fu">c</span>(<span class="st">"outcome.diff"</span>, <span class="st">"GP6"</span>, <span class="st">"ST2"</span>)]</span>
<span id="cb82-61"><a href="#cb82-61" aria-hidden="true" tabindex="-1"></a>      rf.df.test <span class="ot">&lt;-</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, <span class="fu">c</span>(<span class="st">"outcome.diff"</span>, <span class="st">"GP6"</span>, <span class="st">"ST2"</span>)]</span>
<span id="cb82-62"><a href="#cb82-62" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-63"><a href="#cb82-63" aria-hidden="true" tabindex="-1"></a>      rf.mod.train[[i]] <span class="ot">&lt;-</span> <span class="fu">randomForest</span>(outcome.diff <span class="sc">~</span> GP6 <span class="sc">+</span> ST2,</span>
<span id="cb82-64"><a href="#cb82-64" aria-hidden="true" tabindex="-1"></a>                                        <span class="at">data =</span> rf.df.train, <span class="at">ntree =</span> <span class="dv">500</span>, <span class="at">seed =</span> <span class="dv">42</span>)</span>
<span id="cb82-65"><a href="#cb82-65" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb82-66"><a href="#cb82-66" aria-hidden="true" tabindex="-1"></a>      rf.test.rmse[[i]] <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(<span class="fu">mean</span>((</span>
<span id="cb82-67"><a href="#cb82-67" aria-hidden="true" tabindex="-1"></a>        rf.df.test<span class="sc">$</span>outcome.diff <span class="sc">-</span></span>
<span id="cb82-68"><a href="#cb82-68" aria-hidden="true" tabindex="-1"></a>          <span class="fu">predict</span>(rf.mod.train[[i]], rf.df.test)</span>
<span id="cb82-69"><a href="#cb82-69" aria-hidden="true" tabindex="-1"></a>      )<span class="sc">^</span><span class="dv">2</span>))</span>
<span id="cb82-70"><a href="#cb82-70" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb82-71"><a href="#cb82-71" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb82-72"><a href="#cb82-72" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse)) <span class="sc">&lt;</span> bst) {</span>
<span id="cb82-73"><a href="#cb82-73" aria-hidden="true" tabindex="-1"></a>      bst <span class="ot">&lt;-</span> <span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse))</span>
<span id="cb82-74"><a href="#cb82-74" aria-hidden="true" tabindex="-1"></a>      bst.stg[<span class="dv">1</span>] <span class="ot">&lt;-</span> j</span>
<span id="cb82-75"><a href="#cb82-75" aria-hidden="true" tabindex="-1"></a>      bst.stg[<span class="dv">2</span>] <span class="ot">&lt;-</span> k</span>
<span id="cb82-76"><a href="#cb82-76" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb82-77"><a href="#cb82-77" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb82-78"><a href="#cb82-78" aria-hidden="true" tabindex="-1"></a>    cvres.df[idx, ] <span class="ot">&lt;-</span> <span class="fu">c</span>(j, k, <span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse)), <span class="fu">mean</span>(<span class="fu">unlist</span>(rf.test.rmse)))</span>
<span id="cb82-79"><a href="#cb82-79" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb82-80"><a href="#cb82-80" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb82-81"><a href="#cb82-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-82"><a href="#cb82-82" aria-hidden="true" tabindex="-1"></a><span class="co"># Format CV results</span></span>
<span id="cb82-83"><a href="#cb82-83" aria-hidden="true" tabindex="-1"></a>cvres.df<span class="sc">$</span>maxdepth <span class="ot">&lt;-</span> <span class="fu">factor</span>(cvres.df<span class="sc">$</span>maxdepth, <span class="at">levels =</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>)</span>
<span id="cb82-84"><a href="#cb82-84" aria-hidden="true" tabindex="-1"></a>cvres.df<span class="sc">$</span>minbucket <span class="ot">&lt;-</span> <span class="fu">as.integer</span>(cvres.df<span class="sc">$</span>minbucket)</span>
<span id="cb82-85"><a href="#cb82-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-86"><a href="#cb82-86" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate mean RF RMSE for reference line</span></span>
<span id="cb82-87"><a href="#cb82-87" aria-hidden="true" tabindex="-1"></a>rf.mean.sppb <span class="ot">&lt;-</span> <span class="fu">mean</span>(cvres.df<span class="sc">$</span>testrmse.rf, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb82-88"><a href="#cb82-88" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-89"><a href="#cb82-89" aria-hidden="true" tabindex="-1"></a>cvplot1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> cvres.df, <span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> minbucket, <span class="at">color =</span> maxdepth, <span class="at">fill =</span> maxdepth)) <span class="sc">+</span></span>
<span id="cb82-90"><a href="#cb82-90" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">y =</span> testrmse.dt), <span class="at">linewidth =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb82-91"><a href="#cb82-91" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">y =</span> testrmse.dt, <span class="at">shape =</span> maxdepth), <span class="at">size =</span> <span class="dv">3</span>) <span class="sc">+</span></span>
<span id="cb82-92"><a href="#cb82-92" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> rf.mean.sppb, <span class="at">color =</span> <span class="st">'black'</span>, <span class="at">linetype =</span> <span class="st">'dashed'</span>, <span class="at">linewidth =</span> <span class="fl">1.2</span>) <span class="sc">+</span></span>
<span id="cb82-93"><a href="#cb82-93" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, <span class="at">x =</span> <span class="dv">20</span>, <span class="at">y =</span> rf.mean.sppb <span class="sc">+</span> <span class="fl">0.08</span>, </span>
<span id="cb82-94"><a href="#cb82-94" aria-hidden="true" tabindex="-1"></a>           <span class="at">label =</span> <span class="fu">paste</span>(<span class="st">"Random Forest:"</span>, <span class="fu">round</span>(rf.mean.sppb, <span class="dv">3</span>)), </span>
<span id="cb82-95"><a href="#cb82-95" aria-hidden="true" tabindex="-1"></a>           <span class="at">size =</span> <span class="dv">4</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb82-96"><a href="#cb82-96" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">"Average Test RMSE"</span>) <span class="sc">+</span></span>
<span id="cb82-97"><a href="#cb82-97" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">"Minimum Bucket Size"</span>) <span class="sc">+</span></span>
<span id="cb82-98"><a href="#cb82-98" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Average Test RMSE: SPPB Outcome"</span>,</span>
<span id="cb82-99"><a href="#cb82-99" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Decision Trees vs. Random Forest"</span>) <span class="sc">+</span></span>
<span id="cb82-100"><a href="#cb82-100" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylim</span>(<span class="fu">c</span>(<span class="fl">3.0</span>, <span class="fl">3.8</span>)) <span class="sc">+</span></span>
<span id="cb82-101"><a href="#cb82-101" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">5</span>, <span class="dv">30</span>, <span class="dv">5</span>)) <span class="sc">+</span></span>
<span id="cb82-102"><a href="#cb82-102" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">name =</span> <span class="st">"Max Depth"</span>, <span class="at">values =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"#1f77b4"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"#ff7f0e"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"#2ca02c"</span>)) <span class="sc">+</span></span>
<span id="cb82-103"><a href="#cb82-103" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">name =</span> <span class="st">"Max Depth"</span>, <span class="at">values =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"#1f77b4"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"#ff7f0e"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"#2ca02c"</span>)) <span class="sc">+</span></span>
<span id="cb82-104"><a href="#cb82-104" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb82-105"><a href="#cb82-105" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb82-106"><a href="#cb82-106" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">face =</span> <span class="st">"bold"</span>, <span class="at">size =</span> <span class="dv">14</span>),</span>
<span id="cb82-107"><a href="#cb82-107" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">12</span>),</span>
<span id="cb82-108"><a href="#cb82-108" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb82-109"><a href="#cb82-109" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.major =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">"gray90"</span>),</span>
<span id="cb82-110"><a href="#cb82-110" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.minor =</span> <span class="fu">element_blank</span>()</span>
<span id="cb82-111"><a href="#cb82-111" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb82-112"><a href="#cb82-112" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(cvplot1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-22-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb83"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb83-1"><a href="#cb83-1" aria-hidden="true" tabindex="-1"></a>varlist<span class="ot">=</span><span class="fu">c</span>(<span class="st">"intervention_1_control_0"</span>,<span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,</span>
<span id="cb83-2"><a href="#cb83-2" aria-hidden="true" tabindex="-1"></a>          <span class="st">"hf_cat"</span>,<span class="st">"egfr"</span>,protlist)</span>
<span id="cb83-3"><a href="#cb83-3" aria-hidden="true" tabindex="-1"></a>tmpdf <span class="ot">=</span> <span class="fu">rbind</span>(<span class="fu">data.frame</span>(set.total[,varlist],<span class="at">dataset=</span><span class="st">"matched"</span>),</span>
<span id="cb83-4"><a href="#cb83-4" aria-hidden="true" tabindex="-1"></a>              <span class="fu">data.frame</span>(<span class="fu">setdiff</span>(ps.df[,varlist],set.total[,varlist]),<span class="at">dataset=</span><span class="st">"unmatched"</span>))</span>
<span id="cb83-5"><a href="#cb83-5" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>dataset<span class="ot">=</span><span class="fu">factor</span>(tmpdf<span class="sc">$</span>dataset,<span class="at">levels=</span><span class="fu">c</span>(<span class="st">"unmatched"</span>,<span class="st">"matched"</span>))</span>
<span id="cb83-6"><a href="#cb83-6" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">str_c</span>(tmpdf<span class="sc">$</span>intervention_1_control_0,tmpdf<span class="sc">$</span>dataset)</span>
<span id="cb83-7"><a href="#cb83-7" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">as.factor</span>(tmpdf<span class="sc">$</span>grp)</span>
<span id="cb83-8"><a href="#cb83-8" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">fct_collapse</span>(tmpdf<span class="sc">$</span>grp, <span class="at">Unmatched =</span> <span class="fu">c</span>(<span class="st">"0unmatched"</span>,<span class="st">"1unmatched"</span>), <span class="at">MatchedControl =</span> <span class="st">"0matched"</span>,<span class="at">MatchedIntervention =</span> <span class="st">"1matched"</span>)</span>
<span id="cb83-9"><a href="#cb83-9" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp <span class="ot">=</span> <span class="fu">factor</span>(tmpdf<span class="sc">$</span>grp,<span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"MatchedControl"</span>,<span class="st">"MatchedIntervention"</span>,<span class="st">"Unmatched"</span>))</span>
<span id="cb83-10"><a href="#cb83-10" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="ot">=</span>tmpdf[,<span class="sc">-</span><span class="fu">which</span>(<span class="fu">names</span>(tmpdf) <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"intervention_1_control_0"</span>,<span class="st">"dataset"</span>))]</span>
<span id="cb83-11"><a href="#cb83-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb83-12"><a href="#cb83-12" aria-hidden="true" tabindex="-1"></a><span class="fu">tbl_summary</span>(tmpdf,</span>
<span id="cb83-13"><a href="#cb83-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">by=</span>grp,</span>
<span id="cb83-14"><a href="#cb83-14" aria-hidden="true" tabindex="-1"></a>            <span class="at">type =</span> <span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="st">"continuous2"</span>,</span>
<span id="cb83-15"><a href="#cb83-15" aria-hidden="true" tabindex="-1"></a>            <span class="at">statistic =</span> <span class="fu">list</span>(<span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="fu">c</span>(<span class="st">"{mean} ({sd})"</span>,</span>
<span id="cb83-16"><a href="#cb83-16" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{median} ({p25}, {p75})"</span>,</span>
<span id="cb83-17"><a href="#cb83-17" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{min}, {max}"</span>),</span>
<span id="cb83-18"><a href="#cb83-18" aria-hidden="true" tabindex="-1"></a>                             <span class="fu">all_categorical</span>() <span class="sc">~</span> <span class="st">"{n} / {N} ({p}%)"</span></span>
<span id="cb83-19"><a href="#cb83-19" aria-hidden="true" tabindex="-1"></a>            ),<span class="at">label =</span> <span class="fu">list</span>(age <span class="sc">~</span> <span class="st">"Age"</span>,</span>
<span id="cb83-20"><a href="#cb83-20" aria-hidden="true" tabindex="-1"></a>                           sex <span class="sc">~</span> <span class="st">"Sex (Female=1)"</span>,</span>
<span id="cb83-21"><a href="#cb83-21" aria-hidden="true" tabindex="-1"></a>                           race___4 <span class="sc">~</span> <span class="st">"Race(White=1)"</span>,</span>
<span id="cb83-22"><a href="#cb83-22" aria-hidden="true" tabindex="-1"></a>                           hf_cat <span class="sc">~</span> <span class="st">"HF Category (HFpEF=1) (Study)"</span>,</span>
<span id="cb83-23"><a href="#cb83-23" aria-hidden="true" tabindex="-1"></a>                           egfr <span class="sc">~</span> <span class="st">"EGFR (Study)"</span>,</span>
<span id="cb83-24"><a href="#cb83-24" aria-hidden="true" tabindex="-1"></a>                           ALCAM <span class="sc">~</span> <span class="st">"ALCAM"</span>,</span>
<span id="cb83-25"><a href="#cb83-25" aria-hidden="true" tabindex="-1"></a>                           Gal<span class="fl">.4</span> <span class="sc">~</span> <span class="st">"Gal-4"</span>,</span>
<span id="cb83-26"><a href="#cb83-26" aria-hidden="true" tabindex="-1"></a>                           PECAM<span class="fl">.1</span> <span class="sc">~</span> <span class="st">"PECAM-1"</span>,</span>
<span id="cb83-27"><a href="#cb83-27" aria-hidden="true" tabindex="-1"></a>                           GP6 <span class="sc">~</span> <span class="st">"GP6"</span>,</span>
<span id="cb83-28"><a href="#cb83-28" aria-hidden="true" tabindex="-1"></a>                           ST2 <span class="sc">~</span> <span class="st">"ST2"</span></span>
<span id="cb83-29"><a href="#cb83-29" aria-hidden="true" tabindex="-1"></a>            ),</span>
<span id="cb83-30"><a href="#cb83-30" aria-hidden="true" tabindex="-1"></a>            <span class="at">missing_text =</span> <span class="st">"(Missing)"</span></span>
<span id="cb83-31"><a href="#cb83-31" aria-hidden="true" tabindex="-1"></a>)  <span class="sc">%&gt;%</span> <span class="fu">modify_header</span>(label <span class="sc">~</span> <span class="st">"**Variable**"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb83-32"><a href="#cb83-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">modify_caption</span>(<span class="st">"**Baseline Measurements (Unmatched vs. Matched) SPPB**"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="srdqqrfpnl" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#srdqqrfpnl table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#srdqqrfpnl thead, #srdqqrfpnl tbody, #srdqqrfpnl tfoot, #srdqqrfpnl tr, #srdqqrfpnl td, #srdqqrfpnl th {
  border-style: none;
}

#srdqqrfpnl p {
  margin: 0;
  padding: 0;
}

#srdqqrfpnl .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#srdqqrfpnl .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#srdqqrfpnl .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#srdqqrfpnl .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#srdqqrfpnl .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#srdqqrfpnl .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#srdqqrfpnl .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#srdqqrfpnl .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#srdqqrfpnl .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#srdqqrfpnl .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#srdqqrfpnl .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#srdqqrfpnl .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#srdqqrfpnl .gt_spanner_row {
  border-bottom-style: hidden;
}

#srdqqrfpnl .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#srdqqrfpnl .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#srdqqrfpnl .gt_from_md > :first-child {
  margin-top: 0;
}

#srdqqrfpnl .gt_from_md > :last-child {
  margin-bottom: 0;
}

#srdqqrfpnl .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#srdqqrfpnl .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#srdqqrfpnl .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#srdqqrfpnl .gt_row_group_first td {
  border-top-width: 2px;
}

#srdqqrfpnl .gt_row_group_first th {
  border-top-width: 2px;
}

#srdqqrfpnl .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#srdqqrfpnl .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#srdqqrfpnl .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#srdqqrfpnl .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#srdqqrfpnl .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#srdqqrfpnl .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#srdqqrfpnl .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#srdqqrfpnl .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#srdqqrfpnl .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#srdqqrfpnl .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#srdqqrfpnl .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#srdqqrfpnl .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#srdqqrfpnl .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#srdqqrfpnl .gt_left {
  text-align: left;
}

#srdqqrfpnl .gt_center {
  text-align: center;
}

#srdqqrfpnl .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#srdqqrfpnl .gt_font_normal {
  font-weight: normal;
}

#srdqqrfpnl .gt_font_bold {
  font-weight: bold;
}

#srdqqrfpnl .gt_font_italic {
  font-style: italic;
}

#srdqqrfpnl .gt_super {
  font-size: 65%;
}

#srdqqrfpnl .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#srdqqrfpnl .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#srdqqrfpnl .gt_indent_1 {
  text-indent: 5px;
}

#srdqqrfpnl .gt_indent_2 {
  text-indent: 10px;
}

#srdqqrfpnl .gt_indent_3 {
  text-indent: 15px;
}

#srdqqrfpnl .gt_indent_4 {
  text-indent: 20px;
}

#srdqqrfpnl .gt_indent_5 {
  text-indent: 25px;
}

#srdqqrfpnl .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#srdqqrfpnl div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<caption><strong>Baseline Measurements (Unmatched vs.&nbsp;Matched) SPPB</strong></caption>
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="gt_col_headings header">
<th id="label" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><strong>Variable</strong></th>
<th id="stat_1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>MatchedControl</strong><br> N = 81<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>MatchedIntervention</strong><br> N = 81<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>Unmatched</strong><br> N = 19<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">72 (8)</td>
<td class="gt_row gt_center" headers="stat_2">72 (8)</td>
<td class="gt_row gt_center" headers="stat_3">73 (7)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">70 (66, 76)</td>
<td class="gt_row gt_center" headers="stat_2">71 (65, 77)</td>
<td class="gt_row gt_center" headers="stat_3">76 (68, 78)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">60, 90</td>
<td class="gt_row gt_center" headers="stat_2">60, 91</td>
<td class="gt_row gt_center" headers="stat_3">61, 81</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Sex (Female=1)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">34 / 81 (42%)</td>
<td class="gt_row gt_center" headers="stat_2">43 / 81 (53%)</td>
<td class="gt_row gt_center" headers="stat_3">12 / 19 (63%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">47 / 81 (58%)</td>
<td class="gt_row gt_center" headers="stat_2">38 / 81 (47%)</td>
<td class="gt_row gt_center" headers="stat_3">7 / 19 (37%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Race(White=1)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">39 / 81 (48%)</td>
<td class="gt_row gt_center" headers="stat_2">33 / 81 (41%)</td>
<td class="gt_row gt_center" headers="stat_3">8 / 19 (42%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">42 / 81 (52%)</td>
<td class="gt_row gt_center" headers="stat_2">48 / 81 (59%)</td>
<td class="gt_row gt_center" headers="stat_3">11 / 19 (58%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">HF Category (HFpEF=1) (Study)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">46 / 81 (57%)</td>
<td class="gt_row gt_center" headers="stat_2">40 / 81 (49%)</td>
<td class="gt_row gt_center" headers="stat_3">6 / 19 (32%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">35 / 81 (43%)</td>
<td class="gt_row gt_center" headers="stat_2">41 / 81 (51%)</td>
<td class="gt_row gt_center" headers="stat_3">13 / 19 (68%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">EGFR (Study)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">58 (23)</td>
<td class="gt_row gt_center" headers="stat_2">52 (22)</td>
<td class="gt_row gt_center" headers="stat_3">56 (22)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">58 (37, 71)</td>
<td class="gt_row gt_center" headers="stat_2">49 (38, 63)</td>
<td class="gt_row gt_center" headers="stat_3">63 (35, 70)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">18, 101</td>
<td class="gt_row gt_center" headers="stat_2">17, 107</td>
<td class="gt_row gt_center" headers="stat_3">25, 96</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;(Missing)</td>
<td class="gt_row gt_center" headers="stat_1">1</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">ALCAM</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">6.70 (0.31)</td>
<td class="gt_row gt_center" headers="stat_2">6.66 (0.37)</td>
<td class="gt_row gt_center" headers="stat_3">6.72 (0.45)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">6.69 (6.45, 6.91)</td>
<td class="gt_row gt_center" headers="stat_2">6.65 (6.40, 6.88)</td>
<td class="gt_row gt_center" headers="stat_3">6.71 (6.38, 7.03)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">6.08, 7.82</td>
<td class="gt_row gt_center" headers="stat_2">6.06, 7.63</td>
<td class="gt_row gt_center" headers="stat_3">5.99, 7.71</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Gal-4</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">3.90 (0.67)</td>
<td class="gt_row gt_center" headers="stat_2">3.96 (0.65)</td>
<td class="gt_row gt_center" headers="stat_3">3.87 (0.65)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">3.90 (3.54, 4.29)</td>
<td class="gt_row gt_center" headers="stat_2">3.99 (3.59, 4.35)</td>
<td class="gt_row gt_center" headers="stat_3">3.77 (3.40, 4.52)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">2.45, 6.13</td>
<td class="gt_row gt_center" headers="stat_2">2.38, 5.64</td>
<td class="gt_row gt_center" headers="stat_3">3.01, 5.42</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">PECAM-1</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">3.48 (0.42)</td>
<td class="gt_row gt_center" headers="stat_2">3.39 (0.36)</td>
<td class="gt_row gt_center" headers="stat_3">3.39 (0.30)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">3.46 (3.23, 3.73)</td>
<td class="gt_row gt_center" headers="stat_2">3.41 (3.15, 3.61)</td>
<td class="gt_row gt_center" headers="stat_3">3.42 (3.19, 3.55)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">2.52, 4.75</td>
<td class="gt_row gt_center" headers="stat_2">2.36, 4.34</td>
<td class="gt_row gt_center" headers="stat_3">2.76, 4.01</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">GP6</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">1.80 (0.47)</td>
<td class="gt_row gt_center" headers="stat_2">1.76 (0.49)</td>
<td class="gt_row gt_center" headers="stat_3">1.71 (0.95)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">1.83 (1.45, 2.13)</td>
<td class="gt_row gt_center" headers="stat_2">1.75 (1.47, 2.10)</td>
<td class="gt_row gt_center" headers="stat_3">1.54 (1.01, 2.46)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">0.83, 3.15</td>
<td class="gt_row gt_center" headers="stat_2">0.55, 2.96</td>
<td class="gt_row gt_center" headers="stat_3">0.59, 3.55</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">ST2</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">6.52 (0.79)</td>
<td class="gt_row gt_center" headers="stat_2">6.67 (0.73)</td>
<td class="gt_row gt_center" headers="stat_3">7.31 (1.07)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">6.48 (5.95, 7.01)</td>
<td class="gt_row gt_center" headers="stat_2">6.66 (6.20, 7.26)</td>
<td class="gt_row gt_center" headers="stat_3">6.91 (6.75, 8.29)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">4.89, 8.62</td>
<td class="gt_row gt_center" headers="stat_2">5.13, 9.12</td>
<td class="gt_row gt_center" headers="stat_3">5.59, 9.44</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span> n / N (%)</td>
</tr>
</tfoot>

</table>

</div>
</div>
</div>
</section>
<section id="mwd-3" class="level3">
<h3 class="anchored" data-anchor-id="mwd-3">6MWD</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb84"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb84-1"><a href="#cb84-1" aria-hidden="true" tabindex="-1"></a>set.trt <span class="ot">&lt;-</span> ps.df[ps.df[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb84-2"><a href="#cb84-2" aria-hidden="true" tabindex="-1"></a>set.cont <span class="ot">&lt;-</span> ps.df[ps.df[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">0</span>, ]</span>
<span id="cb84-3"><a href="#cb84-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-4"><a href="#cb84-4" aria-hidden="true" tabindex="-1"></a>match.set <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb84-5"><a href="#cb84-5" aria-hidden="true" tabindex="-1"></a>calp.sd <span class="ot">&lt;-</span> <span class="dv">1</span></span>
<span id="cb84-6"><a href="#cb84-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-7"><a href="#cb84-7" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">dim</span>(set.trt)[<span class="dv">1</span>]) {</span>
<span id="cb84-8"><a href="#cb84-8" aria-hidden="true" tabindex="-1"></a>  pair.set1 <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb84-9"><a href="#cb84-9" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb84-10"><a href="#cb84-10" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Check if control subject is within caliper on all three proteins</span></span>
<span id="cb84-11"><a href="#cb84-11" aria-hidden="true" tabindex="-1"></a>  var.select1 <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(<span class="fu">abs</span>(set.cont<span class="sc">$</span>ALCAM <span class="sc">-</span> set.trt<span class="sc">$</span>ALCAM[i]) <span class="sc">&lt;</span> calp.sd <span class="sc">*</span> <span class="fu">sd</span>(set.trt<span class="sc">$</span>ALCAM), <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb84-12"><a href="#cb84-12" aria-hidden="true" tabindex="-1"></a>  var.select2 <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(<span class="fu">abs</span>(set.cont<span class="sc">$</span><span class="st">`</span><span class="at">Gal-4</span><span class="st">`</span> <span class="sc">-</span> set.trt<span class="sc">$</span><span class="st">`</span><span class="at">Gal-4</span><span class="st">`</span>[i]) <span class="sc">&lt;</span> calp.sd <span class="sc">*</span> <span class="fu">sd</span>(set.trt<span class="sc">$</span><span class="st">`</span><span class="at">Gal-4</span><span class="st">`</span>), <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb84-13"><a href="#cb84-13" aria-hidden="true" tabindex="-1"></a>  var.select3 <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(<span class="fu">abs</span>(set.cont<span class="sc">$</span><span class="st">`</span><span class="at">PECAM-1</span><span class="st">`</span> <span class="sc">-</span> set.trt<span class="sc">$</span><span class="st">`</span><span class="at">PECAM-1</span><span class="st">`</span>[i]) <span class="sc">&lt;</span> calp.sd <span class="sc">*</span> <span class="fu">sd</span>(set.trt<span class="sc">$</span><span class="st">`</span><span class="at">PECAM-1</span><span class="st">`</span>), <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb84-14"><a href="#cb84-14" aria-hidden="true" tabindex="-1"></a>  var.selectf <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(var.select1 <span class="sc">+</span> var.select2 <span class="sc">+</span> var.select3 <span class="sc">==</span> <span class="dv">3</span>, <span class="dv">1</span>, <span class="dv">0</span>)</span>
<span id="cb84-15"><a href="#cb84-15" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb84-16"><a href="#cb84-16" aria-hidden="true" tabindex="-1"></a>  id <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(var.selectf))</span>
<span id="cb84-17"><a href="#cb84-17" aria-hidden="true" tabindex="-1"></a>  lenset.cont2 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">cbind</span>(id, set.cont, var.selectf))</span>
<span id="cb84-18"><a href="#cb84-18" aria-hidden="true" tabindex="-1"></a>  lenset.cont3 <span class="ot">&lt;-</span> lenset.cont2[lenset.cont2[, <span class="st">"var.selectf"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb84-19"><a href="#cb84-19" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb84-20"><a href="#cb84-20" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="fu">dim</span>(lenset.cont3)[<span class="dv">1</span>] <span class="sc">&gt;</span> <span class="dv">0</span>) {</span>
<span id="cb84-21"><a href="#cb84-21" aria-hidden="true" tabindex="-1"></a>    cont.index <span class="ot">&lt;-</span> <span class="fu">which.min</span>(<span class="fu">abs</span>(lenset.cont3<span class="sc">$</span>prop.score <span class="sc">-</span> set.trt<span class="sc">$</span>prop.score[i]))</span>
<span id="cb84-22"><a href="#cb84-22" aria-hidden="true" tabindex="-1"></a>    pair.set <span class="ot">&lt;-</span> <span class="fu">rbind</span>(set.trt[i, ], lenset.cont3[cont.index, <span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="sc">-</span><span class="fu">dim</span>(lenset.cont3)[<span class="dv">2</span>])])</span>
<span id="cb84-23"><a href="#cb84-23" aria-hidden="true" tabindex="-1"></a>    pairin <span class="ot">&lt;-</span> <span class="fu">rep</span>(i, <span class="dv">2</span>)</span>
<span id="cb84-24"><a href="#cb84-24" aria-hidden="true" tabindex="-1"></a>    pair.set1 <span class="ot">&lt;-</span> <span class="fu">cbind</span>(pair.set, pairin)</span>
<span id="cb84-25"><a href="#cb84-25" aria-hidden="true" tabindex="-1"></a>    set.cont <span class="ot">&lt;-</span> set.cont[<span class="sc">!</span>set.cont<span class="sc">$</span>prop.score <span class="sc">==</span> lenset.cont3<span class="sc">$</span>prop.score[cont.index], ]</span>
<span id="cb84-26"><a href="#cb84-26" aria-hidden="true" tabindex="-1"></a>  } <span class="cf">else</span> {</span>
<span id="cb84-27"><a href="#cb84-27" aria-hidden="true" tabindex="-1"></a>    pair.set1 <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb84-28"><a href="#cb84-28" aria-hidden="true" tabindex="-1"></a>    set.cont <span class="ot">&lt;-</span> set.cont</span>
<span id="cb84-29"><a href="#cb84-29" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb84-30"><a href="#cb84-30" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb84-31"><a href="#cb84-31" aria-hidden="true" tabindex="-1"></a>  match.set <span class="ot">&lt;-</span> <span class="fu">rbind</span>(match.set, pair.set1)</span>
<span id="cb84-32"><a href="#cb84-32" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb84-33"><a href="#cb84-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-34"><a href="#cb84-34" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate treatment effect</span></span>
<span id="cb84-35"><a href="#cb84-35" aria-hidden="true" tabindex="-1"></a>treat.set <span class="ot">&lt;-</span> match.set[match.set[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">1</span>, ]</span>
<span id="cb84-36"><a href="#cb84-36" aria-hidden="true" tabindex="-1"></a>control.set <span class="ot">&lt;-</span> match.set[match.set[, <span class="st">"trt"</span>] <span class="sc">==</span> <span class="dv">0</span>, ]</span>
<span id="cb84-37"><a href="#cb84-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-38"><a href="#cb84-38" aria-hidden="true" tabindex="-1"></a>outcome.diff <span class="ot">&lt;-</span> (treat.set<span class="sc">$</span>fu_smw <span class="sc">-</span> treat.set<span class="sc">$</span>bl_smw) <span class="sc">-</span></span>
<span id="cb84-39"><a href="#cb84-39" aria-hidden="true" tabindex="-1"></a>  (control.set<span class="sc">$</span>fu_smw <span class="sc">-</span> control.set<span class="sc">$</span>bl_smw)</span>
<span id="cb84-40"><a href="#cb84-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-41"><a href="#cb84-41" aria-hidden="true" tabindex="-1"></a>set.total <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">rbind</span>(</span>
<span id="cb84-42"><a href="#cb84-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(treat.set, outcome.diff),</span>
<span id="cb84-43"><a href="#cb84-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(control.set, outcome.diff)</span>
<span id="cb84-44"><a href="#cb84-44" aria-hidden="true" tabindex="-1"></a>))</span>
<span id="cb84-45"><a href="#cb84-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-46"><a href="#cb84-46" aria-hidden="true" tabindex="-1"></a>matchedset<span class="fl">.6</span>mwd <span class="ot">&lt;-</span> set.total  <span class="co"># Store for later use</span></span>
<span id="cb84-47"><a href="#cb84-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb84-48"><a href="#cb84-48" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"6MWD Matching Summary</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>6MWD Matching Summary</code></pre>
</div>
<div class="sourceCode cell-code" id="cb86"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb86-1"><a href="#cb86-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"====================</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>====================</code></pre>
</div>
<div class="sourceCode cell-code" id="cb88"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Matched pairs:"</span>, <span class="fu">nrow</span>(treat.set), <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Matched pairs: 71 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb90"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb90-1"><a href="#cb90-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Total matched subjects:"</span>, <span class="fu">nrow</span>(set.total), <span class="st">"</span><span class="sc">\n\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Total matched subjects: 142 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb92"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb92-1"><a href="#cb92-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Build matching tree</span></span>
<span id="cb92-2"><a href="#cb92-2" aria-hidden="true" tabindex="-1"></a>tree.rpart <span class="ot">&lt;-</span> <span class="fu">rpart</span>(</span>
<span id="cb92-3"><a href="#cb92-3" aria-hidden="true" tabindex="-1"></a>  outcome.diff <span class="sc">~</span> ALCAM <span class="sc">+</span> <span class="st">`</span><span class="at">Gal-4</span><span class="st">`</span> <span class="sc">+</span> <span class="st">`</span><span class="at">PECAM-1</span><span class="st">`</span>,</span>
<span id="cb92-4"><a href="#cb92-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">method =</span> <span class="st">"anova"</span>,</span>
<span id="cb92-5"><a href="#cb92-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> set.total,</span>
<span id="cb92-6"><a href="#cb92-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">control =</span> <span class="fu">rpart.control</span>(<span class="at">minbucket =</span> <span class="dv">30</span>, <span class="at">maxdepth =</span> <span class="dv">1</span>)</span>
<span id="cb92-7"><a href="#cb92-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb92-8"><a href="#cb92-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-9"><a href="#cb92-9" aria-hidden="true" tabindex="-1"></a>set.total<span class="sc">$</span>outcome <span class="ot">&lt;-</span> set.total<span class="sc">$</span>fu_smw <span class="sc">-</span> set.total<span class="sc">$</span>bl_smw</span>
<span id="cb92-10"><a href="#cb92-10" aria-hidden="true" tabindex="-1"></a>tree.all <span class="ot">&lt;-</span> <span class="fu">jesse.tree.select</span>(tree.rpart, set.total)</span>
<span id="cb92-11"><a href="#cb92-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-12"><a href="#cb92-12" aria-hidden="true" tabindex="-1"></a><span class="fu">rpart.plot</span>(tree.all, <span class="at">box.palette =</span> <span class="st">'Reds'</span>, <span class="at">extra =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-24-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb93"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb93-1"><a href="#cb93-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Extract treatment effects by tree node</span></span>
<span id="cb93-2"><a href="#cb93-2" aria-hidden="true" tabindex="-1"></a>set.total<span class="sc">$</span>smw_node <span class="ot">&lt;-</span> <span class="fu">rpart.predict.leaves</span>(tree.all, set.total, <span class="at">type =</span> <span class="st">"where"</span>)</span>
<span id="cb93-3"><a href="#cb93-3" aria-hidden="true" tabindex="-1"></a>modscore.smw <span class="ot">&lt;-</span> <span class="fu">lm</span>(<span class="at">data =</span> set.total, <span class="at">formula =</span> outcome.diff <span class="sc">~</span> <span class="dv">0</span> <span class="sc">+</span> <span class="fu">as.factor</span>(smw_node))</span>
<span id="cb93-4"><a href="#cb93-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-5"><a href="#cb93-5" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"6MWD Matching Tree - Treatment Effects by Node</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>6MWD Matching Tree - Treatment Effects by Node</code></pre>
</div>
<div class="sourceCode cell-code" id="cb95"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb95-1"><a href="#cb95-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"==============================================</span><span class="sc">\n\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>==============================================</code></pre>
</div>
<div class="sourceCode cell-code" id="cb97"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb97-1"><a href="#cb97-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">confint</span>(modscore.smw))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>                         2.5 %    97.5 %
as.factor(smw_node)2 -16.52650  35.55242
as.factor(smw_node)3  56.36864 113.05136</code></pre>
</div>
<div class="sourceCode cell-code" id="cb99"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb99-1"><a href="#cb99-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="sourceCode cell-code" id="cb100"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb100-1"><a href="#cb100-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(modscore.smw))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = outcome.diff ~ 0 + as.factor(smw_node), data = set.total)

Residuals:
    Min      1Q  Median      3Q     Max 
-438.43  -55.15   -4.37   57.88  439.20 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
as.factor(smw_node)2    9.513     13.171   0.722    0.471    
as.factor(smw_node)3   84.710     14.335   5.909 2.49e-08 ***
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 115.6 on 140 degrees of freedom
Multiple R-squared:  0.202, Adjusted R-squared:  0.1906 
F-statistic: 17.72 on 2 and 140 DF,  p-value: 1.38e-07</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb102"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb102-1"><a href="#cb102-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ---- K-Fold Cross-Validation for 6MWD ----</span></span>
<span id="cb102-2"><a href="#cb102-2" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">324</span>)</span>
<span id="cb102-3"><a href="#cb102-3" aria-hidden="true" tabindex="-1"></a>nfolds <span class="ot">&lt;-</span> <span class="dv">10</span></span>
<span id="cb102-4"><a href="#cb102-4" aria-hidden="true" tabindex="-1"></a>folds <span class="ot">&lt;-</span> <span class="fu">cut</span>(<span class="fu">seq</span>(<span class="dv">1</span>, <span class="fu">nrow</span>(treat.set)), <span class="at">breaks =</span> nfolds, <span class="at">labels =</span> <span class="cn">FALSE</span>)</span>
<span id="cb102-5"><a href="#cb102-5" aria-hidden="true" tabindex="-1"></a>folds <span class="ot">&lt;-</span> folds[<span class="fu">sample</span>(<span class="fu">nrow</span>(treat.set))]</span>
<span id="cb102-6"><a href="#cb102-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-7"><a href="#cb102-7" aria-hidden="true" tabindex="-1"></a>maxdepth <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span></span>
<span id="cb102-8"><a href="#cb102-8" aria-hidden="true" tabindex="-1"></a>minbucket <span class="ot">&lt;-</span> <span class="fu">seq</span>(<span class="dv">5</span>, <span class="dv">30</span>, <span class="dv">5</span>)</span>
<span id="cb102-9"><a href="#cb102-9" aria-hidden="true" tabindex="-1"></a>bst <span class="ot">&lt;-</span> <span class="dv">1000</span></span>
<span id="cb102-10"><a href="#cb102-10" aria-hidden="true" tabindex="-1"></a>bst.stg <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">0</span>)</span>
<span id="cb102-11"><a href="#cb102-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-12"><a href="#cb102-12" aria-hidden="true" tabindex="-1"></a>cvres.df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">maxdepth =</span> <span class="cn">NA</span>, <span class="at">minbucket =</span> <span class="cn">NA</span>, <span class="at">testrmse.dt =</span> <span class="cn">NA</span>, <span class="at">testrmse.rf =</span> <span class="cn">NA</span>)</span>
<span id="cb102-13"><a href="#cb102-13" aria-hidden="true" tabindex="-1"></a>idx <span class="ot">&lt;-</span> <span class="dv">0</span></span>
<span id="cb102-14"><a href="#cb102-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-15"><a href="#cb102-15" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (j <span class="cf">in</span> maxdepth) {</span>
<span id="cb102-16"><a href="#cb102-16" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (k <span class="cf">in</span> minbucket) {</span>
<span id="cb102-17"><a href="#cb102-17" aria-hidden="true" tabindex="-1"></a>    idx <span class="ot">&lt;-</span> idx <span class="sc">+</span> <span class="dv">1</span></span>
<span id="cb102-18"><a href="#cb102-18" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb102-19"><a href="#cb102-19" aria-hidden="true" tabindex="-1"></a>    tree<span class="fl">.2</span> <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-20"><a href="#cb102-20" aria-hidden="true" tabindex="-1"></a>    tree.leaves <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-21"><a href="#cb102-21" aria-hidden="true" tabindex="-1"></a>    tree.mod.train <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-22"><a href="#cb102-22" aria-hidden="true" tabindex="-1"></a>    tree.test.rmse <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-23"><a href="#cb102-23" aria-hidden="true" tabindex="-1"></a>    rf.mod.train <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-24"><a href="#cb102-24" aria-hidden="true" tabindex="-1"></a>    rf.test.rmse <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb102-25"><a href="#cb102-25" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb102-26"><a href="#cb102-26" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>nfolds) {</span>
<span id="cb102-27"><a href="#cb102-27" aria-hidden="true" tabindex="-1"></a>      ids <span class="ot">&lt;-</span> treat.set<span class="sc">$</span>subject_id[folds <span class="sc">!=</span> i]</span>
<span id="cb102-28"><a href="#cb102-28" aria-hidden="true" tabindex="-1"></a>      pairs.train <span class="ot">&lt;-</span> set.total<span class="sc">$</span>pairin[set.total<span class="sc">$</span>subject_id <span class="sc">%in%</span> ids]</span>
<span id="cb102-29"><a href="#cb102-29" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-30"><a href="#cb102-30" aria-hidden="true" tabindex="-1"></a>      ids <span class="ot">&lt;-</span> treat.set<span class="sc">$</span>subject_id[folds <span class="sc">==</span> i]</span>
<span id="cb102-31"><a href="#cb102-31" aria-hidden="true" tabindex="-1"></a>      pairs.test <span class="ot">&lt;-</span> set.total<span class="sc">$</span>pairin[set.total<span class="sc">$</span>subject_id <span class="sc">%in%</span> ids]</span>
<span id="cb102-32"><a href="#cb102-32" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-33"><a href="#cb102-33" aria-hidden="true" tabindex="-1"></a>      tmp <span class="ot">&lt;-</span> <span class="fu">rpart</span>(</span>
<span id="cb102-34"><a href="#cb102-34" aria-hidden="true" tabindex="-1"></a>        outcome.diff <span class="sc">~</span> ALCAM <span class="sc">+</span> <span class="st">`</span><span class="at">Gal-4</span><span class="st">`</span> <span class="sc">+</span> <span class="st">`</span><span class="at">PECAM-1</span><span class="st">`</span>,</span>
<span id="cb102-35"><a href="#cb102-35" aria-hidden="true" tabindex="-1"></a>        <span class="at">method =</span> <span class="st">"anova"</span>,</span>
<span id="cb102-36"><a href="#cb102-36" aria-hidden="true" tabindex="-1"></a>        <span class="at">data =</span> set.total[set.total<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, ],</span>
<span id="cb102-37"><a href="#cb102-37" aria-hidden="true" tabindex="-1"></a>        <span class="at">control =</span> <span class="fu">rpart.control</span>(<span class="at">maxdepth =</span> j, <span class="at">minbucket =</span> k)</span>
<span id="cb102-38"><a href="#cb102-38" aria-hidden="true" tabindex="-1"></a>      )</span>
<span id="cb102-39"><a href="#cb102-39" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-40"><a href="#cb102-40" aria-hidden="true" tabindex="-1"></a>      tree<span class="fl">.2</span>[[i]] <span class="ot">&lt;-</span> <span class="fu">jesse.tree.select</span>(tmp, set.total)</span>
<span id="cb102-41"><a href="#cb102-41" aria-hidden="true" tabindex="-1"></a>      tree.leaves[[i]] <span class="ot">&lt;-</span> <span class="fu">rpart.predict.leaves</span>(tree<span class="fl">.2</span>[[i]], set.total, <span class="at">type =</span> <span class="st">"where"</span>)</span>
<span id="cb102-42"><a href="#cb102-42" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-43"><a href="#cb102-43" aria-hidden="true" tabindex="-1"></a>      df <span class="ot">&lt;-</span> <span class="fu">cbind</span>(set.total, tree.leaves[[i]])</span>
<span id="cb102-44"><a href="#cb102-44" aria-hidden="true" tabindex="-1"></a>      <span class="fu">colnames</span>(df)[<span class="fu">ncol</span>(df)] <span class="ot">&lt;-</span> <span class="st">"tree_leaves_temp"</span></span>
<span id="cb102-45"><a href="#cb102-45" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-46"><a href="#cb102-46" aria-hidden="true" tabindex="-1"></a>      tree.mod.train[[i]] <span class="ot">&lt;-</span> <span class="fu">lm</span>(</span>
<span id="cb102-47"><a href="#cb102-47" aria-hidden="true" tabindex="-1"></a>        outcome.diff <span class="sc">~</span> <span class="dv">0</span> <span class="sc">+</span> <span class="fu">as.factor</span>(tree_leaves_temp),</span>
<span id="cb102-48"><a href="#cb102-48" aria-hidden="true" tabindex="-1"></a>        <span class="at">data =</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, ]</span>
<span id="cb102-49"><a href="#cb102-49" aria-hidden="true" tabindex="-1"></a>      )</span>
<span id="cb102-50"><a href="#cb102-50" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-51"><a href="#cb102-51" aria-hidden="true" tabindex="-1"></a>      tree.test.rmse[[i]] <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(<span class="fu">mean</span>((</span>
<span id="cb102-52"><a href="#cb102-52" aria-hidden="true" tabindex="-1"></a>        df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, <span class="st">'outcome.diff'</span>] <span class="sc">-</span></span>
<span id="cb102-53"><a href="#cb102-53" aria-hidden="true" tabindex="-1"></a>          <span class="fu">predict</span>(tree.mod.train[[i]], df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, ])</span>
<span id="cb102-54"><a href="#cb102-54" aria-hidden="true" tabindex="-1"></a>      )<span class="sc">^</span><span class="dv">2</span>))</span>
<span id="cb102-55"><a href="#cb102-55" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-56"><a href="#cb102-56" aria-hidden="true" tabindex="-1"></a>      <span class="co"># Fit random forest on training data</span></span>
<span id="cb102-57"><a href="#cb102-57" aria-hidden="true" tabindex="-1"></a>      rf.df.train <span class="ot">&lt;-</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.train, <span class="fu">c</span>(<span class="st">"outcome.diff"</span>, <span class="st">"ALCAM"</span>, <span class="st">"Gal-4"</span>, <span class="st">"PECAM-1"</span>)]</span>
<span id="cb102-58"><a href="#cb102-58" aria-hidden="true" tabindex="-1"></a>      rf.df.test <span class="ot">&lt;-</span> df[df<span class="sc">$</span>pairin <span class="sc">%in%</span> pairs.test, <span class="fu">c</span>(<span class="st">"outcome.diff"</span>, <span class="st">"ALCAM"</span>, <span class="st">"Gal-4"</span>, <span class="st">"PECAM-1"</span>)]</span>
<span id="cb102-59"><a href="#cb102-59" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-60"><a href="#cb102-60" aria-hidden="true" tabindex="-1"></a>      <span class="fu">colnames</span>(rf.df.train) <span class="ot">&lt;-</span> <span class="fu">make.names</span>(<span class="fu">colnames</span>(rf.df.train))</span>
<span id="cb102-61"><a href="#cb102-61" aria-hidden="true" tabindex="-1"></a>      <span class="fu">colnames</span>(rf.df.test) <span class="ot">&lt;-</span> <span class="fu">make.names</span>(<span class="fu">colnames</span>(rf.df.test))</span>
<span id="cb102-62"><a href="#cb102-62" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-63"><a href="#cb102-63" aria-hidden="true" tabindex="-1"></a>      rf.mod.train[[i]] <span class="ot">&lt;-</span> <span class="fu">randomForest</span>(outcome.diff <span class="sc">~</span> ALCAM <span class="sc">+</span> Gal<span class="fl">.4</span> <span class="sc">+</span> PECAM<span class="fl">.1</span>,</span>
<span id="cb102-64"><a href="#cb102-64" aria-hidden="true" tabindex="-1"></a>                                        <span class="at">data =</span> rf.df.train, <span class="at">ntree =</span> <span class="dv">500</span>, <span class="at">seed =</span> <span class="dv">42</span>)</span>
<span id="cb102-65"><a href="#cb102-65" aria-hidden="true" tabindex="-1"></a>      </span>
<span id="cb102-66"><a href="#cb102-66" aria-hidden="true" tabindex="-1"></a>      rf.test.rmse[[i]] <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(<span class="fu">mean</span>((</span>
<span id="cb102-67"><a href="#cb102-67" aria-hidden="true" tabindex="-1"></a>        rf.df.test<span class="sc">$</span>outcome.diff <span class="sc">-</span></span>
<span id="cb102-68"><a href="#cb102-68" aria-hidden="true" tabindex="-1"></a>          <span class="fu">predict</span>(rf.mod.train[[i]], rf.df.test)</span>
<span id="cb102-69"><a href="#cb102-69" aria-hidden="true" tabindex="-1"></a>      )<span class="sc">^</span><span class="dv">2</span>))</span>
<span id="cb102-70"><a href="#cb102-70" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb102-71"><a href="#cb102-71" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb102-72"><a href="#cb102-72" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse)) <span class="sc">&lt;</span> bst) {</span>
<span id="cb102-73"><a href="#cb102-73" aria-hidden="true" tabindex="-1"></a>      bst <span class="ot">&lt;-</span> <span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse))</span>
<span id="cb102-74"><a href="#cb102-74" aria-hidden="true" tabindex="-1"></a>      bst.stg[<span class="dv">1</span>] <span class="ot">&lt;-</span> j</span>
<span id="cb102-75"><a href="#cb102-75" aria-hidden="true" tabindex="-1"></a>      bst.stg[<span class="dv">2</span>] <span class="ot">&lt;-</span> k</span>
<span id="cb102-76"><a href="#cb102-76" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb102-77"><a href="#cb102-77" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb102-78"><a href="#cb102-78" aria-hidden="true" tabindex="-1"></a>    cvres.df[idx, ] <span class="ot">&lt;-</span> <span class="fu">c</span>(j, k, <span class="fu">mean</span>(<span class="fu">unlist</span>(tree.test.rmse)), <span class="fu">mean</span>(<span class="fu">unlist</span>(rf.test.rmse)))</span>
<span id="cb102-79"><a href="#cb102-79" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb102-80"><a href="#cb102-80" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb102-81"><a href="#cb102-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-82"><a href="#cb102-82" aria-hidden="true" tabindex="-1"></a><span class="co"># Format CV results</span></span>
<span id="cb102-83"><a href="#cb102-83" aria-hidden="true" tabindex="-1"></a>cvres.df<span class="sc">$</span>maxdepth <span class="ot">&lt;-</span> <span class="fu">factor</span>(cvres.df<span class="sc">$</span>maxdepth, <span class="at">levels =</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>)</span>
<span id="cb102-84"><a href="#cb102-84" aria-hidden="true" tabindex="-1"></a>cvres.df<span class="sc">$</span>minbucket <span class="ot">&lt;-</span> <span class="fu">as.integer</span>(cvres.df<span class="sc">$</span>minbucket)</span>
<span id="cb102-85"><a href="#cb102-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-86"><a href="#cb102-86" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate mean RF RMSE for reference line</span></span>
<span id="cb102-87"><a href="#cb102-87" aria-hidden="true" tabindex="-1"></a>rf.mean<span class="fl">.6</span>mwd <span class="ot">&lt;-</span> <span class="fu">mean</span>(cvres.df<span class="sc">$</span>testrmse.rf, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb102-88"><a href="#cb102-88" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb102-89"><a href="#cb102-89" aria-hidden="true" tabindex="-1"></a>cvplot1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> cvres.df, <span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> minbucket, <span class="at">color =</span> maxdepth, <span class="at">fill =</span> maxdepth)) <span class="sc">+</span></span>
<span id="cb102-90"><a href="#cb102-90" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">y =</span> testrmse.dt), <span class="at">linewidth =</span> <span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb102-91"><a href="#cb102-91" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">y =</span> testrmse.dt, <span class="at">shape =</span> maxdepth), <span class="at">size =</span> <span class="dv">3</span>) <span class="sc">+</span></span>
<span id="cb102-92"><a href="#cb102-92" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> rf.mean<span class="fl">.6</span>mwd, <span class="at">color =</span> <span class="st">'black'</span>, <span class="at">linetype =</span> <span class="st">'dashed'</span>, <span class="at">linewidth =</span> <span class="fl">1.2</span>) <span class="sc">+</span></span>
<span id="cb102-93"><a href="#cb102-93" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, <span class="at">x =</span> <span class="dv">20</span>, <span class="at">y =</span> rf.mean<span class="fl">.6</span>mwd <span class="sc">+</span> <span class="dv">2</span>, </span>
<span id="cb102-94"><a href="#cb102-94" aria-hidden="true" tabindex="-1"></a>           <span class="at">label =</span> <span class="fu">paste</span>(<span class="st">"Random Forest:"</span>, <span class="fu">round</span>(rf.mean<span class="fl">.6</span>mwd, <span class="dv">3</span>)), </span>
<span id="cb102-95"><a href="#cb102-95" aria-hidden="true" tabindex="-1"></a>           <span class="at">size =</span> <span class="dv">4</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb102-96"><a href="#cb102-96" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">"Average Test RMSE"</span>) <span class="sc">+</span></span>
<span id="cb102-97"><a href="#cb102-97" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">"Minimum Bucket Size"</span>) <span class="sc">+</span></span>
<span id="cb102-98"><a href="#cb102-98" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Average Test RMSE: 6MWD Outcome"</span>,</span>
<span id="cb102-99"><a href="#cb102-99" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Decision Trees vs. Random Forest"</span>) <span class="sc">+</span></span>
<span id="cb102-100"><a href="#cb102-100" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylim</span>(<span class="fu">c</span>(<span class="dv">100</span>, <span class="dv">150</span>)) <span class="sc">+</span></span>
<span id="cb102-101"><a href="#cb102-101" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">5</span>, <span class="dv">30</span>, <span class="dv">5</span>)) <span class="sc">+</span></span>
<span id="cb102-102"><a href="#cb102-102" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">name =</span> <span class="st">"Max Depth"</span>, <span class="at">values =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"#1f77b4"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"#ff7f0e"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"#2ca02c"</span>)) <span class="sc">+</span></span>
<span id="cb102-103"><a href="#cb102-103" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">name =</span> <span class="st">"Max Depth"</span>, <span class="at">values =</span> <span class="fu">c</span>(<span class="st">"1"</span> <span class="ot">=</span> <span class="st">"#1f77b4"</span>, <span class="st">"2"</span> <span class="ot">=</span> <span class="st">"#ff7f0e"</span>, <span class="st">"3"</span> <span class="ot">=</span> <span class="st">"#2ca02c"</span>)) <span class="sc">+</span></span>
<span id="cb102-104"><a href="#cb102-104" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb102-105"><a href="#cb102-105" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb102-106"><a href="#cb102-106" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">face =</span> <span class="st">"bold"</span>, <span class="at">size =</span> <span class="dv">14</span>),</span>
<span id="cb102-107"><a href="#cb102-107" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">12</span>),</span>
<span id="cb102-108"><a href="#cb102-108" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"top"</span>,</span>
<span id="cb102-109"><a href="#cb102-109" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.major =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="st">"gray90"</span>),</span>
<span id="cb102-110"><a href="#cb102-110" aria-hidden="true" tabindex="-1"></a>    <span class="at">panel.grid.minor =</span> <span class="fu">element_blank</span>()</span>
<span id="cb102-111"><a href="#cb102-111" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb102-112"><a href="#cb102-112" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(cvplot1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-25-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb103"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb103-1"><a href="#cb103-1" aria-hidden="true" tabindex="-1"></a>varlist<span class="ot">=</span><span class="fu">c</span>(<span class="st">"intervention_1_control_0"</span>,<span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,</span>
<span id="cb103-2"><a href="#cb103-2" aria-hidden="true" tabindex="-1"></a>          <span class="st">"hf_cat"</span>,<span class="st">"egfr"</span>,protlist)</span>
<span id="cb103-3"><a href="#cb103-3" aria-hidden="true" tabindex="-1"></a>tmpdf <span class="ot">=</span> <span class="fu">rbind</span>(<span class="fu">data.frame</span>(set.total[,varlist],<span class="at">dataset=</span><span class="st">"matched"</span>),</span>
<span id="cb103-4"><a href="#cb103-4" aria-hidden="true" tabindex="-1"></a>              <span class="fu">data.frame</span>(<span class="fu">setdiff</span>(ps.df[,varlist],set.total[,varlist]),<span class="at">dataset=</span><span class="st">"unmatched"</span>))</span>
<span id="cb103-5"><a href="#cb103-5" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>dataset<span class="ot">=</span><span class="fu">factor</span>(tmpdf<span class="sc">$</span>dataset,<span class="at">levels=</span><span class="fu">c</span>(<span class="st">"unmatched"</span>,<span class="st">"matched"</span>))</span>
<span id="cb103-6"><a href="#cb103-6" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">str_c</span>(tmpdf<span class="sc">$</span>intervention_1_control_0,tmpdf<span class="sc">$</span>dataset)</span>
<span id="cb103-7"><a href="#cb103-7" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">as.factor</span>(tmpdf<span class="sc">$</span>grp)</span>
<span id="cb103-8"><a href="#cb103-8" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp<span class="ot">=</span><span class="fu">fct_collapse</span>(tmpdf<span class="sc">$</span>grp, <span class="at">Unmatched =</span> <span class="fu">c</span>(<span class="st">"0unmatched"</span>,<span class="st">"1unmatched"</span>), <span class="at">MatchedControl =</span> <span class="st">"0matched"</span>,<span class="at">MatchedIntervention =</span> <span class="st">"1matched"</span>)</span>
<span id="cb103-9"><a href="#cb103-9" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="sc">$</span>grp <span class="ot">=</span> <span class="fu">factor</span>(tmpdf<span class="sc">$</span>grp,<span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"MatchedControl"</span>,<span class="st">"MatchedIntervention"</span>,<span class="st">"Unmatched"</span>))</span>
<span id="cb103-10"><a href="#cb103-10" aria-hidden="true" tabindex="-1"></a>tmpdf<span class="ot">=</span>tmpdf[,<span class="sc">-</span><span class="fu">which</span>(<span class="fu">names</span>(tmpdf) <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"intervention_1_control_0"</span>,<span class="st">"dataset"</span>))]</span>
<span id="cb103-11"><a href="#cb103-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-12"><a href="#cb103-12" aria-hidden="true" tabindex="-1"></a><span class="fu">tbl_summary</span>(tmpdf,</span>
<span id="cb103-13"><a href="#cb103-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">by=</span>grp,</span>
<span id="cb103-14"><a href="#cb103-14" aria-hidden="true" tabindex="-1"></a>            <span class="at">type =</span> <span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="st">"continuous2"</span>,</span>
<span id="cb103-15"><a href="#cb103-15" aria-hidden="true" tabindex="-1"></a>            <span class="at">statistic =</span> <span class="fu">list</span>(<span class="fu">all_continuous</span>() <span class="sc">~</span> <span class="fu">c</span>(<span class="st">"{mean} ({sd})"</span>,</span>
<span id="cb103-16"><a href="#cb103-16" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{median} ({p25}, {p75})"</span>,</span>
<span id="cb103-17"><a href="#cb103-17" aria-hidden="true" tabindex="-1"></a>                                                  <span class="st">"{min}, {max}"</span>),</span>
<span id="cb103-18"><a href="#cb103-18" aria-hidden="true" tabindex="-1"></a>                             <span class="fu">all_categorical</span>() <span class="sc">~</span> <span class="st">"{n} / {N} ({p}%)"</span></span>
<span id="cb103-19"><a href="#cb103-19" aria-hidden="true" tabindex="-1"></a>            ),<span class="at">label =</span> <span class="fu">list</span>(age <span class="sc">~</span> <span class="st">"Age"</span>,</span>
<span id="cb103-20"><a href="#cb103-20" aria-hidden="true" tabindex="-1"></a>                           sex <span class="sc">~</span> <span class="st">"Sex (Female=1)"</span>,</span>
<span id="cb103-21"><a href="#cb103-21" aria-hidden="true" tabindex="-1"></a>                           race___4 <span class="sc">~</span> <span class="st">"Race(White=1)"</span>,</span>
<span id="cb103-22"><a href="#cb103-22" aria-hidden="true" tabindex="-1"></a>                           hf_cat <span class="sc">~</span> <span class="st">"HF Category (HFpEF=1) (Study)"</span>,</span>
<span id="cb103-23"><a href="#cb103-23" aria-hidden="true" tabindex="-1"></a>                           egfr <span class="sc">~</span> <span class="st">"EGFR (Study)"</span>,</span>
<span id="cb103-24"><a href="#cb103-24" aria-hidden="true" tabindex="-1"></a>                           ALCAM <span class="sc">~</span> <span class="st">"ALCAM"</span>,</span>
<span id="cb103-25"><a href="#cb103-25" aria-hidden="true" tabindex="-1"></a>                           Gal<span class="fl">.4</span> <span class="sc">~</span> <span class="st">"Gal-4"</span>,</span>
<span id="cb103-26"><a href="#cb103-26" aria-hidden="true" tabindex="-1"></a>                           PECAM<span class="fl">.1</span> <span class="sc">~</span> <span class="st">"PECAM-1"</span>,</span>
<span id="cb103-27"><a href="#cb103-27" aria-hidden="true" tabindex="-1"></a>                           GP6 <span class="sc">~</span> <span class="st">"GP6"</span>,</span>
<span id="cb103-28"><a href="#cb103-28" aria-hidden="true" tabindex="-1"></a>                           ST2 <span class="sc">~</span> <span class="st">"ST2"</span></span>
<span id="cb103-29"><a href="#cb103-29" aria-hidden="true" tabindex="-1"></a>            ),</span>
<span id="cb103-30"><a href="#cb103-30" aria-hidden="true" tabindex="-1"></a>            <span class="at">missing_text =</span> <span class="st">"(Missing)"</span></span>
<span id="cb103-31"><a href="#cb103-31" aria-hidden="true" tabindex="-1"></a>)  <span class="sc">%&gt;%</span> <span class="fu">modify_header</span>(label <span class="sc">~</span> <span class="st">"**Variable**"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb103-32"><a href="#cb103-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">modify_caption</span>(<span class="st">"**Baseline Measurements (Unmatched vs. Matched) 6MWD**"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="gcyjougpct" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#gcyjougpct table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#gcyjougpct thead, #gcyjougpct tbody, #gcyjougpct tfoot, #gcyjougpct tr, #gcyjougpct td, #gcyjougpct th {
  border-style: none;
}

#gcyjougpct p {
  margin: 0;
  padding: 0;
}

#gcyjougpct .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#gcyjougpct .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#gcyjougpct .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#gcyjougpct .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#gcyjougpct .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#gcyjougpct .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#gcyjougpct .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#gcyjougpct .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#gcyjougpct .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#gcyjougpct .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#gcyjougpct .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#gcyjougpct .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#gcyjougpct .gt_spanner_row {
  border-bottom-style: hidden;
}

#gcyjougpct .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#gcyjougpct .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#gcyjougpct .gt_from_md > :first-child {
  margin-top: 0;
}

#gcyjougpct .gt_from_md > :last-child {
  margin-bottom: 0;
}

#gcyjougpct .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#gcyjougpct .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#gcyjougpct .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#gcyjougpct .gt_row_group_first td {
  border-top-width: 2px;
}

#gcyjougpct .gt_row_group_first th {
  border-top-width: 2px;
}

#gcyjougpct .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#gcyjougpct .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#gcyjougpct .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#gcyjougpct .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#gcyjougpct .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#gcyjougpct .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#gcyjougpct .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#gcyjougpct .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#gcyjougpct .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#gcyjougpct .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#gcyjougpct .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#gcyjougpct .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#gcyjougpct .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#gcyjougpct .gt_left {
  text-align: left;
}

#gcyjougpct .gt_center {
  text-align: center;
}

#gcyjougpct .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#gcyjougpct .gt_font_normal {
  font-weight: normal;
}

#gcyjougpct .gt_font_bold {
  font-weight: bold;
}

#gcyjougpct .gt_font_italic {
  font-style: italic;
}

#gcyjougpct .gt_super {
  font-size: 65%;
}

#gcyjougpct .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#gcyjougpct .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#gcyjougpct .gt_indent_1 {
  text-indent: 5px;
}

#gcyjougpct .gt_indent_2 {
  text-indent: 10px;
}

#gcyjougpct .gt_indent_3 {
  text-indent: 15px;
}

#gcyjougpct .gt_indent_4 {
  text-indent: 20px;
}

#gcyjougpct .gt_indent_5 {
  text-indent: 25px;
}

#gcyjougpct .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#gcyjougpct div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<caption><strong>Baseline Measurements (Unmatched vs.&nbsp;Matched) 6MWD</strong></caption>
<colgroup>
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
<col style="width: 25%">
</colgroup>
<thead>
<tr class="gt_col_headings header">
<th id="label" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"><strong>Variable</strong></th>
<th id="stat_1" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>MatchedControl</strong><br> N = 71<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_2" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>MatchedIntervention</strong><br> N = 71<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
<th id="stat_3" class="gt_col_heading gt_columns_bottom_border gt_center" data-quarto-table-cell-role="th" scope="col"><strong>Unmatched</strong><br> N = 39<span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span></th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="label">Age</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">72 (7)</td>
<td class="gt_row gt_center" headers="stat_2">72 (8)</td>
<td class="gt_row gt_center" headers="stat_3">72 (8)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">72 (66, 77)</td>
<td class="gt_row gt_center" headers="stat_2">73 (64, 79)</td>
<td class="gt_row gt_center" headers="stat_3">69 (65, 77)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">60, 90</td>
<td class="gt_row gt_center" headers="stat_2">60, 91</td>
<td class="gt_row gt_center" headers="stat_3">60, 90</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Sex (Female=1)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">30 / 71 (42%)</td>
<td class="gt_row gt_center" headers="stat_2">41 / 71 (58%)</td>
<td class="gt_row gt_center" headers="stat_3">18 / 39 (46%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">41 / 71 (58%)</td>
<td class="gt_row gt_center" headers="stat_2">30 / 71 (42%)</td>
<td class="gt_row gt_center" headers="stat_3">21 / 39 (54%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">Race(White=1)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">34 / 71 (48%)</td>
<td class="gt_row gt_center" headers="stat_2">29 / 71 (41%)</td>
<td class="gt_row gt_center" headers="stat_3">17 / 39 (44%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">37 / 71 (52%)</td>
<td class="gt_row gt_center" headers="stat_2">42 / 71 (59%)</td>
<td class="gt_row gt_center" headers="stat_3">22 / 39 (56%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">HF Category (HFpEF=1) (Study)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;0</td>
<td class="gt_row gt_center" headers="stat_1">41 / 71 (58%)</td>
<td class="gt_row gt_center" headers="stat_2">33 / 71 (46%)</td>
<td class="gt_row gt_center" headers="stat_3">18 / 39 (46%)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;1</td>
<td class="gt_row gt_center" headers="stat_1">30 / 71 (42%)</td>
<td class="gt_row gt_center" headers="stat_2">38 / 71 (54%)</td>
<td class="gt_row gt_center" headers="stat_3">21 / 39 (54%)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">EGFR (Study)</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">57 (22)</td>
<td class="gt_row gt_center" headers="stat_2">52 (22)</td>
<td class="gt_row gt_center" headers="stat_3">57 (24)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">58 (36, 70)</td>
<td class="gt_row gt_center" headers="stat_2">47 (38, 63)</td>
<td class="gt_row gt_center" headers="stat_3">54 (38, 75)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">19, 101</td>
<td class="gt_row gt_center" headers="stat_2">17, 107</td>
<td class="gt_row gt_center" headers="stat_3">18, 98</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;(Missing)</td>
<td class="gt_row gt_center" headers="stat_1">1</td>
<td class="gt_row gt_center" headers="stat_2">0</td>
<td class="gt_row gt_center" headers="stat_3">0</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">ALCAM</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">6.66 (0.28)</td>
<td class="gt_row gt_center" headers="stat_2">6.66 (0.32)</td>
<td class="gt_row gt_center" headers="stat_3">6.78 (0.50)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">6.68 (6.44, 6.86)</td>
<td class="gt_row gt_center" headers="stat_2">6.66 (6.42, 6.86)</td>
<td class="gt_row gt_center" headers="stat_3">6.75 (6.34, 7.10)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">6.08, 7.22</td>
<td class="gt_row gt_center" headers="stat_2">5.99, 7.40</td>
<td class="gt_row gt_center" headers="stat_3">6.06, 7.82</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">Gal-4</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">3.88 (0.56)</td>
<td class="gt_row gt_center" headers="stat_2">3.95 (0.58)</td>
<td class="gt_row gt_center" headers="stat_3">3.93 (0.93)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">3.92 (3.54, 4.29)</td>
<td class="gt_row gt_center" headers="stat_2">3.95 (3.59, 4.37)</td>
<td class="gt_row gt_center" headers="stat_3">3.82 (3.27, 4.33)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">2.49, 5.48</td>
<td class="gt_row gt_center" headers="stat_2">2.74, 5.50</td>
<td class="gt_row gt_center" headers="stat_3">2.38, 6.13</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">PECAM-1</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">3.41 (0.36)</td>
<td class="gt_row gt_center" headers="stat_2">3.39 (0.34)</td>
<td class="gt_row gt_center" headers="stat_3">3.53 (0.47)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">3.44 (3.19, 3.68)</td>
<td class="gt_row gt_center" headers="stat_2">3.44 (3.15, 3.64)</td>
<td class="gt_row gt_center" headers="stat_3">3.41 (3.19, 3.75)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">2.52, 4.16</td>
<td class="gt_row gt_center" headers="stat_2">2.36, 4.23</td>
<td class="gt_row gt_center" headers="stat_3">2.75, 4.75</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">GP6</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">1.80 (0.46)</td>
<td class="gt_row gt_center" headers="stat_2">1.77 (0.62)</td>
<td class="gt_row gt_center" headers="stat_3">1.74 (0.55)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">1.83 (1.45, 2.13)</td>
<td class="gt_row gt_center" headers="stat_2">1.74 (1.38, 2.10)</td>
<td class="gt_row gt_center" headers="stat_3">1.71 (1.34, 2.17)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">0.83, 3.15</td>
<td class="gt_row gt_center" headers="stat_2">0.55, 3.55</td>
<td class="gt_row gt_center" headers="stat_3">0.69, 2.96</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">ST2</td>
<td class="gt_row gt_center" headers="stat_1"><br>
</td>
<td class="gt_row gt_center" headers="stat_2"><br>
</td>
<td class="gt_row gt_center" headers="stat_3"><br>
</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Mean (SD)</td>
<td class="gt_row gt_center" headers="stat_1">6.57 (0.86)</td>
<td class="gt_row gt_center" headers="stat_2">6.72 (0.81)</td>
<td class="gt_row gt_center" headers="stat_3">6.77 (0.80)</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Median (Q1, Q3)</td>
<td class="gt_row gt_center" headers="stat_1">6.48 (5.95, 7.16)</td>
<td class="gt_row gt_center" headers="stat_2">6.71 (6.20, 7.27)</td>
<td class="gt_row gt_center" headers="stat_3">6.74 (6.21, 7.36)</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="label">&nbsp;&nbsp;&nbsp;&nbsp;Min, Max</td>
<td class="gt_row gt_center" headers="stat_1">5.13, 9.44</td>
<td class="gt_row gt_center" headers="stat_2">5.13, 9.12</td>
<td class="gt_row gt_center" headers="stat_3">4.89, 8.52</td>
</tr>
</tbody><tfoot class="gt_footnotes">
<tr class="odd">
<td colspan="4" class="gt_footnote"><span class="gt_footnote_marks" style="white-space:nowrap;font-style:italic;font-weight:normal;line-height:0;"><sup>1</sup></span> n / N (%)</td>
</tr>
</tfoot>

</table>

</div>
</div>
</div>
</section>
</section>
<section id="change-in-protein-expression" class="level1">
<h1>Change In Protein Expression</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb104"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb104-1"><a href="#cb104-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create long form dataset</span></span>
<span id="cb104-2"><a href="#cb104-2" aria-hidden="true" tabindex="-1"></a>rprotfuds_long<span class="ot">=</span><span class="fu">pivot_longer</span>(<span class="at">data=</span>rprotfuds[,</span>
<span id="cb104-3"><a href="#cb104-3" aria-hidden="true" tabindex="-1"></a>                                           <span class="fu">c</span>(<span class="st">"subject_id"</span>,<span class="st">"timepoint"</span>,<span class="st">"intervention_1_control_0"</span>,</span>
<span id="cb104-4"><a href="#cb104-4" aria-hidden="true" tabindex="-1"></a>                                             <span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,protlist)],</span>
<span id="cb104-5"><a href="#cb104-5" aria-hidden="true" tabindex="-1"></a>                            <span class="at">cols=</span>protlist,</span>
<span id="cb104-6"><a href="#cb104-6" aria-hidden="true" tabindex="-1"></a>                            <span class="at">names_to=</span><span class="st">'Protein'</span>,<span class="at">values_to=</span><span class="st">'ExpressionLevel'</span>)</span>
<span id="cb104-7"><a href="#cb104-7" aria-hidden="true" tabindex="-1"></a>rprotfuds_long<span class="sc">$</span>sppb_chg <span class="ot">=</span> rprotfuds_long<span class="sc">$</span>fu_sppb <span class="sc">-</span> rprotfuds_long<span class="sc">$</span>bl_sppb</span>
<span id="cb104-8"><a href="#cb104-8" aria-hidden="true" tabindex="-1"></a>rprotfuds_long<span class="sc">$</span>smw_chg <span class="ot">=</span> rprotfuds_long<span class="sc">$</span>fu_smw <span class="sc">-</span> rprotfuds_long<span class="sc">$</span>bl_smw</span>
<span id="cb104-9"><a href="#cb104-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(rprotfuds_long)[<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="st">"Intervention"</span></span>
<span id="cb104-10"><a href="#cb104-10" aria-hidden="true" tabindex="-1"></a>rprotfuds_long<span class="sc">$</span>Protein <span class="ot">=</span> <span class="fu">factor</span>(rprotfuds_long<span class="sc">$</span>Protein,<span class="at">levels=</span>protlist)</span>
<span id="cb104-11"><a href="#cb104-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-12"><a href="#cb104-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-13"><a href="#cb104-13" aria-hidden="true" tabindex="-1"></a>rprotds_wide<span class="ot">=</span><span class="fu">merge</span>(rprotblds_long,</span>
<span id="cb104-14"><a href="#cb104-14" aria-hidden="true" tabindex="-1"></a>                   rprotfuds_long,<span class="at">by=</span><span class="fu">c</span>(<span class="st">'subject_id'</span>,<span class="st">'Protein'</span>,<span class="st">'Intervention'</span>,<span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,<span class="st">"sppb_chg"</span>,<span class="st">"smw_chg"</span>),</span>
<span id="cb104-15"><a href="#cb104-15" aria-hidden="true" tabindex="-1"></a>                   <span class="at">all.x=</span><span class="cn">TRUE</span>)</span>
<span id="cb104-16"><a href="#cb104-16" aria-hidden="true" tabindex="-1"></a>rprotds_wide<span class="sc">$</span>Protein_change <span class="ot">=</span> (rprotds_wide<span class="sc">$</span>ExpressionLevel.y) <span class="sc">-</span> (rprotds_wide<span class="sc">$</span>ExpressionLevel.x)</span>
<span id="cb104-17"><a href="#cb104-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-18"><a href="#cb104-18" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">ggplot</span>(<span class="at">data=</span>rprotds_wide,</span>
<span id="cb104-19"><a href="#cb104-19" aria-hidden="true" tabindex="-1"></a>             <span class="at">mapping=</span><span class="fu">aes</span>(<span class="at">x=</span>Intervention, <span class="at">y=</span>ExpressionLevel.y<span class="sc">/</span>ExpressionLevel.x, <span class="at">fill=</span>Intervention))<span class="sc">+</span></span>
<span id="cb104-20"><a href="#cb104-20" aria-hidden="true" tabindex="-1"></a>        <span class="fu">geom_boxplot</span>()<span class="sc">+</span><span class="fu">facet_wrap_paginate</span>(<span class="st">'Protein'</span>,<span class="at">scales=</span><span class="st">'free'</span>,<span class="at">ncol=</span><span class="dv">3</span>,</span>
<span id="cb104-21"><a href="#cb104-21" aria-hidden="true" tabindex="-1"></a>                                          <span class="at">page=</span><span class="dv">1</span>)<span class="sc">+</span>hw<span class="sc">+</span></span>
<span id="cb104-22"><a href="#cb104-22" aria-hidden="true" tabindex="-1"></a>        <span class="fu">labs</span>(<span class="at">y=</span><span class="st">"Ratio of change from baseline to 12-weeks"</span>)<span class="sc">+</span></span>
<span id="cb104-23"><a href="#cb104-23" aria-hidden="true" tabindex="-1"></a>        <span class="fu">geom_hline</span>(<span class="at">yintercept=</span><span class="dv">1</span>,<span class="at">linetype=</span><span class="dv">2</span>)<span class="sc">+</span></span>
<span id="cb104-24"><a href="#cb104-24" aria-hidden="true" tabindex="-1"></a>        <span class="fu">scale_y_continuous</span>(<span class="at">trans=</span><span class="st">'log2'</span>,<span class="at">labels =</span> <span class="cf">function</span>(x) <span class="fu">sprintf</span>(<span class="st">"%.3f"</span>, x),</span>
<span id="cb104-25"><a href="#cb104-25" aria-hidden="true" tabindex="-1"></a>                           <span class="at">breaks=</span><span class="fu">seq</span>(<span class="fl">0.5</span>,<span class="fl">1.5</span>,<span class="at">length.out=</span><span class="dv">6</span>))<span class="sc">+</span></span>
<span id="cb104-26"><a href="#cb104-26" aria-hidden="true" tabindex="-1"></a>        <span class="fu">scale_x_discrete</span>(<span class="at">breaks=</span><span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">0</span>),</span>
<span id="cb104-27"><a href="#cb104-27" aria-hidden="true" tabindex="-1"></a>                         <span class="at">labels=</span><span class="fu">c</span>(<span class="st">"RI"</span>,<span class="st">"AC"</span>))<span class="sc">+</span></span>
<span id="cb104-28"><a href="#cb104-28" aria-hidden="true" tabindex="-1"></a>        <span class="fu">scale_fill_discrete</span>(<span class="at">breaks=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>),</span>
<span id="cb104-29"><a href="#cb104-29" aria-hidden="true" tabindex="-1"></a>                            <span class="at">labels=</span><span class="fu">c</span>(<span class="st">"AC"</span>,<span class="st">"RI"</span>))<span class="sc">+</span></span>
<span id="cb104-30"><a href="#cb104-30" aria-hidden="true" tabindex="-1"></a>        <span class="fu">theme</span>(<span class="at">axis.title.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb104-31"><a href="#cb104-31" aria-hidden="true" tabindex="-1"></a>              <span class="at">axis.title.y =</span> <span class="fu">element_text</span>(<span class="at">face=</span><span class="st">"bold"</span>),</span>
<span id="cb104-32"><a href="#cb104-32" aria-hidden="true" tabindex="-1"></a>              <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb104-33"><a href="#cb104-33" aria-hidden="true" tabindex="-1"></a>              <span class="at">legend.position =</span> <span class="fu">c</span>(<span class="fl">0.85</span>,<span class="fl">0.15</span>),</span>
<span id="cb104-34"><a href="#cb104-34" aria-hidden="true" tabindex="-1"></a>              <span class="at">legend.box.background =</span> <span class="fu">element_rect</span>(<span class="at">color =</span> <span class="st">"black"</span>)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-27-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="causal-mediation-analysis" class="level1">
<h1>Causal Mediation Analysis</h1>
<section id="sppb-4" class="level2">
<h2 class="anchored" data-anchor-id="sppb-4">SPPB</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb105"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb105-1"><a href="#cb105-1" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="ot">=</span><span class="fu">pivot_longer</span>(<span class="at">data=</span>rprotblds[,</span>
<span id="cb105-2"><a href="#cb105-2" aria-hidden="true" tabindex="-1"></a>                                           <span class="fu">c</span>(<span class="st">"subject_id"</span>,<span class="st">"timepoint"</span>,<span class="st">"intervention_1_control_0"</span>,</span>
<span id="cb105-3"><a href="#cb105-3" aria-hidden="true" tabindex="-1"></a>                                             <span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,</span>
<span id="cb105-4"><a href="#cb105-4" aria-hidden="true" tabindex="-1"></a>                                             <span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,<span class="st">"hf_cat"</span>,protlist)],</span>
<span id="cb105-5"><a href="#cb105-5" aria-hidden="true" tabindex="-1"></a>                            <span class="at">cols=</span>protlist,</span>
<span id="cb105-6"><a href="#cb105-6" aria-hidden="true" tabindex="-1"></a>                            <span class="at">names_to=</span><span class="st">'Protein'</span>,<span class="at">values_to=</span><span class="st">'ExpressionLevel'</span>)</span>
<span id="cb105-7"><a href="#cb105-7" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="ot">=</span><span class="fu">pivot_longer</span>(<span class="at">data=</span>rprotfuds[,</span>
<span id="cb105-8"><a href="#cb105-8" aria-hidden="true" tabindex="-1"></a>                                            <span class="fu">c</span>(<span class="st">"subject_id"</span>,<span class="st">"timepoint"</span>,<span class="st">"intervention_1_control_0"</span>,</span>
<span id="cb105-9"><a href="#cb105-9" aria-hidden="true" tabindex="-1"></a>                                              <span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,</span>
<span id="cb105-10"><a href="#cb105-10" aria-hidden="true" tabindex="-1"></a>                                              <span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,<span class="st">"hf_cat"</span>,protlist)],</span>
<span id="cb105-11"><a href="#cb105-11" aria-hidden="true" tabindex="-1"></a>                             <span class="at">cols=</span>protlist,</span>
<span id="cb105-12"><a href="#cb105-12" aria-hidden="true" tabindex="-1"></a>                             <span class="at">names_to=</span><span class="st">'Protein'</span>,<span class="at">values_to=</span><span class="st">'ExpressionLevel'</span>)</span>
<span id="cb105-13"><a href="#cb105-13" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="sc">$</span>sppb_chg <span class="ot">=</span> rprotblds_long2<span class="sc">$</span>fu_sppb <span class="sc">-</span> rprotblds_long2<span class="sc">$</span>bl_sppb</span>
<span id="cb105-14"><a href="#cb105-14" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="sc">$</span>sppb_logchg <span class="ot">=</span> <span class="fu">log2</span>(rprotblds_long2<span class="sc">$</span>fu_sppb) <span class="sc">-</span> <span class="fu">log2</span>(rprotblds_long2<span class="sc">$</span>bl_sppb)</span>
<span id="cb105-15"><a href="#cb105-15" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="sc">$</span>smw_chg <span class="ot">=</span> rprotblds_long2<span class="sc">$</span>fu_smw <span class="sc">-</span> rprotblds_long2<span class="sc">$</span>bl_smw</span>
<span id="cb105-16"><a href="#cb105-16" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="sc">$</span>smw_logchg <span class="ot">=</span> <span class="fu">log2</span>(rprotblds_long2<span class="sc">$</span>fu_smw) <span class="sc">-</span> <span class="fu">log2</span>(rprotblds_long2<span class="sc">$</span>bl_smw)</span>
<span id="cb105-17"><a href="#cb105-17" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="sc">$</span>sppb_chg <span class="ot">=</span> rprotfuds_long2<span class="sc">$</span>fu_sppb <span class="sc">-</span> rprotfuds_long2<span class="sc">$</span>bl_sppb</span>
<span id="cb105-18"><a href="#cb105-18" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="sc">$</span>sppb_logchg <span class="ot">=</span> <span class="fu">log2</span>(rprotfuds_long2<span class="sc">$</span>fu_sppb) <span class="sc">-</span> <span class="fu">log2</span>(rprotfuds_long2<span class="sc">$</span>bl_sppb)</span>
<span id="cb105-19"><a href="#cb105-19" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="sc">$</span>smw_chg <span class="ot">=</span> rprotfuds_long2<span class="sc">$</span>fu_smw <span class="sc">-</span> rprotfuds_long2<span class="sc">$</span>bl_smw</span>
<span id="cb105-20"><a href="#cb105-20" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="sc">$</span>smw_logchg <span class="ot">=</span> <span class="fu">log2</span>(rprotfuds_long2<span class="sc">$</span>fu_smw) <span class="sc">-</span> <span class="fu">log2</span>(rprotfuds_long2<span class="sc">$</span>bl_smw)</span>
<span id="cb105-21"><a href="#cb105-21" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(rprotblds_long2)[<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="st">"Intervention"</span></span>
<span id="cb105-22"><a href="#cb105-22" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(rprotfuds_long2)[<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="st">"Intervention"</span></span>
<span id="cb105-23"><a href="#cb105-23" aria-hidden="true" tabindex="-1"></a>rprotblds_long2<span class="sc">$</span>Protein <span class="ot">=</span> <span class="fu">factor</span>(rprotblds_long2<span class="sc">$</span>Protein,<span class="at">levels=</span>protlist)</span>
<span id="cb105-24"><a href="#cb105-24" aria-hidden="true" tabindex="-1"></a>rprotfuds_long2<span class="sc">$</span>Protein <span class="ot">=</span> <span class="fu">factor</span>(rprotfuds_long2<span class="sc">$</span>Protein,<span class="at">levels=</span>protlist)</span>
<span id="cb105-25"><a href="#cb105-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb105-26"><a href="#cb105-26" aria-hidden="true" tabindex="-1"></a>dftmp.all<span class="ot">=</span><span class="fu">merge</span>(rprotblds_long2,</span>
<span id="cb105-27"><a href="#cb105-27" aria-hidden="true" tabindex="-1"></a>                   rprotfuds_long2,<span class="at">by=</span><span class="fu">c</span>(<span class="st">'subject_id'</span>,<span class="st">'Protein'</span>,<span class="st">'Intervention'</span>,</span>
<span id="cb105-28"><a href="#cb105-28" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">"bl_smw"</span>,<span class="st">"fu_smw"</span>,<span class="st">"bl_sppb"</span>,<span class="st">"fu_sppb"</span>,<span class="st">"sppb_chg"</span>,<span class="st">"smw_chg"</span>,</span>
<span id="cb105-29"><a href="#cb105-29" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">"sppb_logchg"</span>,<span class="st">"smw_logchg"</span>,</span>
<span id="cb105-30"><a href="#cb105-30" aria-hidden="true" tabindex="-1"></a>                                       <span class="st">"age"</span>,<span class="st">"sex"</span>,<span class="st">"race___4"</span>,<span class="st">"hf_cat"</span>),</span>
<span id="cb105-31"><a href="#cb105-31" aria-hidden="true" tabindex="-1"></a>                   <span class="at">all.x=</span><span class="cn">TRUE</span>)</span>
<span id="cb105-32"><a href="#cb105-32" aria-hidden="true" tabindex="-1"></a>dftmp.all<span class="sc">$</span>Protein_change <span class="ot">=</span> (dftmp.all<span class="sc">$</span>ExpressionLevel.y) <span class="sc">-</span> (dftmp.all<span class="sc">$</span>ExpressionLevel.x)</span>
<span id="cb105-33"><a href="#cb105-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb105-34"><a href="#cb105-34" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(protlist)){</span>
<span id="cb105-35"><a href="#cb105-35" aria-hidden="true" tabindex="-1"></a>  <span class="co">#limited to propensity matched participants</span></span>
<span id="cb105-36"><a href="#cb105-36" aria-hidden="true" tabindex="-1"></a>  dftmp <span class="ot">=</span> dftmp.all[<span class="sc">!</span><span class="fu">is.na</span>(dftmp.all<span class="sc">$</span>sppb_chg) <span class="sc">&amp;</span> </span>
<span id="cb105-37"><a href="#cb105-37" aria-hidden="true" tabindex="-1"></a>                      <span class="sc">!</span><span class="fu">is.na</span>(dftmp.all<span class="sc">$</span>Protein_change) <span class="sc">&amp;</span> </span>
<span id="cb105-38"><a href="#cb105-38" aria-hidden="true" tabindex="-1"></a>                      dftmp.all<span class="sc">$</span>subject_id <span class="sc">%in%</span> matchedset.sppb<span class="sc">$</span>subject_id <span class="sc">&amp;</span></span>
<span id="cb105-39"><a href="#cb105-39" aria-hidden="true" tabindex="-1"></a>                      dftmp.all<span class="sc">$</span>Protein<span class="sc">==</span>protlist[i],]</span>
<span id="cb105-40"><a href="#cb105-40" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb105-41"><a href="#cb105-41" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the effect of the intervention on the outcome</span></span>
<span id="cb105-42"><a href="#cb105-42" aria-hidden="true" tabindex="-1"></a>  total_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(sppb_chg <span class="sc">~</span>  Intervention<span class="sc">+</span>ExpressionLevel.x <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat,</span>
<span id="cb105-43"><a href="#cb105-43" aria-hidden="true" tabindex="-1"></a>                    <span class="at">data =</span> dftmp)</span>
<span id="cb105-44"><a href="#cb105-44" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb105-45"><a href="#cb105-45" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the effect of the intervention on the mediator (change in  level)</span></span>
<span id="cb105-46"><a href="#cb105-46" aria-hidden="true" tabindex="-1"></a>  mediator_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(Protein_change <span class="sc">~</span>  Intervention<span class="sc">+</span>ExpressionLevel.x <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat,</span>
<span id="cb105-47"><a href="#cb105-47" aria-hidden="true" tabindex="-1"></a>                       <span class="at">data =</span> dftmp)</span>
<span id="cb105-48"><a href="#cb105-48" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb105-49"><a href="#cb105-49" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the outcome with both the intervention and mediator (protein) included</span></span>
<span id="cb105-50"><a href="#cb105-50" aria-hidden="true" tabindex="-1"></a>  outcome_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(sppb_chg <span class="sc">~</span>  Intervention<span class="sc">+</span>Protein_change <span class="sc">+</span>ExpressionLevel.x<span class="sc">+</span>age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat, </span>
<span id="cb105-51"><a href="#cb105-51" aria-hidden="true" tabindex="-1"></a>                      <span class="at">data =</span> dftmp)</span>
<span id="cb105-52"><a href="#cb105-52" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb105-53"><a href="#cb105-53" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Estimate the direct and indirect effects using the mediation package</span></span>
<span id="cb105-54"><a href="#cb105-54" aria-hidden="true" tabindex="-1"></a>  med_results <span class="ot">&lt;-</span> <span class="fu">mediate</span>(mediator_model, outcome_model, </span>
<span id="cb105-55"><a href="#cb105-55" aria-hidden="true" tabindex="-1"></a>                         <span class="at">treat =</span> <span class="st">"Intervention"</span>, <span class="at">mediator =</span> <span class="st">"Protein_change"</span>,</span>
<span id="cb105-56"><a href="#cb105-56" aria-hidden="true" tabindex="-1"></a>                         <span class="at">boot =</span> <span class="cn">TRUE</span>, <span class="at">sims =</span> <span class="dv">1000</span>)  <span class="co"># Bootstrap for confidence intervals</span></span>
<span id="cb105-57"><a href="#cb105-57" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb105-58"><a href="#cb105-58" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Print summary of mediation results</span></span>
<span id="cb105-59"><a href="#cb105-59" aria-hidden="true" tabindex="-1"></a>  <span class="fu">print</span>(<span class="fu">summary</span>(med_results))</span>
<span id="cb105-60"><a href="#cb105-60" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot</span>(med_results,<span class="at">main=</span>protlist[i],<span class="at">cex.main=</span><span class="fl">1.5</span>,<span class="at">cex.lab=</span><span class="fl">1.2</span>)</span>
<span id="cb105-61"><a href="#cb105-61" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                 Estimate 95% CI Lower 95% CI Upper p-value    
ACME            0.0112777   -0.0443059    0.1470273   0.658    
ADE             1.4370262    0.7396219    2.1016893  &lt;2e-16 ***
Total Effect    1.4483039    0.7638837    2.1222320  &lt;2e-16 ***
Prop. Mediated  0.0077868   -0.0334760    0.1144329   0.658    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 128 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-28-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                  Estimate 95% CI Lower 95% CI Upper p-value    
ACME            0.00031026  -0.07258837   0.08038302   0.932    
ADE             1.45667418   0.80232970   2.14785395  &lt;2e-16 ***
Total Effect    1.45698445   0.78943598   2.14355269  &lt;2e-16 ***
Prop. Mediated  0.00021295  -0.05257309   0.06270897   0.932    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 128 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-28-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                  Estimate 95% CI Lower 95% CI Upper p-value    
ACME           -0.00067210  -0.09150643   0.08813166    0.96    
ADE             1.41428792   0.75421336   2.10143779  &lt;2e-16 ***
Total Effect    1.41361582   0.76131663   2.08730448  &lt;2e-16 ***
Prop. Mediated -0.00047545  -0.06785884   0.06450624    0.96    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 128 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-28-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                Estimate 95% CI Lower 95% CI Upper p-value    
ACME            0.035019    -0.109807     0.202821    0.57    
ADE             1.399091     0.693971     2.084499  &lt;2e-16 ***
Total Effect    1.434110     0.727782     2.116419  &lt;2e-16 ***
Prop. Mediated  0.024419    -0.089992     0.140424    0.57    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 128 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-28-4.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                Estimate 95% CI Lower 95% CI Upper p-value    
ACME           -0.131974    -0.375969     0.052969   0.186    
ADE             1.607860     0.873420     2.279391  &lt;2e-16 ***
Total Effect    1.475887     0.752042     2.154402  &lt;2e-16 ***
Prop. Mediated -0.089420    -0.335047     0.033717   0.186    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 128 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-28-5.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="mwd-4" class="level2">
<h2 class="anchored" data-anchor-id="mwd-4">6MWD</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb111"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb111-1"><a href="#cb111-1" aria-hidden="true" tabindex="-1"></a><span class="co">#6mwd</span></span>
<span id="cb111-2"><a href="#cb111-2" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(protlist)){</span>
<span id="cb111-3"><a href="#cb111-3" aria-hidden="true" tabindex="-1"></a>  <span class="co">#limited to propensity matched participants</span></span>
<span id="cb111-4"><a href="#cb111-4" aria-hidden="true" tabindex="-1"></a>  dftmp <span class="ot">=</span> dftmp.all[<span class="sc">!</span><span class="fu">is.na</span>(dftmp.all<span class="sc">$</span>smw_chg) <span class="sc">&amp;</span> </span>
<span id="cb111-5"><a href="#cb111-5" aria-hidden="true" tabindex="-1"></a>                      <span class="sc">!</span><span class="fu">is.na</span>(dftmp.all<span class="sc">$</span>Protein_change) <span class="sc">&amp;</span> </span>
<span id="cb111-6"><a href="#cb111-6" aria-hidden="true" tabindex="-1"></a>                      dftmp.all<span class="sc">$</span>subject_id <span class="sc">%in%</span> matchedset<span class="fl">.6</span>mwd<span class="sc">$</span>subject_id <span class="sc">&amp;</span></span>
<span id="cb111-7"><a href="#cb111-7" aria-hidden="true" tabindex="-1"></a>                      dftmp.all<span class="sc">$</span>Protein<span class="sc">==</span>protlist[i],]</span>
<span id="cb111-8"><a href="#cb111-8" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb111-9"><a href="#cb111-9" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the effect of the intervention on the outcome</span></span>
<span id="cb111-10"><a href="#cb111-10" aria-hidden="true" tabindex="-1"></a>  total_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(smw_chg <span class="sc">~</span>  Intervention<span class="sc">+</span>ExpressionLevel.x <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat,</span>
<span id="cb111-11"><a href="#cb111-11" aria-hidden="true" tabindex="-1"></a>                       <span class="at">data =</span> dftmp)</span>
<span id="cb111-12"><a href="#cb111-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb111-13"><a href="#cb111-13" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the effect of the intervention on the mediator (change in  level)</span></span>
<span id="cb111-14"><a href="#cb111-14" aria-hidden="true" tabindex="-1"></a>  mediator_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(Protein_change <span class="sc">~</span>  Intervention<span class="sc">+</span>ExpressionLevel.x <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat,</span>
<span id="cb111-15"><a href="#cb111-15" aria-hidden="true" tabindex="-1"></a>                        <span class="at">data =</span> dftmp)</span>
<span id="cb111-16"><a href="#cb111-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb111-17"><a href="#cb111-17" aria-hidden="true" tabindex="-1"></a>  <span class="co">#model the outcome with both the intervention and mediator (protein) included</span></span>
<span id="cb111-18"><a href="#cb111-18" aria-hidden="true" tabindex="-1"></a>  outcome_model <span class="ot">&lt;-</span> <span class="fu">lm</span>(smw_chg <span class="sc">~</span>  Intervention<span class="sc">+</span>Protein_change <span class="sc">+</span>ExpressionLevel.x<span class="sc">+</span>age <span class="sc">+</span> sex <span class="sc">+</span> race___4 <span class="sc">+</span> hf_cat, </span>
<span id="cb111-19"><a href="#cb111-19" aria-hidden="true" tabindex="-1"></a>                       <span class="at">data =</span> dftmp)</span>
<span id="cb111-20"><a href="#cb111-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb111-21"><a href="#cb111-21" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Estimate the direct and indirect effects using the mediation package</span></span>
<span id="cb111-22"><a href="#cb111-22" aria-hidden="true" tabindex="-1"></a>  med_results <span class="ot">&lt;-</span> <span class="fu">mediate</span>(mediator_model, outcome_model, </span>
<span id="cb111-23"><a href="#cb111-23" aria-hidden="true" tabindex="-1"></a>                         <span class="at">treat =</span> <span class="st">"Intervention"</span>, <span class="at">mediator =</span> <span class="st">"Protein_change"</span>,</span>
<span id="cb111-24"><a href="#cb111-24" aria-hidden="true" tabindex="-1"></a>                         <span class="at">boot =</span> <span class="cn">TRUE</span>, <span class="at">sims =</span> <span class="dv">1000</span>)  <span class="co"># Bootstrap for confidence intervals</span></span>
<span id="cb111-25"><a href="#cb111-25" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb111-26"><a href="#cb111-26" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Print summary of mediation results</span></span>
<span id="cb111-27"><a href="#cb111-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">print</span>(<span class="fu">summary</span>(med_results))</span>
<span id="cb111-28"><a href="#cb111-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot</span>(med_results,<span class="at">main=</span>protlist[i],<span class="at">cex.main=</span><span class="fl">1.5</span>,<span class="at">cex.lab=</span><span class="fl">1.2</span>)</span>
<span id="cb111-29"><a href="#cb111-29" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                  Estimate 95% CI Lower 95% CI Upper p-value    
ACME            -0.5775539   -7.3727621    8.8372515   0.866    
ADE             59.2197034   19.8044741  100.1762208   0.002 ** 
Total Effect    58.6421494   22.2009797  100.2017379  &lt;2e-16 ***
Prop. Mediated  -0.0098488   -0.1354355    0.1714835   0.866    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 111 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                  Estimate 95% CI Lower 95% CI Upper p-value   
ACME            -0.2364140   -6.1284251    4.4123838   0.842   
ADE             60.3374524   25.7116235  102.5295121   0.004 **
Total Effect    60.1010384   24.0783575  102.5090392   0.004 **
Prop. Mediated  -0.0039336   -0.1364621    0.0941301   0.842   
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 111 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-29-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                 Estimate 95% CI Lower 95% CI Upper p-value    
ACME            0.1300196   -4.5078270    6.0289991   0.906    
ADE            58.8295925   20.6889338   98.6789209  &lt;2e-16 ***
Total Effect   58.9596122   21.0618437   98.5372832  &lt;2e-16 ***
Prop. Mediated  0.0022052   -0.0926372    0.1362285   0.906    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 111 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-29-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                Estimate 95% CI Lower 95% CI Upper p-value    
ACME           -0.665863    -8.203288     6.810014   0.888    
ADE            59.899482    24.365834    98.173283  &lt;2e-16 ***
Total Effect   59.233619    22.454889    98.443266   0.004 ** 
Prop. Mediated -0.011241    -0.209431     0.126855   0.892    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 111 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-29-4.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
Causal Mediation Analysis 

Nonparametric Bootstrap Confidence Intervals with the Percentile Method

                 Estimate 95% CI Lower 95% CI Upper p-value    
ACME            -2.330332   -12.680389     5.656980     0.6    
ADE             61.292761    27.876866   104.654924  &lt;2e-16 ***
Total Effect    58.962429    24.840812   104.502621  &lt;2e-16 ***
Prop. Mediated  -0.039522    -0.323582     0.100626     0.6    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Sample Size Used: 111 


Simulations: 1000 </code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-29-5.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="wgcna-protein-clustering" class="level1">
<h1>WGCNA Protein Clustering</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb117"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb117-1"><a href="#cb117-1" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">=== WGCNA Module Analysis ===</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
=== WGCNA Module Analysis ===</code></pre>
</div>
<div class="sourceCode cell-code" id="cb119"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb119-1"><a href="#cb119-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualize correlation matrices</span></span>
<span id="cb119-2"><a href="#cb119-2" aria-hidden="true" tabindex="-1"></a><span class="fu">ggcorrplot</span>(<span class="fu">cor</span>(<span class="fu">na.omit</span>(rprotblds[, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>])),</span>
<span id="cb119-3"><a href="#cb119-3" aria-hidden="true" tabindex="-1"></a>                 <span class="at">title =</span> <span class="st">"All Participants (Baseline)"</span>,</span>
<span id="cb119-4"><a href="#cb119-4" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.order =</span> <span class="cn">TRUE</span>,</span>
<span id="cb119-5"><a href="#cb119-5" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.method =</span> <span class="st">"complete"</span></span>
<span id="cb119-6"><a href="#cb119-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span> hw <span class="sc">+</span> <span class="fu">theme</span>(</span>
<span id="cb119-7"><a href="#cb119-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb119-8"><a href="#cb119-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb119-9"><a href="#cb119-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb120"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb120-1"><a href="#cb120-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Correlation matrix for usual care</span></span>
<span id="cb120-2"><a href="#cb120-2" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggcorrplot</span>(<span class="fu">cor</span>(<span class="fu">na.omit</span>(rprotblds[rprotblds<span class="sc">$</span>intervention_1_control_0 <span class="sc">==</span> <span class="dv">0</span>, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>])),</span>
<span id="cb120-3"><a href="#cb120-3" aria-hidden="true" tabindex="-1"></a>                 <span class="at">title =</span> <span class="st">"Usual Care Patients (Baseline)"</span>,</span>
<span id="cb120-4"><a href="#cb120-4" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.order =</span> <span class="cn">TRUE</span>,</span>
<span id="cb120-5"><a href="#cb120-5" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.method =</span> <span class="st">"complete"</span></span>
<span id="cb120-6"><a href="#cb120-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span> hw <span class="sc">+</span></span>
<span id="cb120-7"><a href="#cb120-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb120-8"><a href="#cb120-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb120-9"><a href="#cb120-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb120-10"><a href="#cb120-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>)</span>
<span id="cb120-11"><a href="#cb120-11" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb120-12"><a href="#cb120-12" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb121"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb121-1"><a href="#cb121-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Correlation matrix for rehabilitation</span></span>
<span id="cb121-2"><a href="#cb121-2" aria-hidden="true" tabindex="-1"></a>cormat2 <span class="ot">&lt;-</span> <span class="fu">cor</span>(<span class="fu">na.omit</span>(rprotblds[rprotblds<span class="sc">$</span>intervention_1_control_0 <span class="sc">==</span> <span class="dv">1</span>,</span>
<span id="cb121-3"><a href="#cb121-3" aria-hidden="true" tabindex="-1"></a>                                 <span class="fu">as.character</span>(p1<span class="sc">$</span>data[<span class="dv">1</span><span class="sc">:</span><span class="dv">92</span>, <span class="dv">1</span>])]))</span>
<span id="cb121-4"><a href="#cb121-4" aria-hidden="true" tabindex="-1"></a><span class="fu">ggcorrplot</span>(cormat2,</span>
<span id="cb121-5"><a href="#cb121-5" aria-hidden="true" tabindex="-1"></a>                 <span class="at">title =</span> <span class="st">"Rehab Patients (Baseline)"</span>,</span>
<span id="cb121-6"><a href="#cb121-6" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.order =</span> <span class="cn">FALSE</span>,</span>
<span id="cb121-7"><a href="#cb121-7" aria-hidden="true" tabindex="-1"></a>                 <span class="at">hc.method =</span> <span class="st">"complete"</span></span>
<span id="cb121-8"><a href="#cb121-8" aria-hidden="true" tabindex="-1"></a>) <span class="sc">+</span> hw <span class="sc">+</span></span>
<span id="cb121-9"><a href="#cb121-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb121-10"><a href="#cb121-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb121-11"><a href="#cb121-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb121-12"><a href="#cb121-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>)</span>
<span id="cb121-13"><a href="#cb121-13" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb122"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb122-1"><a href="#cb122-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Test equality of covariance matrices between intervention groups</span></span>
<span id="cb122-2"><a href="#cb122-2" aria-hidden="true" tabindex="-1"></a><span class="fu">testCov</span>(</span>
<span id="cb122-3"><a href="#cb122-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">na.omit</span>(rprotblds[rprotblds<span class="sc">$</span>intervention_1_control_0 <span class="sc">==</span> <span class="dv">0</span>,</span>
<span id="cb122-4"><a href="#cb122-4" aria-hidden="true" tabindex="-1"></a>                    <span class="fu">as.character</span>(p1<span class="sc">$</span>data[<span class="dv">1</span><span class="sc">:</span><span class="dv">92</span>, <span class="dv">1</span>])]),</span>
<span id="cb122-5"><a href="#cb122-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">na.omit</span>(rprotblds[rprotblds<span class="sc">$</span>intervention_1_control_0 <span class="sc">==</span> <span class="dv">1</span>,</span>
<span id="cb122-6"><a href="#cb122-6" aria-hidden="true" tabindex="-1"></a>                    <span class="fu">as.character</span>(p1<span class="sc">$</span>data[<span class="dv">1</span><span class="sc">:</span><span class="dv">92</span>, <span class="dv">1</span>])]),</span>
<span id="cb122-7"><a href="#cb122-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">method =</span> <span class="st">"ALL"</span>,</span>
<span id="cb122-8"><a href="#cb122-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">J =</span> <span class="dv">5000</span>,</span>
<span id="cb122-9"><a href="#cb122-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">alpha =</span> <span class="fl">0.05</span>,</span>
<span id="cb122-10"><a href="#cb122-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">n.core =</span> <span class="dv">1</span></span>
<span id="cb122-11"><a href="#cb122-11" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>$HD

    Two-Sample HD test

data:  na.omit(rprotblds[rprotblds$intervention_1_control_0 == 0, as.character(p1$data[1:92,  and na.omit(rprotblds[rprotblds$intervention_1_control_0 == 1, as.character(p1$data[1:92,     1])]) and     1])])
Statistic = 3.5841, p-value = 0.4858
alternative hypothesis: two.sided


$CLX

    Two-Sample CLX test

data:  na.omit(rprotblds[rprotblds$intervention_1_control_0 == 0, as.character(p1$data[1:92,  and na.omit(rprotblds[rprotblds$intervention_1_control_0 == 1, as.character(p1$data[1:92,     1])]) and     1])])
Statistic = 12.846, p-value = 0.7246
alternative hypothesis: two.sided


$Scott

    Two-Sample Scott test

data:  na.omit(rprotblds[rprotblds$intervention_1_control_0 == 0, as.character(p1$data[1:92,  and na.omit(rprotblds[rprotblds$intervention_1_control_0 == 1, as.character(p1$data[1:92,     1])]) and     1])])
Statistic = 0.36301, p-value = 0.7166
alternative hypothesis: two.sided


[[4]]

    Two-Sample LC test

data:  na.omit(rprotblds[rprotblds$intervention_1_control_0 == 0, as.character(p1$data[1:92,  and na.omit(rprotblds[rprotblds$intervention_1_control_0 == 1, as.character(p1$data[1:92,     1])]) and     1])])
Statistic = -0.58919, p-value = 0.7221
alternative hypothesis: two.sided</code></pre>
</div>
<div class="sourceCode cell-code" id="cb124"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb124-1"><a href="#cb124-1" aria-hidden="true" tabindex="-1"></a><span class="co"># No significant difference between groups at baseline</span></span>
<span id="cb124-2"><a href="#cb124-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb124-3"><a href="#cb124-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Standardize protein expressions</span></span>
<span id="cb124-4"><a href="#cb124-4" aria-hidden="true" tabindex="-1"></a>naset <span class="ot">&lt;-</span> <span class="fu">which</span>(<span class="fu">apply</span>(rprotblds[, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>], <span class="dv">1</span>, <span class="cf">function</span>(x) <span class="fu">sum</span>(<span class="fu">is.na</span>(x))) <span class="sc">==</span> <span class="dv">1</span>)</span>
<span id="cb124-5"><a href="#cb124-5" aria-hidden="true" tabindex="-1"></a>rprotblds.std <span class="ot">&lt;-</span> <span class="fu">apply</span>(rprotblds[<span class="sc">-</span>naset, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>], <span class="dv">2</span>, <span class="cf">function</span>(x) (x <span class="sc">-</span> <span class="fu">mean</span>(x) <span class="sc">/</span> <span class="fu">sd</span>(x)))</span>
<span id="cb124-6"><a href="#cb124-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb124-7"><a href="#cb124-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Find optimal soft-thresholding power for scale-free network</span></span>
<span id="cb124-8"><a href="#cb124-8" aria-hidden="true" tabindex="-1"></a>powers <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">20</span>), <span class="fu">seq</span>(<span class="at">from =</span> <span class="dv">22</span>, <span class="at">to =</span> <span class="dv">30</span>, <span class="at">by =</span> <span class="dv">2</span>))</span>
<span id="cb124-9"><a href="#cb124-9" aria-hidden="true" tabindex="-1"></a>sft <span class="ot">&lt;-</span> <span class="fu">pickSoftThreshold</span>(<span class="fu">na.omit</span>(rprotblds[, <span class="dv">3</span><span class="sc">:</span><span class="dv">94</span>]), <span class="at">powerVector =</span> powers, <span class="at">verbose =</span> <span class="dv">5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>pickSoftThreshold: will use block size 92.
 pickSoftThreshold: calculating connectivity for given powers...
   ..working on genes 1 through 92 of 92</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>   Power SFT.R.sq    slope truncated.R.sq  mean.k. median.k. max.k.
1      1   0.0001  0.00672        0.31400 20.80000  2.09e+01 37.100
2      2   0.7360 -0.76300        0.68100  7.65000  6.53e+00 20.600
3      3   0.8990 -0.93700        0.87200  3.59000  2.44e+00 13.300
4      4   0.9300 -1.04000        0.91200  1.96000  1.06e+00  9.290
5      5   0.8420 -1.11000        0.80700  1.19000  5.38e-01  6.900
6      6   0.0811 -1.53000       -0.08630  0.77200  2.62e-01  5.340
7      7   0.2200 -2.41000        0.05950  0.53100  1.34e-01  4.260
8      8   0.2200 -2.24000        0.07920  0.38200  7.06e-02  3.460
9      9   0.2080 -2.85000       -0.00276  0.28500  3.65e-02  2.870
10    10   0.2570 -3.01000        0.14200  0.21800  1.92e-02  2.410
11    11   0.2330 -3.20000        0.03110  0.17100  1.07e-02  2.040
12    12   0.2460 -3.24000        0.05210  0.13600  6.18e-03  1.750
13    13   0.2750 -3.33000        0.12300  0.11000  3.54e-03  1.500
14    14   0.2570 -3.49000        0.04900  0.09050  2.08e-03  1.310
15    15   0.2670 -3.50000        0.06370  0.07510  1.25e-03  1.140
16    16   0.2750 -3.52000        0.07630  0.06290  7.53e-04  0.997
17    17   0.3020 -3.57000        0.13000  0.05310  4.44e-04  0.877
18    18   0.3100 -3.50000        0.14400  0.04520  2.63e-04  0.774
19    19   0.2680 -3.27000        0.05890  0.03870  1.57e-04  0.685
20    20   0.2760 -3.23000        0.06970  0.03340  9.41e-05  0.608
21    22   0.3170 -3.41000        0.12300  0.02510  3.32e-05  0.482
22    24   0.3330 -3.47000        0.14300  0.01920  1.22e-05  0.386
23    26   0.3660 -3.45000        0.19400  0.01490  4.49e-06  0.316
24    28   0.3340 -3.47000        0.16600  0.01170  1.66e-06  0.260
25    30   0.3470 -3.38000        0.17700  0.00933  5.87e-07  0.216</code></pre>
</div>
<div class="sourceCode cell-code" id="cb127"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb127-1"><a href="#cb127-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot scale-free network fit</span></span>
<span id="cb127-2"><a href="#cb127-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">1</span>], <span class="sc">-</span><span class="fu">sign</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">3</span>]) <span class="sc">*</span> sft<span class="sc">$</span>fitIndices[, <span class="dv">2</span>],</span>
<span id="cb127-3"><a href="#cb127-3" aria-hidden="true" tabindex="-1"></a>     <span class="at">xlab =</span> <span class="st">"Soft Threshold (power)"</span>,</span>
<span id="cb127-4"><a href="#cb127-4" aria-hidden="true" tabindex="-1"></a>     <span class="at">ylab =</span> <span class="st">"Scale Free Topology Model Fit, signed R^2"</span>,</span>
<span id="cb127-5"><a href="#cb127-5" aria-hidden="true" tabindex="-1"></a>     <span class="at">type =</span> <span class="st">"n"</span>,</span>
<span id="cb127-6"><a href="#cb127-6" aria-hidden="true" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">"Scale independence"</span></span>
<span id="cb127-7"><a href="#cb127-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb127-8"><a href="#cb127-8" aria-hidden="true" tabindex="-1"></a><span class="fu">text</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">1</span>], <span class="sc">-</span><span class="fu">sign</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">3</span>]) <span class="sc">*</span> sft<span class="sc">$</span>fitIndices[, <span class="dv">2</span>],</span>
<span id="cb127-9"><a href="#cb127-9" aria-hidden="true" tabindex="-1"></a>     <span class="at">labels =</span> powers, <span class="at">cex =</span> <span class="fl">0.9</span>, <span class="at">col =</span> <span class="st">"red"</span></span>
<span id="cb127-10"><a href="#cb127-10" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb127-11"><a href="#cb127-11" aria-hidden="true" tabindex="-1"></a><span class="fu">abline</span>(<span class="at">h =</span> <span class="fl">0.90</span>, <span class="at">col =</span> <span class="st">"red"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-4.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb128"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb128-1"><a href="#cb128-1" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mar =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>, <span class="dv">1</span>))</span>
<span id="cb128-2"><a href="#cb128-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">1</span>], sft<span class="sc">$</span>fitIndices[, <span class="dv">5</span>],</span>
<span id="cb128-3"><a href="#cb128-3" aria-hidden="true" tabindex="-1"></a>     <span class="at">xlab =</span> <span class="st">"Soft Threshold (power)"</span>,</span>
<span id="cb128-4"><a href="#cb128-4" aria-hidden="true" tabindex="-1"></a>     <span class="at">ylab =</span> <span class="st">"Mean Connectivity"</span>,</span>
<span id="cb128-5"><a href="#cb128-5" aria-hidden="true" tabindex="-1"></a>     <span class="at">type =</span> <span class="st">"n"</span>,</span>
<span id="cb128-6"><a href="#cb128-6" aria-hidden="true" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">"Mean connectivity"</span></span>
<span id="cb128-7"><a href="#cb128-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb128-8"><a href="#cb128-8" aria-hidden="true" tabindex="-1"></a><span class="fu">text</span>(sft<span class="sc">$</span>fitIndices[, <span class="dv">1</span>], sft<span class="sc">$</span>fitIndices[, <span class="dv">5</span>], <span class="at">labels =</span> sft<span class="sc">$</span>fitIndices[, <span class="dv">1</span>], <span class="at">col =</span> <span class="st">"red"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-5.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb129"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb129-1"><a href="#cb129-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Create adjacency matrix using soft-thresholding</span></span>
<span id="cb129-2"><a href="#cb129-2" aria-hidden="true" tabindex="-1"></a>adj1.all <span class="ot">&lt;-</span> <span class="fu">abs</span>(<span class="fu">cor</span>(rprotblds.std))<span class="sc">^</span><span class="dv">4</span></span>
<span id="cb129-3"><a href="#cb129-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb129-4"><a href="#cb129-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert adjacency to topological overlap matrix (TOM)</span></span>
<span id="cb129-5"><a href="#cb129-5" aria-hidden="true" tabindex="-1"></a><span class="co"># TOM is more robust to noise than raw correlation</span></span>
<span id="cb129-6"><a href="#cb129-6" aria-hidden="true" tabindex="-1"></a>TOM.dissimilarity <span class="ot">&lt;-</span> <span class="dv">1</span> <span class="sc">-</span> <span class="fu">TOMsimilarity</span>(adj1.all)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>..connectivity..
..matrix multiplication (system BLAS)..
..normalization..
..done.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb131"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb131-1"><a href="#cb131-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Hierarchical clustering using TOM-based distance</span></span>
<span id="cb131-2"><a href="#cb131-2" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(TOM.dissimilarity) <span class="ot">&lt;-</span> <span class="fu">colnames</span>(rprotblds.std)</span>
<span id="cb131-3"><a href="#cb131-3" aria-hidden="true" tabindex="-1"></a>hierADJ <span class="ot">&lt;-</span> <span class="fu">hclust</span>(<span class="fu">as.dist</span>(TOM.dissimilarity), <span class="at">method =</span> <span class="st">'complete'</span>)</span>
<span id="cb131-4"><a href="#cb131-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb131-5"><a href="#cb131-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Identify modules using dynamic tree cutting</span></span>
<span id="cb131-6"><a href="#cb131-6" aria-hidden="true" tabindex="-1"></a>Modules <span class="ot">&lt;-</span> <span class="fu">cutreeDynamic</span>(</span>
<span id="cb131-7"><a href="#cb131-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">dendro =</span> hierADJ,</span>
<span id="cb131-8"><a href="#cb131-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">distM =</span> TOM.dissimilarity,</span>
<span id="cb131-9"><a href="#cb131-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">deepSplit =</span> <span class="dv">2</span>,</span>
<span id="cb131-10"><a href="#cb131-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">pamRespectsDendro =</span> <span class="cn">FALSE</span>,</span>
<span id="cb131-11"><a href="#cb131-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">minClusterSize =</span> <span class="dv">6</span></span>
<span id="cb131-12"><a href="#cb131-12" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> ..cutHeight not given, setting it to 0.996  ===&gt;  99% of the (truncated) height range in dendro.
 ..done.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb133"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb133-1"><a href="#cb133-1" aria-hidden="true" tabindex="-1"></a>ModuleColors <span class="ot">&lt;-</span> <span class="fu">labels2colors</span>(Modules)</span>
<span id="cb133-2"><a href="#cb133-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb133-3"><a href="#cb133-3" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"Protein module color assignments:</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Protein module color assignments:</code></pre>
</div>
<div class="sourceCode cell-code" id="cb135"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb135-1"><a href="#cb135-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">table</span>(ModuleColors))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>ModuleColors
     blue     brown     green      grey turquoise    yellow 
       21        13         8         5        35        10 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb137"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb137-1"><a href="#cb137-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot dendrogram with module colors</span></span>
<span id="cb137-2"><a href="#cb137-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotDendroAndColors</span>(hierADJ, ModuleColors, <span class="st">"Module"</span>,</span>
<span id="cb137-3"><a href="#cb137-3" aria-hidden="true" tabindex="-1"></a>                    <span class="at">dendroLabels =</span> <span class="cn">NULL</span>,</span>
<span id="cb137-4"><a href="#cb137-4" aria-hidden="true" tabindex="-1"></a>                    <span class="at">cex.dendroLabels =</span> <span class="fl">0.5</span>,</span>
<span id="cb137-5"><a href="#cb137-5" aria-hidden="true" tabindex="-1"></a>                    <span class="at">addGuide =</span> <span class="cn">FALSE</span>,</span>
<span id="cb137-6"><a href="#cb137-6" aria-hidden="true" tabindex="-1"></a>                    <span class="at">main =</span> <span class="st">"Protein dendrogram and module colors"</span></span>
<span id="cb137-7"><a href="#cb137-7" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-6.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb138"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb138-1"><a href="#cb138-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate module eigengenes (summarizes module expression)</span></span>
<span id="cb138-2"><a href="#cb138-2" aria-hidden="true" tabindex="-1"></a>MElist <span class="ot">&lt;-</span> <span class="fu">moduleEigengenes</span>(<span class="fu">as.data.frame</span>(rprotblds.std), <span class="at">colors =</span> ModuleColors)</span>
<span id="cb138-3"><a href="#cb138-3" aria-hidden="true" tabindex="-1"></a>MEs <span class="ot">&lt;-</span> MElist<span class="sc">$</span>eigengenes</span>
<span id="cb138-4"><a href="#cb138-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-5"><a href="#cb138-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Order modules so similar ones are adjacent</span></span>
<span id="cb138-6"><a href="#cb138-6" aria-hidden="true" tabindex="-1"></a>MEs0 <span class="ot">&lt;-</span> <span class="fu">orderMEs</span>(MEs)</span>
<span id="cb138-7"><a href="#cb138-7" aria-hidden="true" tabindex="-1"></a>module_order <span class="ot">&lt;-</span> <span class="fu">names</span>(MEs0) <span class="sc">%&gt;%</span> <span class="fu">gsub</span>(<span class="st">"ME"</span>, <span class="st">""</span>, .)</span>
<span id="cb138-8"><a href="#cb138-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-9"><a href="#cb138-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Add clinical data to module eigengene dataframe</span></span>
<span id="cb138-10"><a href="#cb138-10" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>ID <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>subject_id[<span class="sc">-</span>naset]</span>
<span id="cb138-11"><a href="#cb138-11" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>age <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>age[<span class="sc">-</span>naset]</span>
<span id="cb138-12"><a href="#cb138-12" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>sex <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(rprotblds<span class="sc">$</span>sex[<span class="sc">-</span>naset]) <span class="sc">-</span> <span class="dv">1</span></span>
<span id="cb138-13"><a href="#cb138-13" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>bl_smw <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>bl_smw[<span class="sc">-</span>naset]</span>
<span id="cb138-14"><a href="#cb138-14" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>fu_smw <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>fu_smw[<span class="sc">-</span>naset]</span>
<span id="cb138-15"><a href="#cb138-15" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>chg_smw <span class="ot">&lt;-</span> MEs0<span class="sc">$</span>fu_smw <span class="sc">-</span> MEs0<span class="sc">$</span>bl_smw</span>
<span id="cb138-16"><a href="#cb138-16" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>bl_sppb <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>bl_sppb[<span class="sc">-</span>naset]</span>
<span id="cb138-17"><a href="#cb138-17" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>fu_sppb <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>fu_sppb[<span class="sc">-</span>naset]</span>
<span id="cb138-18"><a href="#cb138-18" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>chg_sppb <span class="ot">&lt;-</span> MEs0<span class="sc">$</span>fu_sppb <span class="sc">-</span> MEs0<span class="sc">$</span>bl_sppb</span>
<span id="cb138-19"><a href="#cb138-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-20"><a href="#cb138-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate correlations between module eigengenes and clinical outcomes</span></span>
<span id="cb138-21"><a href="#cb138-21" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">melt</span>(<span class="fu">cor</span>(MEs0[, <span class="sc">-</span><span class="fu">which</span>(<span class="fu">names</span>(MEs0) <span class="sc">==</span> <span class="st">"ID"</span>)], <span class="at">use =</span> <span class="st">"pairwise.complete.obs"</span>))</span>
<span id="cb138-22"><a href="#cb138-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-23"><a href="#cb138-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Filter to relevant correlations</span></span>
<span id="cb138-24"><a href="#cb138-24" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> df[(df<span class="sc">$</span>Var1 <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">'age'</span>, <span class="st">'bl_smw'</span>, <span class="st">'chg_smw'</span>, <span class="st">'bl_sppb'</span>, <span class="st">'chg_sppb'</span>, <span class="st">'sex'</span>)), ]</span>
<span id="cb138-25"><a href="#cb138-25" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> df[(df<span class="sc">$</span>Var2 <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">'MEturquoise'</span>, <span class="st">'MEbrown'</span>, <span class="st">'MEyellow'</span>, <span class="st">'MEblue'</span>, <span class="st">'MEgreen'</span>)), ]</span>
<span id="cb138-26"><a href="#cb138-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-27"><a href="#cb138-27" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate p-values</span></span>
<span id="cb138-28"><a href="#cb138-28" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(df)) {</span>
<span id="cb138-29"><a href="#cb138-29" aria-hidden="true" tabindex="-1"></a>  b1 <span class="ot">&lt;-</span> <span class="fu">as.character</span>(df<span class="sc">$</span>Var1[i])</span>
<span id="cb138-30"><a href="#cb138-30" aria-hidden="true" tabindex="-1"></a>  b2 <span class="ot">&lt;-</span> <span class="fu">as.character</span>(df<span class="sc">$</span>Var2[i])</span>
<span id="cb138-31"><a href="#cb138-31" aria-hidden="true" tabindex="-1"></a>  df<span class="sc">$</span>pval[i] <span class="ot">&lt;-</span> <span class="fu">cor.test</span>(</span>
<span id="cb138-32"><a href="#cb138-32" aria-hidden="true" tabindex="-1"></a>    MEs0[, <span class="fu">which</span>(<span class="fu">colnames</span>(MEs0) <span class="sc">%in%</span> b1)],</span>
<span id="cb138-33"><a href="#cb138-33" aria-hidden="true" tabindex="-1"></a>    MEs0[, <span class="fu">which</span>(<span class="fu">colnames</span>(MEs0) <span class="sc">%in%</span> b2)],</span>
<span id="cb138-34"><a href="#cb138-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">method =</span> <span class="st">"pearson"</span>,</span>
<span id="cb138-35"><a href="#cb138-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">use =</span> <span class="st">"pairwise.complete.obs"</span></span>
<span id="cb138-36"><a href="#cb138-36" aria-hidden="true" tabindex="-1"></a>  )<span class="sc">$</span>p.value</span>
<span id="cb138-37"><a href="#cb138-37" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb138-38"><a href="#cb138-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-39"><a href="#cb138-39" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>sig <span class="ot">&lt;-</span> df<span class="sc">$</span>pval <span class="sc">&lt;</span> <span class="fl">0.01</span></span>
<span id="cb138-40"><a href="#cb138-40" aria-hidden="true" tabindex="-1"></a>df<span class="sc">$</span>text <span class="ot">&lt;-</span> <span class="fu">ifelse</span>(</span>
<span id="cb138-41"><a href="#cb138-41" aria-hidden="true" tabindex="-1"></a>  df<span class="sc">$</span>sig <span class="sc">==</span> <span class="cn">TRUE</span> <span class="sc">&amp;</span> df<span class="sc">$</span>value <span class="sc">!=</span> <span class="dv">1</span>,</span>
<span id="cb138-42"><a href="#cb138-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">paste0</span>(<span class="fu">round</span>(df<span class="sc">$</span>value, <span class="dv">3</span>), <span class="st">"*"</span>),</span>
<span id="cb138-43"><a href="#cb138-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">round</span>(df<span class="sc">$</span>value, <span class="dv">3</span>)</span>
<span id="cb138-44"><a href="#cb138-44" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb138-45"><a href="#cb138-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb138-46"><a href="#cb138-46" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot module-outcome heatmap</span></span>
<span id="cb138-47"><a href="#cb138-47" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(</span>
<span id="cb138-48"><a href="#cb138-48" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(df, <span class="fu">aes</span>(Var2, Var1, <span class="at">fill =</span> value)) <span class="sc">+</span></span>
<span id="cb138-49"><a href="#cb138-49" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_tile</span>(<span class="at">color =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb138-50"><a href="#cb138-50" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_gradient2</span>(</span>
<span id="cb138-51"><a href="#cb138-51" aria-hidden="true" tabindex="-1"></a>      <span class="at">low =</span> <span class="st">"blue"</span>, <span class="at">high =</span> <span class="st">"red"</span>, <span class="at">mid =</span> <span class="st">"white"</span>,</span>
<span id="cb138-52"><a href="#cb138-52" aria-hidden="true" tabindex="-1"></a>      <span class="at">midpoint =</span> <span class="dv">0</span>, <span class="at">limit =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="fl">0.5</span>, <span class="fl">0.5</span>), <span class="at">space =</span> <span class="st">"Lab"</span>,</span>
<span id="cb138-53"><a href="#cb138-53" aria-hidden="true" tabindex="-1"></a>      <span class="at">name =</span> <span class="st">"Pearson</span><span class="sc">\n</span><span class="st">Correlation"</span></span>
<span id="cb138-54"><a href="#cb138-54" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">+</span></span>
<span id="cb138-55"><a href="#cb138-55" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb138-56"><a href="#cb138-56" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">vjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">12</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb138-57"><a href="#cb138-57" aria-hidden="true" tabindex="-1"></a>    <span class="fu">coord_fixed</span>() <span class="sc">+</span></span>
<span id="cb138-58"><a href="#cb138-58" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_text</span>(<span class="fu">aes</span>(Var2, Var1, <span class="at">label =</span> text), <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">4</span>) <span class="sc">+</span></span>
<span id="cb138-59"><a href="#cb138-59" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(</span>
<span id="cb138-60"><a href="#cb138-60" aria-hidden="true" tabindex="-1"></a>      <span class="at">axis.title.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-61"><a href="#cb138-61" aria-hidden="true" tabindex="-1"></a>      <span class="at">axis.title.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-62"><a href="#cb138-62" aria-hidden="true" tabindex="-1"></a>      <span class="at">panel.grid.major =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-63"><a href="#cb138-63" aria-hidden="true" tabindex="-1"></a>      <span class="at">panel.border =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-64"><a href="#cb138-64" aria-hidden="true" tabindex="-1"></a>      <span class="at">panel.background =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-65"><a href="#cb138-65" aria-hidden="true" tabindex="-1"></a>      <span class="at">axis.ticks =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb138-66"><a href="#cb138-66" aria-hidden="true" tabindex="-1"></a>      <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb138-67"><a href="#cb138-67" aria-hidden="true" tabindex="-1"></a>      <span class="at">legend.direction =</span> <span class="st">"horizontal"</span></span>
<span id="cb138-68"><a href="#cb138-68" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">+</span></span>
<span id="cb138-69"><a href="#cb138-69" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_colorbar</span>(</span>
<span id="cb138-70"><a href="#cb138-70" aria-hidden="true" tabindex="-1"></a>      <span class="at">barwidth =</span> <span class="dv">15</span>, <span class="at">barheight =</span> <span class="dv">1</span>,</span>
<span id="cb138-71"><a href="#cb138-71" aria-hidden="true" tabindex="-1"></a>      <span class="at">title.position =</span> <span class="st">"top"</span>, <span class="at">title.hjust =</span> <span class="fl">0.5</span></span>
<span id="cb138-72"><a href="#cb138-72" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb138-73"><a href="#cb138-73" aria-hidden="true" tabindex="-1"></a>    <span class="fu">coord_flip</span>()</span>
<span id="cb138-74"><a href="#cb138-74" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="REHAB-HF_ProteomicsMarkdown_cleaned_011326_files/figure-html/unnamed-chunk-30-7.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb139"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb139-1"><a href="#cb139-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Add treatment assignment to module eigengenes</span></span>
<span id="cb139-2"><a href="#cb139-2" aria-hidden="true" tabindex="-1"></a>MEs0<span class="sc">$</span>treatment <span class="ot">&lt;-</span> rprotblds<span class="sc">$</span>intervention_1_control_0[<span class="sc">-</span>naset]</span>
<span id="cb139-3"><a href="#cb139-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb139-4"><a href="#cb139-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Test module-treatment interactions for outcomes</span></span>
<span id="cb139-5"><a href="#cb139-5" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">=== Module x Treatment Interactions ===</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
=== Module x Treatment Interactions ===</code></pre>
</div>
<div class="sourceCode cell-code" id="cb141"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb141-1"><a href="#cb141-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_sppb <span class="sc">~</span> bl_sppb <span class="sc">+</span> MEturquoise <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_sppb ~ bl_sppb + MEturquoise * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-9.1556 -1.3491  0.3993  1.8797  5.2605 

Coefficients:
                       Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)             2.28849    0.52762   4.337  2.3e-05 ***
bl_sppb                 0.70709    0.07248   9.756  &lt; 2e-16 ***
MEturquoise            -6.27205    4.35906  -1.439 0.151779    
treatment1              1.41930    0.36754   3.862 0.000153 ***
MEturquoise:treatment1  2.31193    5.68193   0.407 0.684529    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 2.605 on 197 degrees of freedom
  (30 observations deleted due to missingness)
Multiple R-squared:  0.3972,    Adjusted R-squared:  0.385 
F-statistic: 32.46 on 4 and 197 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb143"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb143-1"><a href="#cb143-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_smw <span class="sc">~</span> bl_smw <span class="sc">+</span> MEturquoise <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_smw ~ bl_smw + MEturquoise * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-263.24  -56.13    1.88   55.82  531.28 

Coefficients:
                         Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)              86.78248   16.95734   5.118 8.43e-07 ***
bl_smw                    0.80907    0.06599  12.261  &lt; 2e-16 ***
MEturquoise            -356.50951  175.54876  -2.031   0.0439 *  
treatment1               36.67628   13.77237   2.663   0.0085 ** 
MEturquoise:treatment1  202.10817  220.65888   0.916   0.3610    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 89.59 on 167 degrees of freedom
  (60 observations deleted due to missingness)
Multiple R-squared:  0.5392,    Adjusted R-squared:  0.5282 
F-statistic: 48.86 on 4 and 167 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb145"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb145-1"><a href="#cb145-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_smw <span class="sc">~</span> bl_smw <span class="sc">+</span> MEgreen <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_smw ~ bl_smw + MEgreen * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-263.08  -55.17    8.54   50.25  525.01 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)          79.57198   17.06046   4.664 6.32e-06 ***
bl_smw                0.85223    0.06495  13.121  &lt; 2e-16 ***
MEgreen              32.55467  170.85634   0.191    0.849    
treatment1           34.22478   14.06454   2.433    0.016 *  
MEgreen:treatment1 -138.04937  216.19842  -0.639    0.524    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 90.76 on 167 degrees of freedom
  (60 observations deleted due to missingness)
Multiple R-squared:  0.5271,    Adjusted R-squared:  0.5158 
F-statistic: 46.53 on 4 and 167 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb147"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb147-1"><a href="#cb147-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_smw <span class="sc">~</span> bl_smw <span class="sc">+</span> MEblue <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_smw ~ bl_smw + MEblue * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-285.65  -59.77    7.34   51.59  543.48 

Coefficients:
                    Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)         85.32702   17.16907   4.970 1.65e-06 ***
bl_smw               0.82670    0.06602  12.522  &lt; 2e-16 ***
MEblue            -284.49950  166.03730  -1.713   0.0885 .  
treatment1          35.29331   13.85073   2.548   0.0117 *  
MEblue:treatment1  277.14520  218.11182   1.271   0.2056    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 90.15 on 167 degrees of freedom
  (60 observations deleted due to missingness)
Multiple R-squared:  0.5334,    Adjusted R-squared:  0.5222 
F-statistic: 47.73 on 4 and 167 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb149"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb149-1"><a href="#cb149-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_sppb <span class="sc">~</span> bl_sppb <span class="sc">+</span> MEbrown <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_sppb ~ bl_sppb + MEbrown * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-9.1851 -1.1418  0.3634  1.8630  4.2998 

Coefficients:
                    Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)          2.33819    0.52447   4.458 1.39e-05 ***
bl_sppb              0.68660    0.07288   9.422  &lt; 2e-16 ***
MEbrown            -10.93469    4.54174  -2.408    0.017 *  
treatment1           1.50505    0.36636   4.108 5.85e-05 ***
MEbrown:treatment1   7.07336    5.75308   1.229    0.220    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 2.583 on 197 degrees of freedom
  (30 observations deleted due to missingness)
Multiple R-squared:  0.4078,    Adjusted R-squared:  0.3957 
F-statistic: 33.91 on 4 and 197 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb151"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb151-1"><a href="#cb151-1" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(<span class="fu">summary</span>(<span class="fu">lm</span>(<span class="at">data =</span> MEs0, <span class="at">formula =</span> fu_smw <span class="sc">~</span> bl_smw <span class="sc">+</span> MEyellow <span class="sc">*</span> treatment)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
lm(formula = fu_smw ~ bl_smw + MEyellow * treatment, data = MEs0)

Residuals:
    Min      1Q  Median      3Q     Max 
-262.66  -57.34    3.07   55.79  541.09 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)           84.2256    16.7438   5.030 1.26e-06 ***
bl_smw                 0.8167     0.0646  12.641  &lt; 2e-16 ***
MEyellow            -402.5395   160.1981  -2.513   0.0129 *  
treatment1            37.9462    13.6872   2.772   0.0062 ** 
MEyellow:treatment1  228.9395   221.5374   1.033   0.3029    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Residual standard error: 89 on 167 degrees of freedom
  (60 observations deleted due to missingness)
Multiple R-squared:  0.5453,    Adjusted R-squared:  0.5344 
F-statistic: 50.07 on 4 and 167 DF,  p-value: &lt; 2.2e-16</code></pre>
</div>
<div class="sourceCode cell-code" id="cb153"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb153-1"><a href="#cb153-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Extract most influential proteins within each module (top 5)</span></span>
<span id="cb153-2"><a href="#cb153-2" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\n</span><span class="st">=== Top 5 Proteins per Module ===</span><span class="sc">\n</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
=== Top 5 Proteins per Module ===</code></pre>
</div>
<div class="sourceCode cell-code" id="cb155"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb155-1"><a href="#cb155-1" aria-hidden="true" tabindex="-1"></a>top5mat <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="dv">0</span>,<span class="dv">5</span>,<span class="dv">6</span>)</span>
<span id="cb155-2"><a href="#cb155-2" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(top5mat) <span class="ot">&lt;-</span> <span class="fu">rep</span>(<span class="st">""</span>,<span class="dv">6</span>)</span>
<span id="cb155-3"><a href="#cb155-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span>(<span class="fu">which</span>(<span class="fu">names</span>(MEs0) <span class="sc">==</span> <span class="st">"ID"</span>) <span class="sc">-</span> <span class="dv">1</span>)) {</span>
<span id="cb155-4"><a href="#cb155-4" aria-hidden="true" tabindex="-1"></a>  clr <span class="ot">&lt;-</span> <span class="fu">sapply</span>(<span class="fu">names</span>(MEs0)[<span class="dv">1</span><span class="sc">:</span>(<span class="fu">which</span>(<span class="fu">names</span>(MEs0) <span class="sc">==</span> <span class="st">"ID"</span>) <span class="sc">-</span> <span class="dv">1</span>)],</span>
<span id="cb155-5"><a href="#cb155-5" aria-hidden="true" tabindex="-1"></a>                <span class="cf">function</span>(x) <span class="fu">substring</span>(x, <span class="dv">3</span>, <span class="fu">nchar</span>(x)))[i]</span>
<span id="cb155-6"><a href="#cb155-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">colnames</span>(top5mat)[i] <span class="ot">&lt;-</span> <span class="fu">as.character</span>(clr)</span>
<span id="cb155-7"><a href="#cb155-7" aria-hidden="true" tabindex="-1"></a>  geneModuleMembership <span class="ot">&lt;-</span> </span>
<span id="cb155-8"><a href="#cb155-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">cor</span>(rprotblds.std[, ModuleColors <span class="sc">==</span> clr], MEs0[, i], <span class="at">use =</span> <span class="st">"p"</span>)</span>
<span id="cb155-9"><a href="#cb155-9" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb155-10"><a href="#cb155-10" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb155-11"><a href="#cb155-11" aria-hidden="true" tabindex="-1"></a>  MMPvalue <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(</span>
<span id="cb155-12"><a href="#cb155-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">corPvalueStudent</span>(<span class="fu">as.matrix</span>(geneModuleMembership), <span class="fu">nrow</span>(rprotblds.std))</span>
<span id="cb155-13"><a href="#cb155-13" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb155-14"><a href="#cb155-14" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb155-15"><a href="#cb155-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">names</span>(geneModuleMembership) <span class="ot">&lt;-</span> <span class="fu">paste</span>(<span class="st">"MM"</span>, <span class="fu">substring</span>(</span>
<span id="cb155-16"><a href="#cb155-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(MEs0)[<span class="dv">1</span><span class="sc">:</span>(<span class="fu">which</span>(<span class="fu">names</span>(MEs0) <span class="sc">==</span> <span class="st">"ID"</span>) <span class="sc">-</span> <span class="dv">1</span>)], <span class="dv">3</span></span>
<span id="cb155-17"><a href="#cb155-17" aria-hidden="true" tabindex="-1"></a>  ), <span class="at">sep =</span> <span class="st">""</span>)[i]</span>
<span id="cb155-18"><a href="#cb155-18" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb155-19"><a href="#cb155-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">names</span>(MMPvalue) <span class="ot">&lt;-</span> <span class="fu">paste</span>(<span class="st">"p.MM"</span>, <span class="fu">substring</span>(</span>
<span id="cb155-20"><a href="#cb155-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(MEs0)[<span class="dv">1</span><span class="sc">:</span>(<span class="fu">which</span>(<span class="fu">names</span>(MEs0) <span class="sc">==</span> <span class="st">"ID"</span>) <span class="sc">-</span> <span class="dv">1</span>)], <span class="dv">3</span></span>
<span id="cb155-21"><a href="#cb155-21" aria-hidden="true" tabindex="-1"></a>  ), <span class="at">sep =</span> <span class="st">""</span>)[i]</span>
<span id="cb155-22"><a href="#cb155-22" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb155-23"><a href="#cb155-23" aria-hidden="true" tabindex="-1"></a>  top5mat[,i] <span class="ot">&lt;-</span> <span class="fu">matrix</span>(<span class="fu">rownames</span>(geneModuleMembership[<span class="fu">order</span>(geneModuleMembership[, <span class="dv">1</span>], <span class="at">decreasing =</span> <span class="cn">TRUE</span>)[<span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>], <span class="dv">1</span>, <span class="at">drop =</span> <span class="cn">FALSE</span>]),<span class="at">ncol=</span><span class="dv">1</span>)</span>
<span id="cb155-24"><a href="#cb155-24" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb155-25"><a href="#cb155-25" aria-hidden="true" tabindex="-1"></a><span class="fu">gt</span>(<span class="fu">as.data.frame</span>(top5mat), <span class="at">rownames_to_stub =</span> <span class="cn">TRUE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb155-26"><a href="#cb155-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(<span class="at">title =</span> <span class="st">"Top 5 Proteins per Module"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="zzkyfmehtn" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#zzkyfmehtn table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#zzkyfmehtn thead, #zzkyfmehtn tbody, #zzkyfmehtn tfoot, #zzkyfmehtn tr, #zzkyfmehtn td, #zzkyfmehtn th {
  border-style: none;
}

#zzkyfmehtn p {
  margin: 0;
  padding: 0;
}

#zzkyfmehtn .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#zzkyfmehtn .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#zzkyfmehtn .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#zzkyfmehtn .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#zzkyfmehtn .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zzkyfmehtn .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zzkyfmehtn .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zzkyfmehtn .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#zzkyfmehtn .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#zzkyfmehtn .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#zzkyfmehtn .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#zzkyfmehtn .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#zzkyfmehtn .gt_spanner_row {
  border-bottom-style: hidden;
}

#zzkyfmehtn .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#zzkyfmehtn .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#zzkyfmehtn .gt_from_md > :first-child {
  margin-top: 0;
}

#zzkyfmehtn .gt_from_md > :last-child {
  margin-bottom: 0;
}

#zzkyfmehtn .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#zzkyfmehtn .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#zzkyfmehtn .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#zzkyfmehtn .gt_row_group_first td {
  border-top-width: 2px;
}

#zzkyfmehtn .gt_row_group_first th {
  border-top-width: 2px;
}

#zzkyfmehtn .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zzkyfmehtn .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#zzkyfmehtn .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#zzkyfmehtn .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zzkyfmehtn .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zzkyfmehtn .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#zzkyfmehtn .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#zzkyfmehtn .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#zzkyfmehtn .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zzkyfmehtn .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zzkyfmehtn .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zzkyfmehtn .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zzkyfmehtn .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zzkyfmehtn .gt_left {
  text-align: left;
}

#zzkyfmehtn .gt_center {
  text-align: center;
}

#zzkyfmehtn .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#zzkyfmehtn .gt_font_normal {
  font-weight: normal;
}

#zzkyfmehtn .gt_font_bold {
  font-weight: bold;
}

#zzkyfmehtn .gt_font_italic {
  font-style: italic;
}

#zzkyfmehtn .gt_super {
  font-size: 65%;
}

#zzkyfmehtn .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#zzkyfmehtn .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#zzkyfmehtn .gt_indent_1 {
  text-indent: 5px;
}

#zzkyfmehtn .gt_indent_2 {
  text-indent: 10px;
}

#zzkyfmehtn .gt_indent_3 {
  text-indent: 15px;
}

#zzkyfmehtn .gt_indent_4 {
  text-indent: 20px;
}

#zzkyfmehtn .gt_indent_5 {
  text-indent: 25px;
}

#zzkyfmehtn .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#zzkyfmehtn div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="7" class="gt_heading gt_title gt_font_normal gt_bottom_border">Top 5 Proteins per Module</th>
</tr>
<tr class="gt_col_headings even">
<th id="a::stub" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col"></th>
<th id="green" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">green</th>
<th id="blue" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">blue</th>
<th id="yellow" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">yellow</th>
<th id="brown" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">brown</th>
<th id="turquoise" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">turquoise</th>
<th id="grey" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">grey</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td id="stub_1_1" class="gt_row gt_right gt_stub" data-quarto-table-cell-role="th" scope="row">1</td>
<td class="gt_row gt_left" headers="stub_1_1 green">MMP-9</td>
<td class="gt_row gt_left" headers="stub_1_1 blue">Notch 3</td>
<td class="gt_row gt_left" headers="stub_1_1 yellow">IGFBP-2</td>
<td class="gt_row gt_left" headers="stub_1_1 brown">CSTB</td>
<td class="gt_row gt_left" headers="stub_1_1 turquoise">EPHB4</td>
<td class="gt_row gt_left" headers="stub_1_1 grey">PON3</td>
</tr>
<tr class="even">
<td id="stub_1_2" class="gt_row gt_right gt_stub" data-quarto-table-cell-role="th" scope="row">2</td>
<td class="gt_row gt_left" headers="stub_1_2 green">AZU1</td>
<td class="gt_row gt_left" headers="stub_1_2 blue">IGFBP-7</td>
<td class="gt_row gt_left" headers="stub_1_2 yellow">OPN</td>
<td class="gt_row gt_left" headers="stub_1_2 brown">U-PAR</td>
<td class="gt_row gt_left" headers="stub_1_2 turquoise">TNF-R1</td>
<td class="gt_row gt_left" headers="stub_1_2 grey">Ep-CAM</td>
</tr>
<tr class="odd">
<td id="stub_1_3" class="gt_row gt_right gt_stub" data-quarto-table-cell-role="th" scope="row">3</td>
<td class="gt_row gt_left" headers="stub_1_3 green">PRTN3</td>
<td class="gt_row gt_left" headers="stub_1_3 blue">ALCAM</td>
<td class="gt_row gt_left" headers="stub_1_3 yellow">AXL</td>
<td class="gt_row gt_left" headers="stub_1_3 brown">JAM-A</td>
<td class="gt_row gt_left" headers="stub_1_3 turquoise">LTBR</td>
<td class="gt_row gt_left" headers="stub_1_3 grey">CCL24</td>
</tr>
<tr class="even">
<td id="stub_1_4" class="gt_row gt_right gt_stub" data-quarto-table-cell-role="th" scope="row">4</td>
<td class="gt_row gt_left" headers="stub_1_4 green">MPO</td>
<td class="gt_row gt_left" headers="stub_1_4 blue">CDH5</td>
<td class="gt_row gt_left" headers="stub_1_4 yellow">SHPS-1</td>
<td class="gt_row gt_left" headers="stub_1_4 brown">TFF3</td>
<td class="gt_row gt_left" headers="stub_1_4 turquoise">TNFRSF14</td>
<td class="gt_row gt_left" headers="stub_1_4 grey">t-PA</td>
</tr>
<tr class="odd">
<td id="stub_1_5" class="gt_row gt_right gt_stub" data-quarto-table-cell-role="th" scope="row">5</td>
<td class="gt_row gt_left" headers="stub_1_5 green">PAI</td>
<td class="gt_row gt_left" headers="stub_1_5 blue">IL-1RT1</td>
<td class="gt_row gt_left" headers="stub_1_5 yellow">IGFBP-1</td>
<td class="gt_row gt_left" headers="stub_1_5 brown">FABP4</td>
<td class="gt_row gt_left" headers="stub_1_5 turquoise">IL-18BP</td>
<td class="gt_row gt_left" headers="stub_1_5 grey">CPA1</td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>